// (temp) mock data for the UI development

export default {
  SearchInput: {
    token: { placeholder: 'Search accessions, pathways, diseases, etc.,' }
  },
  SearchBulkActions: {
    PAGE_SIZES: [10, 20, 50],
    SORTBY_OPTIONS: [
      'Best Match',
      'Most No. of samples',
      'Least No. of samples',
      'Newest Experiment',
      'Oldest Experiment'
    ],
    totalResults: 120
  },
  SearchCardHeader: {
    title:
      'Transcription profiling of human neuroblast tumours reveals two distinct gene signatures identify malignant Neuroblast and Schwannian stromal cells',
    accessionId: 'GSE7530',
    token: {
      learnmore: '#',
      added: 'Added to Dataset',
      not_supported: 'Platform not supported',
      add_remaining: '4 samples already in My Dataset',
      unavailable: 'Not available',
      processing: 'Processing Dataset',
      qn_skipped: 'Quantile normailzation will be skipped'
    }
  },
  SearchCardMeta: [
    { id: 1, name: 'Organism', label: 'Homo sapiens' },
    { id: 2, name: 'Samples', label: '23 Downloadable Samples' },
    { id: 3, name: 'MicroArray', label: 'Affymetrix-335' }
  ],
  SearchCardBody: {
    url: '#',
    meta_fields: ['Sex', 'Age', ' Specimen part']
  },
  // For filters
  // Taken from the endpoint: https://api.refine.bio/v1/search/
  SearchFilters: {
    technology: {
      microarray: 609303,
      'rna-seq': 350551,
      unknown: 1200
    },
    organism: {
      HOMO_SAPIENS: 521539,
      MUS_MUSCULUS: 292681,
      ARABIDOPSIS_THALIANA: 23999,
      RATTUS_NORVEGICUS: 39311,
      DROSOPHILA_MELANOGASTER: 17239,
      SACCHAROMYCES_CEREVISIAE: 12989,
      DANIO_RERIO: 28598,
      CAENORHABDITIS_ELEGANS: 7764,
      SUS_SCROFA: 5974,
      ZEA_MAYS: 3391,
      GALLUS_GALLUS: 4038,
      GLYCINE_MAX: 5153,
      ORYZA_SATIVA: 2520,
      PSEUDOMONAS_AERUGINOSA: 2263,
      BOS_TAURUS: 5526,
      ESCHERICHIA_COLI: 1387,
      MACACA_MULATTA: 2307,
      TRITICUM_AESTIVUM: 2083,
      XENOPUS_LAEVIS: 759,
      POPULUS_TRICHOCARPA: 1698,
      HORDEUM_VULGARE: 2206,
      ANOPHELES_GAMBIAE: 808,
      OVIS_ARIES: 925,
      PSEUDOMONAS_AERUGINOSA_PAO1: 342,
      STAPHYLOCOCCUS_AUREUS: 420,
      SCHIZOSACCHAROMYCES_POMBE: 310,
      VITIS_VINIFERA: 793,
      ORYZA_SATIVA_INDICA_GROUP: 706,
      MACACA_FASCICULARIS: 979,
      POPULUS_TREMULA_X_POPULUS_ALBA: 246,
      'ESCHERICHIA_COLI_STR._K-12_SUBSTR._MG1655': 551,
      GOSSYPIUM_HIRSUTUM: 318,
      CITRUS_SINENSIS: 248,
      'ESCHERICHIA_COLI_K-12': 152,
      MUSTELA_PUTORIUS_FURO: 217,
      PLASMODIUM_FALCIPARUM: 106,
      POPULUS_X_CANADENSIS: 143,
      BOS_INDICUS: 57,
      LEPIDIUM_SATIVUM: 139,
      VITIS_HYBRID_CULTIVAR: 251,
      VITIS_RIPARIA: 251
    },
    platforms: {
      IlluminaHiSeq2000: 125755,
      IlluminaHiSeq2500: 142474,
      hgu133plus2: 146159,
      mouse4302: 59093,
      NextSeq500: 48894,
      'Illumina_HumanHT-12_V4.0': 25203,
      mogene10st: 29454,
      hugene10st: 38914,
      IlluminaHiSeq4000: 20752,
      ath1121501: 17553,
      'Illumina_MouseWG-6_v2.0': 5873,
      hgu133a: 50743,
      IlluminaGenomeAnalyzerIIx: 9676,
      'Illumina_MouseRef-8_v2.0': 5092,
      mogene20st: 8487,
      IlluminaGenomeAnalyzerII: 6374,
      rat2302: 21304,
      mgu74av2: 7353,
      hugene20st: 3879,
      hgu133a2: 24621,
      'Illumina_HumanHT-12_V3.0': 11388,
      'HuEx-1_0-st': 25055,
      mouse430a2: 8776,
      moe430a: 7306,
      IlluminaHiSeq3000: 3488,
      IlluminaMiSeq: 944,
      IlluminaGenomeAnalyzer: 2590,
      hta20: 12895,
      IlluminaHiSeq1500: 4359,
      yeast2: 4320,
      'Illumina_HumanWG-6_V3.0': 2003,
      drosophila2: 4684,
      primeview: 869,
      IonTorrentProton: 4424,
      hgu95av2: 3408,
      IlluminaHiSeq1000: 2333,
      'Illumina_HumanRef-8_v3.0': 3812,
      htmg430pm: 150,
      ragene10st: 3232,
      'Illumina_HumanRef-8_v2.0': 2357,
      'Illumina_Human-6_V2.0': 3778,
      hugene11st: 8579,
      ygs98: 2558,
      rice: 3020,
      rgu34a: 3465,
      paeg1a: 1314,
      ecoli2: 1710,
      porcine: 3486,
      'Illumina_RatRef-12_V1.0': 913,
      hgu219: 12227,
      mogene11st: 4234,
      celegans: 2469,
      rae230a: 4895,
      chicken: 2211,
      'Illumina_Mouse-6_v1.1': 723,
      zebrafish: 1852,
      mogene21st: 2599,
      bovine: 2389,
      hugene21st: 4373,
      mta10: 6,
      'Illumina HiSeq 2000': 957,
      hthgu133pluspm: 12261,
      IlluminaNovaSeq6000: 167,
      drosgenome1: 1271,
      NextSeq550: 1280,
      wheat: 2006,
      IlluminaHiScanSQ: 1364,
      clariomsmouse: 1090,
      rhesus: 1734,
      hthgu133a: 13003,
      'Illumina_MouseRef-8_V1.1': 627,
      poplar: 517,
      ragene20st: 2910,
      barley1: 2206,
      hgu95a: 2868,
      'Illumina_Human-6_v1.0': 819,
      clariomshuman: 787,
      maize: 735,
      soybean: 3896,
      'Illumina HiSeq 2500': 101,
      UNKNOWN: 644,
      saureus: 420,
      'Illumina_Mouse-6_V1.0': 116,
      xenopuslaevis: 386,
      clariomdhuman: 8,
      vitisvinifera: 1038,
      aragene10st: 525,
      GPL13534: 446,
      mgu74a: 710,
      xlaevis2: 301,
      'Illumina_MouseRef-8_V1.0': 190,
      ag: 149,
      hgu133b: 1150,
      'Illumina Genome Analyzer IIx': 116,
      aragene11st: 685,
      mu11ksuba: 272,
      plasmodiumanopheles: 537,
      'Illumina Genome Analyzer': 84,
      GPL5175: 60,
      GPL80: 592,
      'Illumina Genome Analyzer II': 98,
      citrus: 248,
      htmg430a: 3198,
      cotton: 318,
      moe430b: 465,
      ragene11st: 530,
      GPL16384: 116,
      zebgene10st: 257,
      GPL6801: 575,
      bovgene10st: 277,
      'Illumina_HumanRef-8_v1.0': 38,
      ragene21st: 462,
      canine2: 156,
      GPL8490: 47,
      porgene10st: 222,
      GPL3718: 219,
      GPL5811: 39,
      chigene10st: 416,
      elegene10st: 152,
      GPL6096: 109,
      mgu74bv2: 132,
      GPL5082: 139,
      'Illumina HiSeq 4000': 6,
      porgene11st: 299,
      GPL4134: 22,
      drogene10st: 123,
      clariomsrat: 137,
      hgu95b: 616,
      zebgene11st: 304,
      GPL8179: 24,
      IonTorrentS5: 82,
      mgu74cv2: 105,
      GPL4091: 46,
      bovgene11st: 181,
      GPL10787: 20,
      GPL4133: 58,
      MinION: 127,
      'ABSOLiDSystem3.0': 204,
      GPL16284: 60,
      chigene11st: 59,
      clariomshumanht: 144,
      hgu95c: 518,
      huex10st: 111,
      ovigene11st: 97,
      rae230b: 233,
      GPL3720: 15,
      IonTorrentS5XL: 65,
      rhegene10st: 110,
      GPL1352: 68,
      GPL14550: 11,
      GPL2004: 18,
      GPL2005: 184,
      GPL9128: 19,
      drogene11st: 38,
      ovigene10st: 37,
      GPL23541: 31,
      mu11ksubb: 82,
      GPL10989: 18,
      GPL13497: 6,
      GPL15018: 12,
      GPL15076: 56,
      GPL15236: 5,
      GPL16131: 12,
      GPL18609: 180,
      GPL7546: 75,
      GPL9138: 14,
      'Illumina HiScanSQ': 12,
      elegene11st: 34,
      rcngene10st: 12,
      rcngene11st: 12,
      rgu34b: 37,
      rusgene10st: 24,
      soygene11st: 35,
      GPL13118: 4,
      GPL13147: 16,
      GPL13328: 24,
      GPL16304: 25,
      GPL2040: 23,
      GPL21145: 64,
      GPL3562: 48,
      GPL4124: 14,
      GPL5770: 2,
      GPL6193: 2,
      GPL7566: 87,
      GPL8234: 4,
      IlluminaMiniSeq: 44,
      hgu95d: 15,
      hgu95e: 15,
      rgu34c: 17,
      rhegene11st: 16,
      rusgene11st: 44
    }
  },
  // For SearchCard
  // Taken from the endpoint: https://api.refine.bio/v1/search/
  /* TEMPORARY
  (1) In the first item, edit the 'publication_title' to be '' for no publication state(in mockup)
  (2) The following property is added in all items for demo purpose
  property name: 'status' 
   - ''(default)
   - added
   - processing 
   - add_remaining
   - not_supported
   - request
   - unavailable
   - qn_skipped
  */
  SearchResults: {
    results: [
      {
        status: '',
        id: 51993,
        title: 'Transcriptional profiling of cochlear neurons',
        publication_title: '',
        description:
          "In order to characterize the various subpopulations of neurons present in the inner ear, we dissected out the cochleas from wild-type mice, removing non-neuronal populations as possible, then dissociated the tissue to obtain a single cell suspension and profiled these by single cell RNA-Seq. Overall design: This study''s goal was to characterize the native populations of murine cochlear neurons. To ensure reproducibility of our findings, neurons from 4 separate mice were profiled.",
        technology: 'RNA-SEQ',
        accession_code: 'SRP148677',
        alternate_accession_code: 'GSE114759',
        submitter_institution: '',
        has_publication: true,
        publication_doi: '',
        publication_authors: [
          'Sun S',
          'Babola T',
          'Pregernig G',
          'So KS',
          'Nguyen M',
          'Su SM',
          'Palermo AT',
          'Bergles DE',
          'Burns JC',
          'MÃ¼ller U'
        ],
        sample_metadata_fields: [
          'sex',
          'age',
          'specimen_part',
          'cell_line',
          'subject'
        ],
        platform_names: ['Illumina NovaSeq 6000'],
        platform_accession_codes: ['IlluminaNovaSeq6000'],
        organism_names: ['MUS_MUSCULUS'],
        downloadable_organism_names: [],
        pubmed_id: '30078710',
        num_total_samples: 6713,
        num_processed_samples: 0,
        num_downloadable_samples: 0,
        source_first_published: '2018-08-15T00:00:00+00:00'
      },
      {
        status: 'added',
        id: 52090,
        title:
          'Shared and distinct transcriptomic cell types across neocortical areas',
        publication_title:
          'Shared and distinct transcriptomic cell types across neocortical areas.',
        description:
          "Neocortex contains a multitude of cell types segregated into layers and functionally distinct regions. To investigate the diversity of cell types across the mouse neocortex, we analyzed 12,714 cells from the primary visual cortex (VISp), and 9,035 cells from the anterior lateral motor cortex (ALM) by deep single-cell RNA-sequencing (scRNA-seq), identifying 116 transcriptomic cell types. These two regions represent distant poles of the neocortex and perform distinct functions. We define 50 inhibitory transcriptomic cell types, all of which are shared across both cortical regions. In contrast, 49 of 52 excitatory transcriptomic types were found in either VISp or ALM, with only three present in both. By combining single cell RNA-seq and retrograde labeling, we demonstrate correspondence between excitatory transcriptomic types and their region-specific long-range target specificity. This study establishes a combined transcriptomic and projectional taxonomy of cortical cell types from functionally distinct regions of the mouse cortex. Overall design: scRNA-seq of 25,481 single cells from two regions of adult Mouse cortex: anterior lateral motor cortex (ALM) and primary visual cortex (VISp) and 2,986 control samples Please note that the current records represent 232-grouped sample records (i.e. 28706-samples collaped into 232-logical groups) and the complete metadata for each single sample is provided as 'complete_metadata_28706-cells.csv'. The file 'accession_table.csv' provides all of the GEO and SRA IDs along with the sample_name/title that matches the metadata.",
        technology: 'RNA-SEQ',
        accession_code: 'SRP150473',
        alternate_accession_code: 'GSE115746',
        submitter_institution: '',
        has_publication: true,
        publication_doi: '',
        publication_authors: [
          'Tasic B',
          'Yao Z',
          'Graybuck LT',
          'Smith KA',
          'Nguyen TN',
          'Bertagnolli D',
          'Goldy J',
          'Garren E',
          'Economo MN',
          'Viswanathan S',
          'Penn O',
          'Bakken T',
          'Menon V',
          'Miller J',
          'Fong O',
          'Hirokawa KE',
          'Lathia K',
          'Rimorin C',
          'Tieu M',
          'Larsen R',
          'Casper T',
          'Barkan E',
          'Kroll M',
          'Parry S',
          'Shapovalova NV',
          'Hirschstein D',
          'Pendergraft J',
          'Sullivan HA',
          'Kim TK',
          'Szafer A',
          'Dee N',
          'Groblewski P',
          'Wickersham I',
          'Cetin A',
          'Harris JA',
          'Levi BP',
          'Sunkin SM',
          'Madisen L',
          'Daigle TL',
          'Looger L',
          'Bernard A',
          'Phillips J',
          'Lein E',
          'Hawrylycz M',
          'Svoboda K',
          'Jones AR',
          'Koch C',
          'Zeng H'
        ],
        sample_metadata_fields: ['specimen_part', 'disease', 'subject'],
        platform_names: ['Illumina HiSeq 2500'],
        platform_accession_codes: ['IlluminaHiSeq2500'],
        organism_names: ['MUS_MUSCULUS'],
        downloadable_organism_names: [],
        pubmed_id: '30382198',
        num_total_samples: 28706,
        num_processed_samples: 0,
        num_downloadable_samples: 0,
        source_first_published: '2018-11-02T00:00:00+00:00'
      },
      {
        status: 'processing',
        id: 51049,
        title:
          'Transcriptome analysis of single cells from the developing mouse dentate gyrus',
        publication_title:
          'Conserved properties of dentate gyrus neurogenesis across postnatal development revealed by single-cell RNA sequencing.',
        description:
          'RNA sequencing was performed on 24185 single cells from dentate gyrus of CD-1, C57Bl/6, or hGFAP-GFP reporter mice, sampled at ages ranging from embryonal day 16.5 to postnatal day 132, with the aim of comparing peri- and postnatal neurogenesis in the dentate gyrus Overall design: 24185 single cells isolated from the dentate gyrus of mice',
        technology: 'RNA-SEQ',
        accession_code: 'SRP118985',
        alternate_accession_code: 'GSE104323',
        submitter_institution: '',
        has_publication: true,
        publication_doi: '',
        publication_authors: [
          'Hochgerner H',
          'Zeisel A',
          'LÃ¶nnerberg P',
          'Linnarsson S'
        ],
        sample_metadata_fields: ['cell_line', 'subject'],
        platform_names: ['Illumina HiSeq 4000'],
        platform_accession_codes: ['IlluminaHiSeq4000'],
        organism_names: ['MUS_MUSCULUS'],
        downloadable_organism_names: [],
        pubmed_id: '29335606',
        num_total_samples: 24185,
        num_processed_samples: 0,
        num_downloadable_samples: 0,
        source_first_published: '2017-12-16T00:00:00+00:00'
      },
      {
        status: 'add_remaining',
        id: 47744,
        title:
          'In vivo molecular signatures of severe dengue infection revealed by viscRNA-Seq',
        publication_title:
          'Broadly neutralizing human antibodies against dengue virus identified by single B cell transcriptomics.',
        description:
          'Dengue virus infection can result in severe symptoms including shock and hemorrhage, but an understanding of the molecular correlates of disease severity is lacking. Bulk transcriptomics on blood samples are difficult to interpret because the blood is composed of different cell types that may react differently to virus infection. Dengue virus RNA can be detected in human plasma, however identifying the cells carrying dengue virus through the bloodstream in vivo has proven challenging. Here we used our recently developed viscRNA-Seq approach to profile transcriptomes of thousands of single blood peripheral mononuclear cells from 6 human subjects with dengue fever and severe dengue, as well as to characterize the cell types associated with dengue virus in the human blood. We found that although no bulk transcriptome marker for severe dengue exists, the expression of MX2 in naive B cells, of CD163 in CD14+/CD16+ monocytes and of other genes in specific cell types is highly predictive for severe dengue. We detected virus-associated cells in the blood of two severe dengue patients with high viral load and discovered the majority of these to be B cells expressing germline IgM or IgD immunoglobulin chains and naive markers but also showing signs of activation and expression of CD69, CXCR4, and other surface receptors. In bystander B cells we detected signs of strong immune activation, parallel hypersomatic evolution and, in one severe degue subject, an anomalously large clone of highly mutated, IgG1 plasmablasts that could be reactive to dengue virus. This study presents a high-resolution molecular exploration into dengue virus infection in humans and can be generalized to any RNA virus. Overall design: Blood cells from dengue virus infected human patients were subjected to virus-inclusive single cell RNA-Seq.',
        technology: 'RNA-SEQ',
        accession_code: 'SRP152576',
        alternate_accession_code: 'GSE116672',
        submitter_institution: '',
        has_publication: true,
        publication_doi: '',
        publication_authors: [
          'Durham ND',
          'Agrawal A',
          'Waltari E',
          'Croote D',
          'Zanini F',
          'Fouch M',
          'Davidson E',
          'Smith O',
          'Carabajal E',
          'Pak JE',
          'Doranz BJ',
          'Robinson M',
          'Sanz AM',
          'Albornoz LL',
          'Rosso F',
          'Einav S',
          'Quake SR',
          'McCutcheon KM',
          'Goo L'
        ],
        sample_metadata_fields: ['sex', 'age', 'specimen_part', 'subject'],
        platform_names: ['Illumina NovaSeq 6000', 'NextSeq 500'],
        platform_accession_codes: ['IlluminaNovaSeq6000', 'NextSeq500'],
        organism_names: ['HOMO_SAPIENS'],
        downloadable_organism_names: [],
        pubmed_id: '31820734',
        num_total_samples: 13127,
        num_processed_samples: 0,
        num_downloadable_samples: 0,
        source_first_published: '2018-11-21T00:00:00+00:00'
      },
      {
        status: 'not_supported',
        id: 67685,
        title:
          'Drug-induced change in gene expression across NCI-60 cell lines after exposure to 15 anticancer agents for 2, 6 and 24h',
        publication_title:
          'The NCI Transcriptional Pharmacodynamics Workbench: A Tool to Examine Dynamic Expression Profiling of Therapeutic Response in the NCI-60 Cell Line Panel.',
        description:
          'To identify patterns of drug-induced gene modulation that occur across different cell types, we measured gene expression changes across NCI-60 cell lines after exposure to 15 anticancer agents. The results were integrated into a database and set of interactive analysis tools, the NCI Transcriptional Pharmacodynamics Workbench (NCI TPW), intended to allow exploration of gene expression modulation, including by molecular pathway, drug target, and association with drug sensitivity. We identified common transcriptional responses across drugs and cell types and uncovered cell signaling pathwayâspecific gene expression changes associated with drug sensitivity. We also demonstrated the value of this tool for investigating clinically relevant molecular hypotheses, utilizing the NCI TPW to assess drug-induced expression changes in genes associated with immune function and epithelial-mesenchymal transition, and to identify candidate biomarkers for drug activity. The NCI TPW provides a comprehensive resource to facilitate understanding of tumor cell characteristics that define sensitivity to anticancer drugs.',
        technology: 'MICROARRAY',
        accession_code: 'GSE116436',
        alternate_accession_code: 'E-GEOD-116436',
        submitter_institution: 'National Cancer Institute',
        has_publication: false,
        publication_doi: '',
        publication_authors: [
          'Monks A',
          'Zhao Y',
          'Hose C',
          'Hamed H',
          'Krushkal J',
          'Fang J',
          'Sonkin D',
          'Palmisano A',
          'Polley EC',
          'Fogli LK',
          'KonatÃ© MM',
          'Miller SB',
          'Simpson MA',
          'Voth AR',
          'Li MC',
          'Harris E',
          'Wu X',
          'Connelly JW',
          'Rapisarda A',
          'Teicher BA',
          'Simon R',
          'Doroshow JH'
        ],
        sample_metadata_fields: ['specimen_part'],
        platform_names: [
          '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          '[HT_HG-U133A] Affymetrix HT Human Genome U133A Array'
        ],
        platform_accession_codes: ['hgu133a2', 'hthgu133a'],
        organism_names: ['HOMO_SAPIENS'],
        downloadable_organism_names: ['HOMO_SAPIENS'],
        pubmed_id: '30355619',
        num_total_samples: 7623,
        num_processed_samples: 6633,
        num_downloadable_samples: 6633,
        source_first_published: '2018-06-29T00:00:00+00:00'
      },
      {
        status: 'request',
        id: 49187,
        title:
          'RNA-seq analysis of single cells of the oligodendrocyte lineage from nine distinct regions of the anterior-posterior and dorsal-ventral axis of the mouse juvenile central nervous system',
        publication_title:
          'Oligodendrocyte heterogeneity in the mouse juvenile and adult central nervous system.',
        description:
          '5069 transcriptomes of single oligodendrocyte cells from spinal cord, substantia nigra-ventral tegmental area, striatum, amygdala, hypothalamic nuclei, zona incerta, hippocampus, and somatosensory cortex of male and female mice between post-natal day 21 and 90. The study aimed at identifying diverse populations of oligodendrocytes, and revealing dynamics of oligodendrocyte maturation. Overall design: 5069 individual cells were sampled from CNS regions of mice of various strains as detailed in the protocols section',
        technology: 'RNA-SEQ',
        accession_code: 'SRP066613',
        alternate_accession_code: 'GSE75330',
        submitter_institution: '',
        has_publication: true,
        publication_doi: '',
        publication_authors: [
          'Marques S',
          'Zeisel A',
          'Codeluppi S',
          'van Bruggen D',
          'Mendanha FalcÃ£o A',
          'Xiao L',
          'Li H',
          'HÃ¤ring M',
          'Hochgerner H',
          'Romanov RA',
          'Gyllborg D',
          'MuÃ±oz Manchado A',
          'La Manno G',
          'LÃ¶nnerberg P',
          'Floriddia EM',
          'Rezayee F',
          'Ernfors P',
          'Arenas E',
          'Hjerling-Leffler J',
          'Harkany T',
          'Richardson WD',
          'Linnarsson S',
          'Castelo-Branco G'
        ],
        sample_metadata_fields: ['sex', 'cell_line', 'treatment', 'subject'],
        platform_names: ['Illumina HiSeq 2000'],
        platform_accession_codes: ['IlluminaHiSeq2000'],
        organism_names: ['MUS_MUSCULUS'],
        downloadable_organism_names: ['MUS_MUSCULUS'],
        pubmed_id: '27284195',
        num_total_samples: 6282,
        num_processed_samples: 6282,
        num_downloadable_samples: 6282,
        source_first_published: '2016-06-14T00:00:00+00:00'
      },
      {
        status: 'unavailable',
        id: 50699,
        title:
          'Single-cell transcriptomic analysis of mouse CA1 inhibitory neurons',
        publication_title:
          'Classes and continua of hippocampal CA1 inhibitory neurons revealed by single-cell transcriptomics.',
        description:
          'We  studied  the  transcriptomes of mouse CA1 inhibitory cells. Novel clustering methods identified all 23 previously  described  CA1  inhibitory  types,  while  also  suggesting  6  new  inhibitory  classes.  Latent-factor  analysis revealed a common continuum of expression of many genes within and between classes,  which  we  hypothesized  correlates  with  a  continuum  from  faster-spiking  cells  that  proximally  target pyramidal cells, to slower active cells targeting pyramidal distal dendrites or interneurons. Overall design: 6971 single cells isolated from the mouse cortex CA1 region',
        technology: 'RNA-SEQ',
        accession_code: 'SRP109000',
        alternate_accession_code: 'GSE99888',
        submitter_institution: '',
        has_publication: true,
        publication_doi: '',
        publication_authors: [
          'Harris KD',
          'Hochgerner H',
          'Skene NG',
          'Magno L',
          'Katona L',
          'Bengtsson Gonzales C',
          'Somogyi P',
          'Kessaris N',
          'Linnarsson S',
          'Hjerling-Leffler J'
        ],
        sample_metadata_fields: ['specimen_part', 'subject'],
        platform_names: ['Illumina HiSeq 2500'],
        platform_accession_codes: ['IlluminaHiSeq2500'],
        organism_names: ['MUS_MUSCULUS'],
        downloadable_organism_names: [],
        pubmed_id: '29912866',
        num_total_samples: 6971,
        num_processed_samples: 0,
        num_downloadable_samples: 0,
        source_first_published: '2018-05-12T00:00:00+00:00'
      },
      {
        status: 'qn_skipped',
        id: 51542,
        title:
          'Single cell RNA sequencing of interneurons of the mouse dorsolateral striatum II',
        publication_title:
          'Diversity of Interneurons in the Dorsal Striatum Revealed by Single-Cell RNA Sequencing and PatchSeq.',
        description:
          'In order to investigate age-dependent mRNA expression in mouse striatal interneurons, we performed single cell RNA-seq on FACS-isolated fluorescently labeled cells from the dorsal striatum of two mouse lines, 5ht3aEGFP or Lhx6cre::R26R-tdTomat,  from either P21-26 or P55-76 animals. We included the ventricular side of the striatum that contains adult born neuroblasts destined for the olfactory bulb. Overall design: 3417 single cells isolated from the mouse dorsal striatum.',
        technology: 'RNA-SEQ',
        accession_code: 'SRP124669',
        alternate_accession_code: 'GSE106707',
        submitter_institution: '',
        has_publication: true,
        publication_doi: '',
        publication_authors: [
          'MuÃ±oz-Manchado AB',
          'Bengtsson Gonzales C',
          'Zeisel A',
          'Munguba H',
          'Bekkouche B',
          'Skene NG',
          'LÃ¶nnerberg P',
          'Ryge J',
          'Harris KD',
          'Linnarsson S',
          'Hjerling-Leffler J'
        ],
        sample_metadata_fields: [
          'sex',
          'specimen_part',
          'cell_line',
          'subject'
        ],
        platform_names: ['Illumina HiSeq 4000'],
        platform_accession_codes: ['IlluminaHiSeq4000'],
        organism_names: ['MUS_MUSCULUS'],
        downloadable_organism_names: [],
        pubmed_id: '30134177',
        num_total_samples: 6834,
        num_processed_samples: 0,
        num_downloadable_samples: 0,
        source_first_published: '2018-08-23T00:00:00+00:00'
      },
      {
        status: '',
        id: 47154,
        title:
          'Epigenomic and genome structural diversity in a worldwide collection of Arabidopsis thaliana',
        publication_title:
          'Epigenomic Diversity in a Global Collection of Arabidopsis thaliana Accessions.',
        description:
          "Epigenetic variation can impact gene transcription and may play roles in phenotypic diversity and adaptation. Here we report 1,107 high quality single-base resolution methylomes, and 1,210 transcriptomes from the 1001 Arabidopsis Genomes population. Analyses reveal strong effects of geographic origin on average DNA methylation levels, alterations of gene expression by epialleles and a highly complex genetic basis for DNA methylation. Physical genome maps for nine of the most diverse accessions revealed how transposable elements and other structural variations shaped the epigenome to allow rapid adaptation to environmental changes, with strong emphasis on disease resistance. Analysis of the cistromes and epicistromes in these accessions revealed a significant association between both methylation and nucleotide variation and the conservation of transcription factor binding sites. The Arabidopsis thaliana 1001 Epigenomes Project now provides a comprehensive resource to help further understand how epigenetic variation contributes to both molecular and phenotypes in natural populations of the most widely studied reference plant. Overall design: RNA-seq profiling of 728 Arabidopsis thaliana accessions Please note that the sample ''Cod-0 (9836)'' is missing from the data columns in the normalized expression matrix  (''ath1001_tx_norm_2016-04-21-UQ_gNorm_normCounts_k4.tsv'') since the read count distribution from this sample was incompatible with the upper-quartile normalization method applied to all the samples, therefore excluded  from the expression matrix.",
        technology: 'RNA-SEQ',
        accession_code: 'SRP074107',
        alternate_accession_code: 'GSE80744',
        submitter_institution: '',
        has_publication: true,
        publication_doi: '',
        publication_authors: [
          'Kawakatsu T',
          'Huang SC',
          'Jupe F',
          'Sasaki E',
          'Schmitz RJ',
          'Urich MA',
          'Castanon R',
          'Nery JR',
          'Barragan C',
          'He Y',
          'Chen H',
          'Dubin M',
          'Lee CR',
          'Wang C',
          'Bemm F',
          'Becker C',
          "O'Neil R",
          "O'Malley RC",
          'Quarless DX',
          '1001 Genomes Consortium.',
          'Schork NJ',
          'Weigel D',
          'Nordborg M',
          'Ecker JR'
        ],
        sample_metadata_fields: ['specimen_part', 'subject'],
        platform_names: ['Illumina HiSeq 2500', 'Illumina HiSeq 4000'],
        platform_accession_codes: ['IlluminaHiSeq2500', 'IlluminaHiSeq4000'],
        organism_names: ['ARABIDOPSIS_THALIANA'],
        downloadable_organism_names: [],
        pubmed_id: '27419873',
        num_total_samples: 6583,
        num_processed_samples: 0,
        num_downloadable_samples: 0,
        source_first_published: '2016-07-16T00:00:00+00:00'
      },
      {
        status: '',
        id: 52550,
        title:
          'Single cell transcriptomic analysis (InDrops) of mouse mesenchymal stromal cells (mSCs or FAPs, fibro-adipogenic progenitors) isolated from lymph node, muscle and visceral adipose tissue',
        publication_title:
          'Distinct immunocyte-promoting and adipocyte-generating stromal components coordinate adipose tissue immune and metabolic tenors.',
        description:
          'Regulatory T cells (Tregs) are key brakes on the VAT inflammation that regulates local and systemic metabolic tenor. The cytokine, IL-33, expands and sustains the unique Treg population residing within VAT. Making use of  single-cell RNA sequencing, we identified the major IL-33 producers in VAT to be particular mSC subtypes, related to but distinct from adipocyte progenitor cells. We further characterize these subsets by individually isolating them and performing bulk-RNA sequencing. We explored modulation of the VAT-mSC (VmSC) landscape with physiologic variables such as age and sex, as well as pathogenic states like obesity. We uncovered a VAT Treg:stromal-cell negative regulatory loop that keeps the potent effect of IL-33 under rein. Overall design: Single cells were sorted by flow cytometry for single cell RNAseq library preparation (InDrops)',
        technology: 'RNA-SEQ',
        accession_code: 'SRP186786',
        alternate_accession_code: 'GSE127005',
        submitter_institution: '',
        has_publication: true,
        publication_doi: '',
        publication_authors: [
          'Spallanzani RG',
          'Zemmour D',
          'Xiao T',
          'Jayewickreme T',
          'Li C',
          'Bryce PJ',
          'Benoist C',
          'Mathis D'
        ],
        sample_metadata_fields: [
          'sex',
          'specimen_part',
          'cell_line',
          'subject'
        ],
        platform_names: ['NextSeq 500'],
        platform_accession_codes: ['NextSeq500'],
        organism_names: ['MUS_MUSCULUS'],
        downloadable_organism_names: [],
        pubmed_id: '31053654',
        num_total_samples: 5287,
        num_processed_samples: 0,
        num_downloadable_samples: 0,
        source_first_published: '2019-02-28T00:00:00+00:00'
      }
    ]
  }
}
