// https://api.refine.bio/v1/samples/?experiment_accession_code=accessionCode

export default [
  [
    // pageSize 10
    {
      count: 7623,
      next: 'http://api.refine.bio/v1/samples/?experiment_accession_code=GSE116436&limit=10&offset=10',
      previous: null,
      results: [
        {
          id: 877426,
          title: 'K-562_cisplatin_15000nM_6h',
          accession_code: 'GSM3232768',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232768',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'leukemia',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:03:00.458740Z',
          last_modified: '2018-12-20T08:40:00.806734Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 879523,
          title: 'K-562_cisplatin_3000nM_6h',
          accession_code: 'GSM3232771',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232771',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'leukemia',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:08:18.176002Z',
          last_modified: '2018-12-20T08:46:42.804963Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 875498,
          title: 'K-562_cisplatin_0nM_6h',
          accession_code: 'GSM3232765',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232765',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'leukemia',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:57:48.278191Z',
          last_modified: '2018-12-20T08:33:46.152858Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 877127,
          title: 'K-562_cisplatin_15000nM_2h',
          accession_code: 'GSM3232767',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232767',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'leukemia',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:02:09.306466Z',
          last_modified: '2018-12-20T08:38:37.681064Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 835128,
          title: 'K-562_cisplatin_3000nM_24h',
          accession_code: 'GSM3232769',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hthgu133a',
          platform_name: '[HT_HG-U133A] Affymetrix HT Human Genome U133A Array',
          pretty_platform: 'Affymetrix HT Human Genome U133A Array (hthgu133a)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232769',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'leukemia',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T19:32:30.470928Z',
          last_modified: '2019-08-08T18:31:55.356346Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 861088,
          title: 'K-562_cisplatin_3000nM_2h',
          accession_code: 'GSM3232770',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232770',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'leukemia',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:24:14.841506Z',
          last_modified: '2018-12-20T07:57:02.720451Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 879698,
          title: 'K-562_cisplatin_0nM_24h',
          accession_code: 'GSM3232763',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232763',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'leukemia',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:08:46.003952Z',
          last_modified: '2018-12-20T08:47:39.709996Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 821595,
          title: 'K-562_cisplatin_0nM_2h',
          accession_code: 'GSM3232764',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hthgu133a',
          platform_name: '[HT_HG-U133A] Affymetrix HT Human Genome U133A Array',
          pretty_platform: 'Affymetrix HT Human Genome U133A Array (hthgu133a)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232764',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'leukemia',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T18:45:13.262343Z',
          last_modified: '2019-08-08T13:01:50.371079Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 879712,
          title: 'K-562_cisplatin_15000nM_24h',
          accession_code: 'GSM3232766',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232766',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'leukemia',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:08:49.200200Z',
          last_modified: '2018-12-20T08:46:47.550205Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 873497,
          title: 'KM12_cisplatin_0nM_24h',
          accession_code: 'GSM3232772',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232772',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:52:04.385735Z',
          last_modified: '2018-12-20T08:06:37.435698Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        }
      ]
    },
    // pageSize 20
    {
      count: 7623,
      next: 'http://api.refine.bio/v1/samples/?experiment_accession_code=GSE116436&limit=20&offset=20',
      previous: null,
      results: [
        {
          id: 877976,
          title: 'CAKI-1_cisplatin_3000nM_24h',
          accession_code: 'GSM3232652',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232652',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:04:25.819066Z',
          last_modified: '2018-12-20T08:41:59.529069Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 874329,
          title: 'COLO-205_cisplatin_15000nM_2h',
          accession_code: 'GSM3232659',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232659',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:54:28.048855Z',
          last_modified: '2018-12-20T08:27:16.118299Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 875284,
          title: 'CAKI-1_cisplatin_0nM_2h',
          accession_code: 'GSM3232647',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232647',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:57:11.264073Z',
          last_modified: '2018-12-20T08:22:08.513633Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 868891,
          title: 'CAKI-1_cisplatin_15000nM_6h',
          accession_code: 'GSM3232651',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232651',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:39:53.349243Z',
          last_modified: '2018-12-20T07:58:40.961090Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 876593,
          title: 'COLO-205_cisplatin_0nM_24h',
          accession_code: 'GSM3232655',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232655',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:00:39.616645Z',
          last_modified: '2018-12-20T08:33:53.421552Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 873997,
          title: 'COLO-205_cisplatin_15000nM_24h',
          accession_code: 'GSM3232658',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232658',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:53:31.711287Z',
          last_modified: '2018-12-20T08:07:00.199809Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 860730,
          title: 'BT-549_cisplatin_3000nM_24h',
          accession_code: 'GSM3232643',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232643',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'breast',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:23:46.424614Z',
          last_modified: '2018-12-20T07:55:32.876889Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 877682,
          title: 'CAKI-1_cisplatin_0nM_24h',
          accession_code: 'GSM3232646',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232646',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:03:40.625815Z',
          last_modified: '2018-12-20T08:41:14.668436Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 866145,
          title: 'CAKI-1_cisplatin_15000nM_24h',
          accession_code: 'GSM3232649',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232649',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:33:19.476915Z',
          last_modified: '2018-12-20T08:49:01.846419Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 878141,
          title: 'CAKI-1_cisplatin_15000nM_2h',
          accession_code: 'GSM3232650',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232650',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:04:51.555416Z',
          last_modified: '2018-12-20T08:43:30.615677Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 876806,
          title: 'CAKI-1_cisplatin_3000nM_2h',
          accession_code: 'GSM3232653',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232653',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:01:14.467258Z',
          last_modified: '2018-12-20T08:38:21.697126Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 879542,
          title: 'CAKI-1_cisplatin_3000nM_6h',
          accession_code: 'GSM3232654',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232654',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:08:20.887198Z',
          last_modified: '2018-12-20T08:47:49.113039Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 873696,
          title: 'COLO-205_cisplatin_0nM_2h',
          accession_code: 'GSM3232656',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232656',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:52:38.465879Z',
          last_modified: '2018-12-20T08:21:48.951887Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 876535,
          title: 'COLO-205_cisplatin_0nM_6h',
          accession_code: 'GSM3232657',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232657',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:00:30.664155Z',
          last_modified: '2018-12-20T08:35:07.560711Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 2177627,
          title: 'BT-549_bortezomib_0nM_2h',
          accession_code: 'GSM3232123',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hthgu133a',
          platform_name: '[HT_HG-U133A] Affymetrix HT Human Genome U133A Array',
          pretty_platform: 'Affymetrix HT Human Genome U133A Array (hthgu133a)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232123',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 µg each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturer’s suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 µg per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [
            {
              data: {
                type: ['RNA'],
                title: ['BT-549_bortezomib_0nM_2h'],
                status: ['Public on Oct 09 2018'],
                label_ch1: ['biotin'],
                series_id: ['GSE116436', 'GSE116438'],
                taxid_ch1: ['9606'],
                description: [
                  'gene expression data from BT-549 treated with 0nM bortezomib for 2h'
                ],
                geo_columns: ['ID_REF', 'VALUE'],
                platform_id: ['GPL571'],
                contact_city: ['Rockville'],
                contact_name: ['Anne,,Monks'],
                hyb_protocol: [
                  'Biotin labeled antisense cRNA (4 µg per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
                ],
                molecule_ch1: ['total RNA'],
                organism_ch1: ['Homo sapiens'],
                channel_count: ['1'],
                contact_state: ['MD'],
                geo_accession: ['GSM3232123'],
                scan_protocol: [
                  'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
                ],
                data_row_count: ['22277'],
                contact_address: ['9609 Medical Center Dr'],
                contact_country: ['USA'],
                data_processing: [
                  'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
                ],
                source_name_ch1: ['human tumor cell line BT-549'],
                submission_date: ['Jun 29 2018'],
                last_update_date: ['Oct 09 2018'],
                contact_institute: ['National Cancer Institute'],
                contact_department: ['Developmental Therapeutics Program'],
                contact_laboratory: ['Molecular Pharmacology Branch'],
                label_protocol_ch1: [
                  'Samples (96-well format, 1 µg each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturer’s suggested protocols'
                ],
                supplementary_file: [
                  'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3232nnn/GSM3232123/suppl/GSM3232123_BT-549_bortezomib_0nM_2h_U133A.CEL.gz'
                ],
                characteristics_ch1: [
                  'group: member of the NCI-60 cell line panel',
                  'tissue: Breast'
                ],
                growth_protocol_ch1: [
                  'All cell lines were cultured in RPMI-1640 media with 10% FBS and L-Glutamine and incubated at 37oC, 5%CO2.'
                ],
                extract_protocol_ch1: [
                  'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
                ],
                treatment_protocol_ch1: [
                  'All drugs were dissolved in DMSO, diluted to their appropriate concentrations in culture media and added to the cells pre-plated for 24h (DMSO final concentration of 0.25%) , and incubated for the indicated times'
                ],
                'contact_zip/postal_code': ['20850']
              },
              is_ccdl: false,
              created_at: '2021-12-20T17:40:05.940638Z',
              last_modified: '2021-12-20T17:40:05.940638Z'
            }
          ],
          results: [
            {
              id: 2804594,
              processor: {
                id: 629,
                name: 'Affymetrix SCAN',
                version: 'v1.42.64-hotfix',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.36.0',
                    affy: '1.66.0',
                    limma: '3.44.3',
                    oligo: '1.52.1',
                    affyio: '1.58.0',
                    'SCAN.UPC': '2.30.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.10',
                    AnnotationDbi: '1.50.3'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.8.2-0ubuntu2',
                    'python3-pip': '20.0.2-5ubuntu1.6'
                  },
                  python: {
                    rpy2: '3.4.5',
                    Django: '3.2.7',
                    'data-refinery-common': 'v1.42.64-hotfix'
                  },
                  os_distribution: 'Ubuntu 20.04.3 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'breast',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2021-12-20T17:40:05.935968Z',
          last_modified: '2021-12-20T18:41:44.056666Z',
          original_files: [2982214, 2983475],
          computed_files: [5856827],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436']
        },
        {
          id: 877115,
          title: 'BT-549_cisplatin_15000nM_6h',
          accession_code: 'GSM3232642',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232642',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'breast',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:02:07.661300Z',
          last_modified: '2018-12-20T08:51:39.521162Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 867384,
          title: 'BT-549_cisplatin_3000nM_2h',
          accession_code: 'GSM3232644',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232644',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'breast',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:36:16.008907Z',
          last_modified: '2018-12-20T09:45:04.101467Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 872064,
          title: 'BT-549_cisplatin_3000nM_6h',
          accession_code: 'GSM3232645',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232645',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'breast',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:47:58.165300Z',
          last_modified: '2018-12-20T08:05:54.156401Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 875797,
          title: 'CAKI-1_cisplatin_0nM_6h',
          accession_code: 'GSM3232648',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232648',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:58:35.988183Z',
          last_modified: '2018-12-20T08:35:28.923801Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 871053,
          title: 'COLO-205_cisplatin_15000nM_6h',
          accession_code: 'GSM3232660',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232660',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:44:51.652070Z',
          last_modified: '2018-12-20T08:02:25.928718Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        }
      ]
    },
    // pageSize 50
    {
      count: 7623,
      next: 'http://api.refine.bio/v1/samples/?experiment_accession_code=GSE116436&limit=50&offset=50',
      previous: null,
      results: [
        {
          id: 878181,
          title: 'BT-549_cisplatin_15000nM_24h',
          accession_code: 'GSM3232640',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232640',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'breast',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:04:57.323260Z',
          last_modified: '2018-12-20T08:39:55.194151Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 877796,
          title: 'M14_cisplatin_0nM_6h',
          accession_code: 'GSM3232792',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232792',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'melanoma',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:03:58.139565Z',
          last_modified: '2018-12-20T10:29:01.151382Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 858706,
          title: 'A549_cisplatin_15000nM_6h',
          accession_code: 'GSM3232624',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232624',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'lung',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:20:57.476230Z',
          last_modified: '2018-12-20T07:56:19.045027Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 867280,
          title: 'BT-549_cisplatin_0nM_6h',
          accession_code: 'GSM3232639',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232639',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'breast',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:36:01.831494Z',
          last_modified: '2018-12-20T07:59:09.664477Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 871416,
          title: 'HT29_cisplatin_3000nM_24h',
          accession_code: 'GSM3232751',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232751',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:45:54.024766Z',
          last_modified: '2018-12-20T08:04:28.531545Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 876211,
          title: 'M14_cisplatin_0nM_2h',
          accession_code: 'GSM3232791',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232791',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'melanoma',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:59:41.213509Z',
          last_modified: '2018-12-20T08:36:46.572745Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 879563,
          title: 'A498_cisplatin_3000nM_24h',
          accession_code: 'GSM3232616',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232616',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:08:24.179365Z',
          last_modified: '2018-12-20T08:46:02.979650Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 876017,
          title: 'A549_cisplatin_15000nM_2h',
          accession_code: 'GSM3232623',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232623',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'lung',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:59:11.709162Z',
          last_modified: '2018-12-20T08:34:58.192981Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 870385,
          title: 'ACHN_cisplatin_15000nM_24h',
          accession_code: 'GSM3232631',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232631',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:43:07.899647Z',
          last_modified: '2018-12-20T07:59:35.004398Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 878204,
          title: 'BT-549_cisplatin_0nM_2h',
          accession_code: 'GSM3232638',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232638',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'breast',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:05:00.107380Z',
          last_modified: '2018-12-20T08:42:42.031694Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 870768,
          title: 'HCT-15_cisplatin_0nM_24h',
          accession_code: 'GSM3232700',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232700',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:44:04.431722Z',
          last_modified: '2018-12-20T08:04:39.436666Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 879926,
          title: 'HCT-15_cisplatin_15000nM_24h',
          accession_code: 'GSM3232703',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232703',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:09:22.503623Z',
          last_modified: '2018-12-20T08:48:57.364937Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 872599,
          title: 'LOX_cisplatin_3000nM_24h',
          accession_code: 'GSM3232787',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232787',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'melanoma',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:49:40.093655Z',
          last_modified: '2018-12-20T08:02:42.606418Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 875259,
          title: 'M14_cisplatin_0nM_24h',
          accession_code: 'GSM3232790',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232790',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'melanoma',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:57:07.266209Z',
          last_modified: '2018-12-20T08:37:15.529071Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 879632,
          title: 'A498_cisplatin_0nM_6h',
          accession_code: 'GSM3232612',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232612',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:08:35.411560Z',
          last_modified: '2018-12-20T12:27:47.128636Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 880019,
          title: 'A498_cisplatin_15000nM_6h',
          accession_code: 'GSM3232615',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232615',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:09:37.107015Z',
          last_modified: '2018-12-20T08:46:24.956021Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 877602,
          title: 'A549_cisplatin_0nM_24h',
          accession_code: 'GSM3232619',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232619',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'lung',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:03:28.204544Z',
          last_modified: '2018-12-20T08:38:32.349303Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 879417,
          title: 'A549_cisplatin_15000nM_24h',
          accession_code: 'GSM3232622',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232622',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'lung',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:08:01.232677Z',
          last_modified: '2018-12-20T11:33:45.509742Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 864538,
          title: 'A549_cisplatin_3000nM_6h',
          accession_code: 'GSM3232627',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232627',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'lung',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:29:52.191643Z',
          last_modified: '2018-12-20T07:55:18.900713Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 879913,
          title: 'ACHN_cisplatin_0nM_6h',
          accession_code: 'GSM3232630',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232630',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:09:20.649087Z',
          last_modified: '2018-12-20T08:49:13.520425Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 875565,
          title: 'ACHN_cisplatin_3000nM_24h',
          accession_code: 'GSM3232634',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232634',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:58:00.440841Z',
          last_modified: '2018-12-20T08:34:23.402024Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 879035,
          title: 'BT-549_cisplatin_0nM_24h',
          accession_code: 'GSM3232637',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232637',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'breast',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:07:03.520525Z',
          last_modified: '2018-12-20T08:44:43.952958Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 876293,
          title: 'HCT-116_cisplatin_3000nM_24h',
          accession_code: 'GSM3232697',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232697',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:59:53.409588Z',
          last_modified: '2018-12-20T08:37:40.455978Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 872927,
          title: 'HCT-116_cisplatin_3000nM_6h',
          accession_code: 'GSM3232699',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232699',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:50:34.169784Z',
          last_modified: '2018-12-20T08:22:06.319822Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 876384,
          title: 'HCT-15_cisplatin_0nM_2h',
          accession_code: 'GSM3232701',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232701',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:00:06.794079Z',
          last_modified: '2018-12-20T08:37:39.715889Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 872194,
          title: 'HCT-15_cisplatin_0nM_6h',
          accession_code: 'GSM3232702',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232702',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:48:23.346129Z',
          last_modified: '2018-12-20T08:06:26.284380Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 824933,
          title: 'LOX_cisplatin_15000nM_2h',
          accession_code: 'GSM3232785',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hthgu133a',
          platform_name: '[HT_HG-U133A] Affymetrix HT Human Genome U133A Array',
          pretty_platform: 'Affymetrix HT Human Genome U133A Array (hthgu133a)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232785',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'melanoma',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T18:53:26.070595Z',
          last_modified: '2019-08-08T14:03:03.402047Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 876652,
          title: 'LOX_cisplatin_15000nM_6h',
          accession_code: 'GSM3232786',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232786',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'melanoma',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:00:48.740894Z',
          last_modified: '2018-12-20T08:39:38.789064Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 814875,
          title: 'LOX_cisplatin_3000nM_2h',
          accession_code: 'GSM3232788',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hthgu133a',
          platform_name: '[HT_HG-U133A] Affymetrix HT Human Genome U133A Array',
          pretty_platform: 'Affymetrix HT Human Genome U133A Array (hthgu133a)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232788',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'melanoma',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T18:28:37.815050Z',
          last_modified: '2019-08-08T10:37:37.160620Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 875452,
          title: 'LOX_cisplatin_3000nM_6h',
          accession_code: 'GSM3232789',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232789',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'melanoma',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:57:40.227014Z',
          last_modified: '2018-12-20T08:33:39.107156Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 876747,
          title: 'A498_cisplatin_0nM_24h',
          accession_code: 'GSM3232610',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232610',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:01:03.251712Z',
          last_modified: '2018-12-20T08:37:30.838313Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 877109,
          title: 'A498_cisplatin_0nM_2h',
          accession_code: 'GSM3232611',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232611',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:02:05.413565Z',
          last_modified: '2018-12-20T08:38:01.574515Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 878295,
          title: 'A498_cisplatin_15000nM_24h',
          accession_code: 'GSM3232613',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232613',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:05:13.427733Z',
          last_modified: '2018-12-20T08:42:11.697004Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 876611,
          title: 'A498_cisplatin_15000nM_2h',
          accession_code: 'GSM3232614',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232614',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:00:43.129445Z',
          last_modified: '2018-12-20T10:38:56.309577Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 872024,
          title: 'A498_cisplatin_3000nM_2h',
          accession_code: 'GSM3232617',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232617',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:47:50.192246Z',
          last_modified: '2018-12-20T11:30:43.666699Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 875822,
          title: 'A498_cisplatin_3000nM_6h',
          accession_code: 'GSM3232618',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232618',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:58:40.064872Z',
          last_modified: '2018-12-20T08:33:20.695834Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 879842,
          title: 'A549_cisplatin_0nM_2h',
          accession_code: 'GSM3232620',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232620',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'lung',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:09:08.810870Z',
          last_modified: '2018-12-20T08:45:26.476203Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 877997,
          title: 'A549_cisplatin_0nM_6h',
          accession_code: 'GSM3232621',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232621',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'lung',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:04:28.767957Z',
          last_modified: '2018-12-20T08:43:11.875105Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 878222,
          title: 'A549_cisplatin_3000nM_24h',
          accession_code: 'GSM3232625',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232625',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'lung',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:05:03.485071Z',
          last_modified: '2018-12-20T08:40:51.043032Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 877621,
          title: 'A549_cisplatin_3000nM_2h',
          accession_code: 'GSM3232626',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232626',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'lung',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:03:32.443292Z',
          last_modified: '2018-12-20T08:42:05.237312Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 872520,
          title: 'ACHN_cisplatin_0nM_24h',
          accession_code: 'GSM3232628',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232628',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:49:25.190363Z',
          last_modified: '2018-12-20T08:07:01.513655Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 875746,
          title: 'ACHN_cisplatin_0nM_2h',
          accession_code: 'GSM3232629',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232629',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:58:29.123818Z',
          last_modified: '2018-12-20T08:34:23.185953Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 857297,
          title: 'ACHN_cisplatin_15000nM_2h',
          accession_code: 'GSM3232632',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232632',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:19:21.752372Z',
          last_modified: '2018-12-20T08:00:34.389588Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 875644,
          title: 'ACHN_cisplatin_15000nM_6h',
          accession_code: 'GSM3232633',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232633',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:58:13.437065Z',
          last_modified: '2018-12-20T08:23:21.840706Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 863515,
          title: 'ACHN_cisplatin_3000nM_2h',
          accession_code: 'GSM3232635',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232635',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:27:59.667040Z',
          last_modified: '2018-12-20T07:58:54.571750Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 879898,
          title: 'ACHN_cisplatin_3000nM_6h',
          accession_code: 'GSM3232636',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232636',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'renal',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:09:18.294789Z',
          last_modified: '2018-12-20T08:47:03.364952Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 877503,
          title: 'BT-549_cisplatin_15000nM_2h',
          accession_code: 'GSM3232641',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232641',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'breast',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:03:12.902547Z',
          last_modified: '2018-12-20T08:41:12.872788Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 874666,
          title: 'HCT-116_cisplatin_15000nM_6h',
          accession_code: 'GSM3232696',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232696',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:55:23.580426Z',
          last_modified: '2018-12-20T08:24:08.201045Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 878433,
          title: 'HCT-116_cisplatin_3000nM_2h',
          accession_code: 'GSM3232698',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232698',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'colon',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-20T00:05:33.848403Z',
          last_modified: '2018-12-20T08:44:04.381269Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        },
        {
          id: 876269,
          title: 'M14_cisplatin_15000nM_24h',
          accession_code: 'GSM3232793',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hgu133a2',
          platform_name: '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
          pretty_platform: 'Affymetrix Human Genome U133A 2.0 Array (hgu133a2)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3232793',
            'Scan protocol': [
              'Plates were washed, stained, then scanned on an Affymetrix GeneTitan HT array plate scanner'
            ],
            'Label protocol': [
              'Samples (96-well format, 1 g each) were labeled using Affymetrix (Cleveland OH) HT One-Cycle Eukaryotic Labeling Kit on Biomek FX Target Prep Express system following manufacturers suggested protocols'
            ],
            'Data processing': [
              'Data were collected using Affymetrix AGCC software.  Initial data quality control was performed using percent present calls generated by Affymetrix Quality Reporter software. Samples failing QC were repeated (R2)'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using the Qiagen Rneasy mini kit, quantitated by measuring absorbance at A260, quality of RNA was determined on a Caliper LabChip GX all samples used had an RQS>7.5'
            ],
            'Hybridization protocol': [
              'Biotin labeled antisense cRNA (4 g per sample) was hybridized to Affymetrix GeneChip HT Human Genome U133A 96-Array plates overnight'
            ]
          },
          annotations: [],
          results: [],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'melanoma',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T23:59:48.901640Z',
          last_modified: '2018-12-20T08:41:13.497750Z',
          original_files: [],
          computed_files: [],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE116436', 'GSE116439']
        }
      ]
    }
  ],
  [
    {
      count: 6282,
      next: 'http://api.refine.bio/v1/samples/?experiment_accession_code=SRP066613&limit=10&offset=10',
      previous: null,
      results: [
        {
          id: 1488156,
          title: 'C1-1771017-030-E08',
          accession_code: 'SRR3548207',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1867801,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1867805,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2596549,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2603132,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2608879,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2614312,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2620546,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624035,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624407,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627047,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627703,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2628863,
              processor: {
                id: 447,
                name: 'Salmon Quant',
                version: 'v1.27.13-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.13-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:48.740476Z',
          last_modified: '2019-09-04T22:46:39.458553Z',
          original_files: [2148883],
          computed_files: [3287587],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488161,
          title: 'C1-1771017-030-E12',
          accession_code: 'SRR3548210',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1894303,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1894306,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2583647,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2593455,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613446,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2621425,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622322,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2647827,
              processor: {
                id: 459,
                name: 'Salmon Quant',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:54.161297Z',
          last_modified: '2019-09-04T22:46:17.768113Z',
          original_files: [2148888],
          computed_files: [3335074],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488165,
          title: 'C1-1771017-030-F01',
          accession_code: 'SRR3548212',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1894238,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1894241,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576086,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2609063,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617609,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622459,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2710842,
              processor: {
                id: 487,
                name: 'Salmon Quant',
                version: 'v1.27.32-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.32-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2711117,
              processor: {
                id: 487,
                name: 'Salmon Quant',
                version: 'v1.27.32-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.32-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:57.804793Z',
          last_modified: '2019-09-04T22:46:37.794329Z',
          original_files: [2148892],
          computed_files: [3334977],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488172,
          title: 'C1-1771017-030-F09',
          accession_code: 'SRR3548215',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1894283,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1894285,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576259,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577160,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2589485,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2607448,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2611989,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:03.165126Z',
          last_modified: '2019-09-04T22:46:24.148917Z',
          original_files: [2148899],
          computed_files: [3335043],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488185,
          title: 'C1-1771017-030-G04',
          accession_code: 'SRR3548221',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1893703,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1893711,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576858,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2589620,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2591714,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2595379,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2596708,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2598716,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2609839,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617704,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622040,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627475,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2628193,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:13.833051Z',
          last_modified: '2019-09-04T22:46:17.669839Z',
          original_files: [2148912],
          computed_files: [3333116],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488190,
          title: 'C1-1771017-031-C05',
          accession_code: 'SRR3548223',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1895536,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1895538,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576508,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2600744,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2604146,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:17.392328Z',
          last_modified: '2019-09-04T22:47:27.140231Z',
          original_files: [2148917],
          computed_files: [3338802],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {
            MetaSRA: ['cortex', 'female organism', 'male organism']
          },
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488194,
          title: 'C1-1771017-031-C07',
          accession_code: 'SRR3548225',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1893461,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1893464,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2596222,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2597137,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622996,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2662019,
              processor: {
                id: 466,
                name: 'Salmon Quant',
                version: 'v1.27.17-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.17-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:21.089594Z',
          last_modified: '2019-09-04T22:46:33.358041Z',
          original_files: [2148921],
          computed_files: [3332745],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488198,
          title: 'C1-1771017-031-D10',
          accession_code: 'SRR3548227',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1888916,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1888921,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2575866,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2576243,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577398,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:24.657528Z',
          last_modified: '2019-09-04T22:47:23.495756Z',
          original_files: [2148925],
          computed_files: [3324078],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488206,
          title: 'C1-1771017-031-E05',
          accession_code: 'SRR3548230',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1844882,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1844890,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576510,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2580944,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2586619,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2593952,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2602821,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2608469,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2618080,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622513,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624531,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624657,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2652626,
              processor: {
                id: 459,
                name: 'Salmon Quant',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2662686,
              processor: {
                id: 466,
                name: 'Salmon Quant',
                version: 'v1.27.17-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.17-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:30.335035Z',
          last_modified: '2019-09-04T22:46:46.842628Z',
          original_files: [2148933],
          computed_files: [3250411],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488148,
          title: 'C1-1771017-030-E01',
          accession_code: 'SRR3548204',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1853211,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1853213,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2577168,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2580511,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2581973,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2594228,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2595393,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2603502,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2606738,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2619611,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2623694,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:43.334043Z',
          last_modified: '2019-09-04T22:47:26.627806Z',
          original_files: [2148875],
          computed_files: [3264348],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        }
      ]
    },
    {
      count: 6282,
      next: 'http://api.refine.bio/v1/samples/?experiment_accession_code=SRP066613&limit=20&offset=20',
      previous: null,
      results: [
        {
          id: 1488156,
          title: 'C1-1771017-030-E08',
          accession_code: 'SRR3548207',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1867801,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1867805,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2596549,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2603132,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2608879,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2614312,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2620546,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624035,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624407,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627047,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627703,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2628863,
              processor: {
                id: 447,
                name: 'Salmon Quant',
                version: 'v1.27.13-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.13-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:48.740476Z',
          last_modified: '2019-09-04T22:46:39.458553Z',
          original_files: [2148883],
          computed_files: [3287587],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488161,
          title: 'C1-1771017-030-E12',
          accession_code: 'SRR3548210',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1894303,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1894306,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2583647,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2593455,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613446,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2621425,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622322,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2647827,
              processor: {
                id: 459,
                name: 'Salmon Quant',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:54.161297Z',
          last_modified: '2019-09-04T22:46:17.768113Z',
          original_files: [2148888],
          computed_files: [3335074],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488165,
          title: 'C1-1771017-030-F01',
          accession_code: 'SRR3548212',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1894238,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1894241,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576086,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2609063,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617609,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622459,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2710842,
              processor: {
                id: 487,
                name: 'Salmon Quant',
                version: 'v1.27.32-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.32-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2711117,
              processor: {
                id: 487,
                name: 'Salmon Quant',
                version: 'v1.27.32-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.32-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:57.804793Z',
          last_modified: '2019-09-04T22:46:37.794329Z',
          original_files: [2148892],
          computed_files: [3334977],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488172,
          title: 'C1-1771017-030-F09',
          accession_code: 'SRR3548215',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1894283,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1894285,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576259,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577160,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2589485,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2607448,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2611989,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:03.165126Z',
          last_modified: '2019-09-04T22:46:24.148917Z',
          original_files: [2148899],
          computed_files: [3335043],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488185,
          title: 'C1-1771017-030-G04',
          accession_code: 'SRR3548221',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1893703,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1893711,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576858,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2589620,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2591714,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2595379,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2596708,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2598716,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2609839,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617704,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622040,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627475,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2628193,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:13.833051Z',
          last_modified: '2019-09-04T22:46:17.669839Z',
          original_files: [2148912],
          computed_files: [3333116],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488190,
          title: 'C1-1771017-031-C05',
          accession_code: 'SRR3548223',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1895536,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1895538,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576508,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2600744,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2604146,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:17.392328Z',
          last_modified: '2019-09-04T22:47:27.140231Z',
          original_files: [2148917],
          computed_files: [3338802],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {
            MetaSRA: ['cortex', 'female organism', 'male organism']
          },
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488194,
          title: 'C1-1771017-031-C07',
          accession_code: 'SRR3548225',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1893461,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1893464,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2596222,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2597137,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622996,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2662019,
              processor: {
                id: 466,
                name: 'Salmon Quant',
                version: 'v1.27.17-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.17-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:21.089594Z',
          last_modified: '2019-09-04T22:46:33.358041Z',
          original_files: [2148921],
          computed_files: [3332745],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488198,
          title: 'C1-1771017-031-D10',
          accession_code: 'SRR3548227',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1888916,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1888921,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2575866,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2576243,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577398,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:24.657528Z',
          last_modified: '2019-09-04T22:47:23.495756Z',
          original_files: [2148925],
          computed_files: [3324078],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488206,
          title: 'C1-1771017-031-E05',
          accession_code: 'SRR3548230',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1844882,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1844890,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576510,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2580944,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2586619,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2593952,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2602821,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2608469,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2618080,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622513,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624531,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624657,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2652626,
              processor: {
                id: 459,
                name: 'Salmon Quant',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2662686,
              processor: {
                id: 466,
                name: 'Salmon Quant',
                version: 'v1.27.17-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.17-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:30.335035Z',
          last_modified: '2019-09-04T22:46:46.842628Z',
          original_files: [2148933],
          computed_files: [3250411],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488211,
          title: 'C1-1771017-031-E11',
          accession_code: 'SRR3548232',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1846358,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1846360,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2588882,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2600813,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2603191,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2616523,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2620897,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2638345,
              processor: {
                id: 447,
                name: 'Salmon Quant',
                version: 'v1.27.13-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.13-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2638412,
              processor: {
                id: 453,
                name: 'Salmon Quant',
                version: 'v1.27.14-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.14-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2641624,
              processor: {
                id: 459,
                name: 'Salmon Quant',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2656414,
              processor: {
                id: 459,
                name: 'Salmon Quant',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:33.890914Z',
          last_modified: '2019-09-04T22:46:39.448046Z',
          original_files: [2148938],
          computed_files: [3252699],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488214,
          title: 'C1-1771017-031-F05',
          accession_code: 'SRR3548233',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1844864,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1844875,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2575923,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577413,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2586447,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2587375,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2595258,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2603923,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2606622,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617211,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2626588,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:35.678800Z',
          last_modified: '2019-09-04T22:47:20.398846Z',
          original_files: [2148941],
          computed_files: [3250392],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488218,
          title: 'C1-1771017-031-F05',
          accession_code: 'SRR3548234',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1888927,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1888930,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2575902,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577094,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2595522,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2600341,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2704736,
              processor: {
                id: 487,
                name: 'Salmon Quant',
                version: 'v1.27.32-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.32-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2709937,
              processor: {
                id: 487,
                name: 'Salmon Quant',
                version: 'v1.27.32-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.32-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2711120,
              processor: {
                id: 487,
                name: 'Salmon Quant',
                version: 'v1.27.32-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.32-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:37.479610Z',
          last_modified: '2019-09-04T22:46:28.315427Z',
          original_files: [2148945],
          computed_files: [3324088],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488223,
          title: 'C1-1771017-031-F06',
          accession_code: 'SRR3548236',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1895626,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1895629,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576117,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2576870,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2583341,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2596483,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613129,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2616970,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:41.057112Z',
          last_modified: '2019-09-04T22:46:23.159451Z',
          original_files: [2148950],
          computed_files: [3338938],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488230,
          title: 'C1-1771017-031-G02',
          accession_code: 'SRR3548239',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1889152,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1889162,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2593167,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2594949,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2597829,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2610761,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2615322,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2626666,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627234,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:46.437218Z',
          last_modified: '2019-09-04T22:47:21.083033Z',
          original_files: [2148957],
          computed_files: [3325200],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488232,
          title: 'C1-1771017-031-G02',
          accession_code: 'SRR3548240',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1889191,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1889197,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576136,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2578663,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2593262,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2597772,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2609566,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2616444,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:48.218022Z',
          last_modified: '2019-09-04T22:46:55.527341Z',
          original_files: [2148959],
          computed_files: [3325269],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488251,
          title: 'C1-1771017-032-A09',
          accession_code: 'SRR3548247',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1836321,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1836324,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576113,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2586490,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2587401,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2599564,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2605732,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2611556,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2614184,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617339,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:01.673101Z',
          last_modified: '2019-09-04T22:46:22.593182Z',
          original_files: [2148978],
          computed_files: [3236823],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488261,
          title: 'C1-1771017-032-B02',
          accession_code: 'SRR3548250',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1836500,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1836505,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2584451,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2585597,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2605519,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2605997,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2609375,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613010,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2616845,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624242,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:07.039926Z',
          last_modified: '2019-09-04T22:47:17.760200Z',
          original_files: [2148988],
          computed_files: [3237096],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488266,
          title: 'C1-1771017-032-B08',
          accession_code: 'SRR3548251',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1836489,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1836492,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2581449,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2608114,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2612544,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2618488,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2625860,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:08.839694Z',
          last_modified: '2019-09-04T22:46:24.790666Z',
          original_files: [2148993],
          computed_files: [3237075],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488310,
          title: 'C1-1771017-032-D04',
          accession_code: 'SRR3548262',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1844067,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1844069,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2581736,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2588250,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2589906,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2592313,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2592952,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2603043,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2606060,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2612546,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2621460,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2621917,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:29.018796Z',
          last_modified: '2019-09-04T22:47:02.575289Z',
          original_files: [2149037],
          computed_files: [3249064],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488148,
          title: 'C1-1771017-030-E01',
          accession_code: 'SRR3548204',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1853211,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1853213,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2577168,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2580511,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2581973,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2594228,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2595393,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2603502,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2606738,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2619611,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2623694,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:43.334043Z',
          last_modified: '2019-09-04T22:47:26.627806Z',
          original_files: [2148875],
          computed_files: [3264348],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        }
      ]
    },
    {
      count: 6282,
      next: 'http://api.refine.bio/v1/samples/?experiment_accession_code=SRP066613&limit=50&offset=50',
      previous: null,
      results: [
        {
          id: 1488156,
          title: 'C1-1771017-030-E08',
          accession_code: 'SRR3548207',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1867801,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1867805,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2596549,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2603132,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2608879,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2614312,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2620546,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624035,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624407,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627047,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627703,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2628863,
              processor: {
                id: 447,
                name: 'Salmon Quant',
                version: 'v1.27.13-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.13-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:48.740476Z',
          last_modified: '2019-09-04T22:46:39.458553Z',
          original_files: [2148883],
          computed_files: [3287587],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488161,
          title: 'C1-1771017-030-E12',
          accession_code: 'SRR3548210',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1894303,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1894306,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2583647,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2593455,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613446,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2621425,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622322,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2647827,
              processor: {
                id: 459,
                name: 'Salmon Quant',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:54.161297Z',
          last_modified: '2019-09-04T22:46:17.768113Z',
          original_files: [2148888],
          computed_files: [3335074],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488165,
          title: 'C1-1771017-030-F01',
          accession_code: 'SRR3548212',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1894238,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1894241,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576086,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2609063,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617609,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622459,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2710842,
              processor: {
                id: 487,
                name: 'Salmon Quant',
                version: 'v1.27.32-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.32-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2711117,
              processor: {
                id: 487,
                name: 'Salmon Quant',
                version: 'v1.27.32-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.32-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:57.804793Z',
          last_modified: '2019-09-04T22:46:37.794329Z',
          original_files: [2148892],
          computed_files: [3334977],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488172,
          title: 'C1-1771017-030-F09',
          accession_code: 'SRR3548215',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1894283,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1894285,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576259,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577160,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2589485,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2607448,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2611989,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:03.165126Z',
          last_modified: '2019-09-04T22:46:24.148917Z',
          original_files: [2148899],
          computed_files: [3335043],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488185,
          title: 'C1-1771017-030-G04',
          accession_code: 'SRR3548221',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1893703,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1893711,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576858,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2589620,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2591714,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2595379,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2596708,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2598716,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2609839,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617704,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622040,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627475,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2628193,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:13.833051Z',
          last_modified: '2019-09-04T22:46:17.669839Z',
          original_files: [2148912],
          computed_files: [3333116],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488190,
          title: 'C1-1771017-031-C05',
          accession_code: 'SRR3548223',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1895536,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1895538,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576508,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2600744,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2604146,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:17.392328Z',
          last_modified: '2019-09-04T22:47:27.140231Z',
          original_files: [2148917],
          computed_files: [3338802],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {
            MetaSRA: ['cortex', 'female organism', 'male organism']
          },
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488194,
          title: 'C1-1771017-031-C07',
          accession_code: 'SRR3548225',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1893461,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1893464,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2596222,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2597137,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622996,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2662019,
              processor: {
                id: 466,
                name: 'Salmon Quant',
                version: 'v1.27.17-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.17-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:21.089594Z',
          last_modified: '2019-09-04T22:46:33.358041Z',
          original_files: [2148921],
          computed_files: [3332745],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488198,
          title: 'C1-1771017-031-D10',
          accession_code: 'SRR3548227',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1888916,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1888921,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2575866,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2576243,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577398,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:24.657528Z',
          last_modified: '2019-09-04T22:47:23.495756Z',
          original_files: [2148925],
          computed_files: [3324078],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488206,
          title: 'C1-1771017-031-E05',
          accession_code: 'SRR3548230',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1844882,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1844890,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576510,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2580944,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2586619,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2593952,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2602821,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2608469,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2618080,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622513,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624531,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624657,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2652626,
              processor: {
                id: 459,
                name: 'Salmon Quant',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2662686,
              processor: {
                id: 466,
                name: 'Salmon Quant',
                version: 'v1.27.17-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.17-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:30.335035Z',
          last_modified: '2019-09-04T22:46:46.842628Z',
          original_files: [2148933],
          computed_files: [3250411],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488211,
          title: 'C1-1771017-031-E11',
          accession_code: 'SRR3548232',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1846358,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1846360,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2588882,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2600813,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2603191,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2616523,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2620897,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2638345,
              processor: {
                id: 447,
                name: 'Salmon Quant',
                version: 'v1.27.13-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.13-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2638412,
              processor: {
                id: 453,
                name: 'Salmon Quant',
                version: 'v1.27.14-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.14-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2641624,
              processor: {
                id: 459,
                name: 'Salmon Quant',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2656414,
              processor: {
                id: 459,
                name: 'Salmon Quant',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:33.890914Z',
          last_modified: '2019-09-04T22:46:39.448046Z',
          original_files: [2148938],
          computed_files: [3252699],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488214,
          title: 'C1-1771017-031-F05',
          accession_code: 'SRR3548233',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1844864,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1844875,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2575923,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577413,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2586447,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2587375,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2595258,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2603923,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2606622,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617211,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2626588,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:35.678800Z',
          last_modified: '2019-09-04T22:47:20.398846Z',
          original_files: [2148941],
          computed_files: [3250392],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488218,
          title: 'C1-1771017-031-F05',
          accession_code: 'SRR3548234',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1888927,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1888930,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2575902,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577094,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2595522,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2600341,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2704736,
              processor: {
                id: 487,
                name: 'Salmon Quant',
                version: 'v1.27.32-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.32-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2709937,
              processor: {
                id: 487,
                name: 'Salmon Quant',
                version: 'v1.27.32-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.32-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2711120,
              processor: {
                id: 487,
                name: 'Salmon Quant',
                version: 'v1.27.32-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.32-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:37.479610Z',
          last_modified: '2019-09-04T22:46:28.315427Z',
          original_files: [2148945],
          computed_files: [3324088],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488223,
          title: 'C1-1771017-031-F06',
          accession_code: 'SRR3548236',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1895626,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1895629,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576117,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2576870,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2583341,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2596483,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613129,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2616970,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:41.057112Z',
          last_modified: '2019-09-04T22:46:23.159451Z',
          original_files: [2148950],
          computed_files: [3338938],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488230,
          title: 'C1-1771017-031-G02',
          accession_code: 'SRR3548239',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1889152,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1889162,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2593167,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2594949,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2597829,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2610761,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2615322,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2626666,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627234,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:46.437218Z',
          last_modified: '2019-09-04T22:47:21.083033Z',
          original_files: [2148957],
          computed_files: [3325200],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488232,
          title: 'C1-1771017-031-G02',
          accession_code: 'SRR3548240',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1889191,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1889197,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576136,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2578663,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2593262,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2597772,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2609566,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2616444,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:48.218022Z',
          last_modified: '2019-09-04T22:46:55.527341Z',
          original_files: [2148959],
          computed_files: [3325269],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488251,
          title: 'C1-1771017-032-A09',
          accession_code: 'SRR3548247',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1836321,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1836324,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576113,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2586490,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2587401,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2599564,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2605732,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2611556,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2614184,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617339,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:01.673101Z',
          last_modified: '2019-09-04T22:46:22.593182Z',
          original_files: [2148978],
          computed_files: [3236823],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488261,
          title: 'C1-1771017-032-B02',
          accession_code: 'SRR3548250',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1836500,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1836505,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2584451,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2585597,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2605519,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2605997,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2609375,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613010,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2616845,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624242,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:07.039926Z',
          last_modified: '2019-09-04T22:47:17.760200Z',
          original_files: [2148988],
          computed_files: [3237096],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488266,
          title: 'C1-1771017-032-B08',
          accession_code: 'SRR3548251',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1836489,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1836492,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2581449,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2608114,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2612544,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2618488,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2625860,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:08.839694Z',
          last_modified: '2019-09-04T22:46:24.790666Z',
          original_files: [2148993],
          computed_files: [3237075],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488310,
          title: 'C1-1771017-032-D04',
          accession_code: 'SRR3548262',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1844067,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1844069,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2581736,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2588250,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2589906,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2592313,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2592952,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2603043,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2606060,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2612546,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2621460,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2621917,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:29.018796Z',
          last_modified: '2019-09-04T22:47:02.575289Z',
          original_files: [2149037],
          computed_files: [3249064],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488317,
          title: 'C1-1771017-032-D05',
          accession_code: 'SRR3548264',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1843791,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1843792,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2584636,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2585608,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2587064,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2594089,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2597248,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2602498,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2610423,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613921,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2626640,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:32.614967Z',
          last_modified: '2019-09-04T22:46:17.564920Z',
          original_files: [2149044],
          computed_files: [3248603],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488320,
          title: 'C1-1771017-032-D07',
          accession_code: 'SRR3548265',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1843797,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1843798,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2582067,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2594191,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2595697,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2603870,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2616613,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2620094,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:34.900908Z',
          last_modified: '2019-09-04T22:46:23.586755Z',
          original_files: [2149047],
          computed_files: [3248611],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488323,
          title: 'C1-1771017-032-D07',
          accession_code: 'SRR3548266',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1843871,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1843873,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2586509,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2587426,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2602682,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2608650,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613843,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617341,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622769,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624614,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:36.716441Z',
          last_modified: '2019-09-04T22:47:15.884759Z',
          original_files: [2149050],
          computed_files: [3248723],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488326,
          title: 'C1-1771017-032-D11',
          accession_code: 'SRR3548267',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1843800,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1843802,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2599823,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2611490,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2614050,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622997,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:38.510566Z',
          last_modified: '2019-09-04T22:46:48.715612Z',
          original_files: [2149053],
          computed_files: [3248617],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488329,
          title: 'C1-1771017-032-D11',
          accession_code: 'SRR3548268',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1890930,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1890937,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2585847,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2600267,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2616026,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622104,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2626670,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2638338,
              processor: {
                id: 447,
                name: 'Salmon Quant',
                version: 'v1.27.13-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.13-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2638406,
              processor: {
                id: 453,
                name: 'Salmon Quant',
                version: 'v1.27.14-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.14-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:40.300420Z',
          last_modified: '2019-09-04T22:46:25.530169Z',
          original_files: [2149056],
          computed_files: [3328107],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488339,
          title: 'C1-1771017-032-E03',
          accession_code: 'SRR3548271',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1890902,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1890906,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2592075,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2593107,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2600697,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2608591,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613608,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2626001,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:45.699013Z',
          last_modified: '2019-09-04T22:47:01.169975Z',
          original_files: [2149066],
          computed_files: [3328062],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488342,
          title: 'C1-1771017-032-E03',
          accession_code: 'SRR3548272',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1889476,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1889485,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2592212,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2593189,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2600858,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2607187,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627346,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:47.475237Z',
          last_modified: '2019-09-04T22:47:03.992934Z',
          original_files: [2149069],
          computed_files: [3325715],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488350,
          title: 'C1-1771017-032-E07',
          accession_code: 'SRR3548274',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1891495,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1891504,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2592261,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2600272,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2606096,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2615328,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622460,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:51.026988Z',
          last_modified: '2019-09-04T22:46:39.998946Z',
          original_files: [2149077],
          computed_files: [3329064],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488370,
          title: 'C1-1771017-032-F02',
          accession_code: 'SRR3548278',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1847970,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1847975,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2592292,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2611491,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2618046,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622248,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:31:58.464949Z',
          last_modified: '2019-09-04T22:47:14.505202Z',
          original_files: [2149097],
          computed_files: [3255293],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488024,
          title: 'C1-1771017-028-H09',
          accession_code: 'SRR3548164',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1867667,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1867676,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2609328,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613974,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617553,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'hippocampus ca1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:28:31.685643Z',
          last_modified: '2019-09-04T22:47:01.688531Z',
          original_files: [2148751],
          computed_files: [3287379],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {
            MetaSRA: [
              'CA1 field of hippocampus',
              'female organism',
              'obsolete_dorsal telencephalon'
            ]
          },
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488032,
          title: 'C1-1771017-029-A09',
          accession_code: 'SRR3548166',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1864813,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1864820,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2581664,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2588284,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2612996,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624608,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2672719,
              processor: {
                id: 472,
                name: 'Salmon Quant',
                version: 'v1.27.18-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.18-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2675124,
              processor: {
                id: 472,
                name: 'Salmon Quant',
                version: 'v1.27.18-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.18-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2675279,
              processor: {
                id: 472,
                name: 'Salmon Quant',
                version: 'v1.27.18-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.18-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'hippocampus ca1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:28:35.556433Z',
          last_modified: '2019-09-04T22:46:42.174002Z',
          original_files: [2148759],
          computed_files: [3283058],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488036,
          title: 'C1-1771017-029-D05',
          accession_code: 'SRR3548167',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1867634,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1867637,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2593235,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2596812,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2600764,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2604484,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2607473,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2612440,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2612983,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2619701,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2623848,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2626555,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'hippocampus ca1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:28:37.320217Z',
          last_modified: '2019-09-04T22:46:54.101726Z',
          original_files: [2148763],
          computed_files: [3287321],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488082,
          title: 'C1-1771017-030-A07',
          accession_code: 'SRR3548182',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1852732,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1852739,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2612155,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2614718,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2618172,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:04.164632Z',
          last_modified: '2019-09-04T22:46:53.134445Z',
          original_files: [2148809],
          computed_files: [3263556],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488084,
          title: 'C1-1771017-030-A09',
          accession_code: 'SRR3548183',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1852763,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1852768,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2582003,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2594128,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2604181,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2610285,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617129,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2620867,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624226,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:05.945386Z',
          last_modified: '2019-09-04T22:47:23.572570Z',
          original_files: [2148811],
          computed_files: [3263598],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488097,
          title: 'C1-1771017-030-A12',
          accession_code: 'SRR3548188',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1852882,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1852899,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576358,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2582288,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2589475,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2596861,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2598823,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2605356,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2609003,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613824,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627933,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:14.823669Z',
          last_modified: '2019-09-04T22:46:49.896887Z',
          original_files: [2148824],
          computed_files: [3263794],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {
            MetaSRA: ['cortex', 'female organism', 'male organism']
          },
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488110,
          title: 'C1-1771017-030-B04',
          accession_code: 'SRR3548193',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1852711,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1852718,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576766,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2596217,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2597199,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2605319,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2610738,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613387,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613814,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2617625,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2619108,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2625514,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:23.731689Z',
          last_modified: '2019-09-04T22:47:25.779998Z',
          original_files: [2148837],
          computed_files: [3263523],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488124,
          title: 'C1-1771017-030-D02',
          accession_code: 'SRR3548197',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1853044,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1853046,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2577149,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2580491,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2581369,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2590830,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2606849,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2608064,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2611448,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2621846,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2626729,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627324,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:30.846459Z',
          last_modified: '2019-09-04T22:46:59.732420Z',
          original_files: [2148851],
          computed_files: [3264022],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488128,
          title: 'C1-1771017-030-D02',
          accession_code: 'SRR3548198',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1852865,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1852880,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576096,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2586504,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2604180,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2609501,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2624957,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2642625,
              processor: {
                id: 459,
                name: 'Salmon Quant',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2655042,
              processor: {
                id: 459,
                name: 'Salmon Quant',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:32.611683Z',
          last_modified: '2019-09-04T22:46:17.393851Z',
          original_files: [2148855],
          computed_files: [3263766],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488139,
          title: 'C1-1771017-030-D11',
          accession_code: 'SRR3548201',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1852592,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1852597,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576260,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577127,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2580124,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2580429,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2591802,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2592148,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2606829,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2607964,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2612412,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2612980,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2618638,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2621387,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622080,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627416,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627490,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2628182,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:37.995267Z',
          last_modified: '2019-09-04T22:46:58.913696Z',
          original_files: [2148866],
          computed_files: [3263342],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488142,
          title: 'C1-1771017-030-D11',
          accession_code: 'SRR3548202',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1852624,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1852627,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576424,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2606711,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2627436,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2628184,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:39.790449Z',
          last_modified: '2019-09-04T22:46:21.672406Z',
          original_files: [2148869],
          computed_files: [3263386],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488151,
          title: 'C1-1771017-030-E06',
          accession_code: 'SRR3548205',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1852681,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1852690,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576081,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2578306,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2590610,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2601371,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2604853,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2606331,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613889,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2621712,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2622218,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2710655,
              processor: {
                id: 487,
                name: 'Salmon Quant',
                version: 'v1.27.32-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.32-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:45.133954Z',
          last_modified: '2019-09-04T22:46:46.772890Z',
          original_files: [2148878],
          computed_files: [3263482],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488167,
          title: 'C1-1771017-030-F02',
          accession_code: 'SRR3548213',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1894323,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1894330,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2575864,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2576293,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577349,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:59.586155Z',
          last_modified: '2019-09-04T22:47:29.334965Z',
          original_files: [2148894],
          computed_files: [3335111],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488174,
          title: 'C1-1771017-030-F09',
          accession_code: 'SRR3548216',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1894279,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1894282,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576457,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577489,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2593625,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2602949,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2604784,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2610302,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613901,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2648566,
              processor: {
                id: 459,
                name: 'Salmon Quant',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:04.951918Z',
          last_modified: '2019-09-04T22:46:44.151028Z',
          original_files: [2148901],
          computed_files: [3335039],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488178,
          title: 'C1-1771017-030-F11',
          accession_code: 'SRR3548218',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1893536,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1893546,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576160,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2588492,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2604004,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613169,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2619090,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:08.484492Z',
          last_modified: '2019-09-04T22:46:46.389062Z',
          original_files: [2148905],
          computed_files: [3332868],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488180,
          title: 'C1-1771017-030-F12',
          accession_code: 'SRR3548219',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1896003,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1896007,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576090,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2578531,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2584800,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2608286,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2619477,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2623631,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2625592,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:10.271767Z',
          last_modified: '2019-09-04T22:47:26.981039Z',
          original_files: [2148907],
          computed_files: [3340574],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488182,
          title: 'C1-1771017-030-F12',
          accession_code: 'SRR3548220',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1893579,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1893586,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2575893,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2576410,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577477,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:12.051628Z',
          last_modified: '2019-09-04T22:47:03.084491Z',
          original_files: [2148909],
          computed_files: [3332929],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488188,
          title: 'C1-1771017-030-G04',
          accession_code: 'SRR3548222',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1893472,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1893483,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576262,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577128,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2580784,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2593317,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2600806,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2607546,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2611218,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2626470,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:15.615752Z',
          last_modified: '2019-09-04T22:46:22.798378Z',
          original_files: [2148915],
          computed_files: [3332776],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488235,
          title: 'C1-1771017-031-H03',
          accession_code: 'SRR3548241',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1889196,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1889207,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2575880,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2576308,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2577467,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:50.706401Z',
          last_modified: '2019-09-04T22:46:34.403419Z',
          original_files: [2148962],
          computed_files: [3325283],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488237,
          title: 'C1-1771017-031-H03',
          accession_code: 'SRR3548242',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1889284,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1889288,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2589233,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2609489,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2613128,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2623021,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2625084,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2628389,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:52.547400Z',
          last_modified: '2019-09-04T22:47:05.753661Z',
          original_files: [2148964],
          computed_files: [3325404],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488238,
          title: 'C1-1771017-032-A06',
          accession_code: 'SRR3548243',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1889201,
              processor: {
                id: 379,
                name: 'Salmon Quant',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1889206,
              processor: {
                id: 380,
                name: 'Salmontools',
                version: 'v1.25.8-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.8-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2576047,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2576540,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2606496,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2619837,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2623500,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:30:54.342029Z',
          last_modified: '2019-09-04T22:46:59.284196Z',
          original_files: [2148965],
          computed_files: [3325282],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        },
        {
          id: 1488148,
          title: 'C1-1771017-030-E01',
          accession_code: 'SRR3548204',
          source_database: 'SRA',
          organism: {
            name: 'MUS_MUSCULUS',
            taxonomy_id: 10090
          },
          platform_accession_code: 'IlluminaHiSeq2000',
          platform_name: 'Illumina HiSeq 2000',
          pretty_platform: 'Illumina HiSeq 2000 (IlluminaHiSeq2000)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/SRP066613',
              Description:
                "Selected CNS regions were dissected in 300µm thick vibratome sections and cut out with scalpel. Tissue was then dissociated into a single cell suspension (described in Zeisel & Munoz-Manchado, Science 2015 Mar 6;347(6226):1138). For the spinal cord, an extra gradient step was used after the strainer and final centrifugation step. The pellet was ressuspended in 2ml Neurobasal-A medium including supplements and transferred onto an Optiprep Density Gradient, which consisted of 3 phases (Bottom phase: 160µl Optiprep + 340µl Neurobasal-A; middle phase: 220µl Optiprep + 1780µl Neurobasal-A; top phase: 75µl Optiprep + 925µl Neurobasal-A). The gradient was then centrifuged at 300g for 10min at 8°C and cells were located between the bottom and the middle phase. Cell suspension in a concentration of 600-1000 cells/μL was used. C1 Suspension Reagent was added in a ratio of 4 μL to every 7 μL cell suspension. 11 μL of the cell suspension mix was loaded on a C1 Single-Cell AutoPrep IFC  microfluidic chip designed for 10- to 17-μm cells, and the chip was then processed on a Fluidigm C1 instrument using the 'mRNA Seq: Cell Load (1772x/1773x)' script (30 min at 4°C). Lysis mix, RT mix and PCR mix (described in Islam et al., Nat Methods. 2014 Feb;11(2):163-6) were added to the chip. The plate was placed in the Fluidigm C1 instrument and the 'mRNA Seq: RT + Amp (1772x/1773x)' script was executed, and included lysis, reverse transcription and 21 cycles of PCR. The amplified cDNA was harvested in a total of 13 μL C1 Harvesting Reagent. Amplified cDNA was simultaneously fragmented and barcoded by tagmentation using Tn5 DNA transposase to transfer adaptors to the target DNA as described (Ibid.). 100 µl Dynabeads MyOne Streptavidin C1 beads (Invitrogen) were washed in 2× BWT (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 M NaCl, 0.02% Tween-20) and resuspended in 2 ml 2× BWT. Twenty microliters of beads were added to each well and incubated at room temperature for 15 min. All fractions were pooled, the beads were immobilized and the supernatant removed. The beads were resuspended in 100 μL TNT (20 mM Tris, pH 7.5, 50 mM NaCl, 0.02% Tween), washed in 100 μL Qiagen Qiaquick PB, and twice in 100 μL TNT. The beads were resuspended in 100 μL restriction mix (1× NEB CutSmart, 0.4 U/μL PvuI-HF enzyme), designed to cleave 3′ fragments carrying a PvuI recognition site. The mix was incubated for 1 h at 37 °C, then washed three times in TNT. Finally, to elute DNA, beads were resuspend in 30 µL ddH2O and incubated 10 minutes at 70°C. Beads were then immediately bound to magnet and the supernatant was collected. To remove short fragments,  Ampure beads (Beckman Coulter) were used at 1.8× volume and eluted in 30 µL."
            }
          ],
          annotations: [],
          results: [
            {
              id: 1853211,
              processor: {
                id: 373,
                name: 'Salmon Quant',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 1853213,
              processor: {
                id: 374,
                name: 'Salmontools',
                version: 'v1.25.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.25.6-hotfix'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 2577168,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2580511,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2581973,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2594228,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2595393,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2603502,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2606738,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2619611,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            },
            {
              id: 2623694,
              processor: {
                id: 442,
                name: 'Salmon Quant',
                version: 'v1.27.6-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.6-hotfix'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.13.1',
                    'sra-stat --version': 'sra-stat : 2.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: {
                id: 529,
                assembly_name: 'GRCm38',
                organism_name: 'MUS_MUSCULUS',
                database_name: 'EnsemblMain',
                release_version: '96',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.13.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/MUS_MUSCULUS_TRANSCRIPTOME_SHORT_1559762392.tar.gz',
                result_id: 1242727,
                last_modified: '2019-06-05T19:20:34.596829Z'
              }
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'pooled male and female',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: 'pdgfra-gfp',
          treatment: 'no',
          race: '',
          subject: 'cortex s1',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-08-22T08:29:43.334043Z',
          last_modified: '2019-09-04T22:47:26.627806Z',
          original_files: [2148875],
          computed_files: [3264348],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['SRP066613']
        }
      ]
    }
  ],
  [
    {
      count: 2822,
      next: 'http://api.refine.bio/v1/samples/?experiment_accession_code=ERP006132&limit=10&offset=10',
      previous: null,
      results: [
        {
          id: 5681,
          title: 'Danio rerio',
          accession_code: 'ERR657520',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 10812,
              processor: {
                id: 10,
                name: 'Salmon Quant',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 10814,
              processor: {
                id: 11,
                name: 'MultiQC',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:40.827343Z',
          last_modified: '2019-02-13T05:20:09.309276Z',
          original_files: [6614],
          computed_files: [13487, 1076690, 1079515],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5809,
          title: 'Danio rerio',
          accession_code: 'ERR657522',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 47803,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 47809,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:44.520005Z',
          last_modified: '2019-02-13T05:20:09.296100Z',
          original_files: [6742],
          computed_files: [61103, 1076690, 1079513],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5595,
          title: 'Danio rerio',
          accession_code: 'ERR657519',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 33067,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 33092,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:38.879278Z',
          last_modified: '2019-02-13T05:20:09.315939Z',
          original_files: [6524],
          computed_files: [41919, 1076690, 1079516],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5868,
          title: 'Danio rerio',
          accession_code: 'ERR657523',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 32188,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 32220,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:46.339153Z',
          last_modified: '2019-02-13T05:20:09.288725Z',
          original_files: [6801],
          computed_files: [40741, 1076690, 1079512],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5511,
          title: 'Danio rerio',
          accession_code: 'ERR657518',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 3549,
              processor: {
                id: 3,
                name: 'Salmon Quant',
                version: 'v1.0.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.1'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 3602,
              processor: {
                id: 4,
                name: 'Salmontools',
                version: 'v1.0.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.1'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 3603,
              processor: {
                id: 5,
                name: 'MultiQC',
                version: 'v1.0.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.1'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:36.981417Z',
          last_modified: '2019-02-13T05:20:09.326800Z',
          original_files: [6435],
          computed_files: [4227, 4228, 1076690, 1079517],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5926,
          title: 'Danio rerio',
          accession_code: 'ERR657524',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 26349,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 184347,
              processor: {
                id: 34,
                name: 'Salmon Quant',
                version: 'v1.2.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.2.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 184360,
              processor: {
                id: 35,
                name: 'MultiQC',
                version: 'v1.2.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.2.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:48.112651Z',
          last_modified: '2019-02-13T05:20:09.281926Z',
          original_files: [6859],
          computed_files: [32711, 230751, 1076690, 1079511],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5985,
          title: 'Danio rerio',
          accession_code: 'ERR657525',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 34342,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 34353,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:50.087543Z',
          last_modified: '2019-02-13T05:20:09.275051Z',
          original_files: [6918],
          computed_files: [43612, 1076690, 1079510],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6246,
          title: 'Danio rerio',
          accession_code: 'ERR657530',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 28546,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 28570,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:59.417451Z',
          last_modified: '2019-02-13T05:20:09.240117Z',
          original_files: [7179],
          computed_files: [35756, 1076690, 1079505],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6511,
          title: 'Danio rerio',
          accession_code: 'ERR657533',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 35081,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 35106,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:34:05.000826Z',
          last_modified: '2019-02-13T05:20:09.215618Z',
          original_files: [7469],
          computed_files: [44563, 1076690, 1079502],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5743,
          title: 'Danio rerio',
          accession_code: 'ERR657521',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 330257,
              processor: {
                id: 45,
                name: 'Salmon Quant',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 330271,
              processor: {
                id: 46,
                name: 'MultiQC',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:42.651300Z',
          last_modified: '2019-02-13T05:20:09.302717Z',
          original_files: [6676],
          computed_files: [399915, 1076690, 1079514],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        }
      ]
    },
    {
      count: 2822,
      next: 'http://api.refine.bio/v1/samples/?experiment_accession_code=ERP006132&limit=20&offset=20',
      previous: null,
      results: [
        {
          id: 5681,
          title: 'Danio rerio',
          accession_code: 'ERR657520',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 10812,
              processor: {
                id: 10,
                name: 'Salmon Quant',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 10814,
              processor: {
                id: 11,
                name: 'MultiQC',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:40.827343Z',
          last_modified: '2019-02-13T05:20:09.309276Z',
          original_files: [6614],
          computed_files: [13487, 1076690, 1079515],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5809,
          title: 'Danio rerio',
          accession_code: 'ERR657522',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 47803,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 47809,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:44.520005Z',
          last_modified: '2019-02-13T05:20:09.296100Z',
          original_files: [6742],
          computed_files: [61103, 1076690, 1079513],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5595,
          title: 'Danio rerio',
          accession_code: 'ERR657519',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 33067,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 33092,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:38.879278Z',
          last_modified: '2019-02-13T05:20:09.315939Z',
          original_files: [6524],
          computed_files: [41919, 1076690, 1079516],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5868,
          title: 'Danio rerio',
          accession_code: 'ERR657523',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 32188,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 32220,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:46.339153Z',
          last_modified: '2019-02-13T05:20:09.288725Z',
          original_files: [6801],
          computed_files: [40741, 1076690, 1079512],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5511,
          title: 'Danio rerio',
          accession_code: 'ERR657518',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 3549,
              processor: {
                id: 3,
                name: 'Salmon Quant',
                version: 'v1.0.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.1'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 3602,
              processor: {
                id: 4,
                name: 'Salmontools',
                version: 'v1.0.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.1'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 3603,
              processor: {
                id: 5,
                name: 'MultiQC',
                version: 'v1.0.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.1'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:36.981417Z',
          last_modified: '2019-02-13T05:20:09.326800Z',
          original_files: [6435],
          computed_files: [4227, 4228, 1076690, 1079517],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5926,
          title: 'Danio rerio',
          accession_code: 'ERR657524',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 26349,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 184347,
              processor: {
                id: 34,
                name: 'Salmon Quant',
                version: 'v1.2.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.2.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 184360,
              processor: {
                id: 35,
                name: 'MultiQC',
                version: 'v1.2.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.2.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:48.112651Z',
          last_modified: '2019-02-13T05:20:09.281926Z',
          original_files: [6859],
          computed_files: [32711, 230751, 1076690, 1079511],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5985,
          title: 'Danio rerio',
          accession_code: 'ERR657525',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 34342,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 34353,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:50.087543Z',
          last_modified: '2019-02-13T05:20:09.275051Z',
          original_files: [6918],
          computed_files: [43612, 1076690, 1079510],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6246,
          title: 'Danio rerio',
          accession_code: 'ERR657530',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 28546,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 28570,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:59.417451Z',
          last_modified: '2019-02-13T05:20:09.240117Z',
          original_files: [7179],
          computed_files: [35756, 1076690, 1079505],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6511,
          title: 'Danio rerio',
          accession_code: 'ERR657533',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 35081,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 35106,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:34:05.000826Z',
          last_modified: '2019-02-13T05:20:09.215618Z',
          original_files: [7469],
          computed_files: [44563, 1076690, 1079502],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6613,
          title: 'Danio rerio',
          accession_code: 'ERR657535',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 61901,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 61909,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:34:08.601281Z',
          last_modified: '2019-02-13T05:20:09.201281Z',
          original_files: [7574],
          computed_files: [79176, 1076690, 1079500],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6301,
          title: 'Danio rerio',
          accession_code: 'ERR657531',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 26860,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 26897,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:34:01.262915Z',
          last_modified: '2019-02-13T05:20:09.233096Z',
          original_files: [7234],
          computed_files: [33465, 1076690, 1079504],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6040,
          title: 'Danio rerio',
          accession_code: 'ERR657526',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 61849,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 61853,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:52.147203Z',
          last_modified: '2019-02-13T05:20:09.268193Z',
          original_files: [6973],
          computed_files: [79100, 1076690, 1079509],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6093,
          title: 'Danio rerio',
          accession_code: 'ERR657527',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 24981,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 25011,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:53.928655Z',
          last_modified: '2019-02-13T05:20:09.261310Z',
          original_files: [7026],
          computed_files: [30954, 1076690, 1079508],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6146,
          title: 'Danio rerio',
          accession_code: 'ERR657528',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 42062,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 42299,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 294115,
              processor: {
                id: 45,
                name: 'Salmon Quant',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 294124,
              processor: {
                id: 46,
                name: 'MultiQC',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:55.736033Z',
          last_modified: '2019-02-13T05:20:09.254248Z',
          original_files: [7079],
          computed_files: [53676, 54005, 361199, 1076690, 1079507],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6198,
          title: 'Danio rerio',
          accession_code: 'ERR657529',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 61639,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 61644,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:57.581382Z',
          last_modified: '2019-02-13T05:20:09.247055Z',
          original_files: [7131],
          computed_files: [78834, 1076690, 1079506],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6427,
          title: 'Danio rerio',
          accession_code: 'ERR657532',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 61641,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 61645,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:34:03.114941Z',
          last_modified: '2019-02-13T05:20:09.225386Z',
          original_files: [7385],
          computed_files: [78836, 1076690, 1079503],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6563,
          title: 'Danio rerio',
          accession_code: 'ERR657534',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 33152,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 33175,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:34:06.743054Z',
          last_modified: '2019-02-13T05:20:09.208168Z',
          original_files: [7523],
          computed_files: [42039, 1076690, 1079501],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8679,
          title: 'Danio rerio',
          accession_code: 'ERR657536',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 39217,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 39228,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:17.778515Z',
          last_modified: '2019-02-13T05:20:09.194591Z',
          original_files: [9770],
          computed_files: [50008, 1076690, 1079499],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8700,
          title: 'Danio rerio',
          accession_code: 'ERR657537',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 94821,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 94830,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:19.528444Z',
          last_modified: '2019-02-13T05:20:09.187792Z',
          original_files: [9791],
          computed_files: [121888, 1076690, 1079498],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5743,
          title: 'Danio rerio',
          accession_code: 'ERR657521',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 330257,
              processor: {
                id: 45,
                name: 'Salmon Quant',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 330271,
              processor: {
                id: 46,
                name: 'MultiQC',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:42.651300Z',
          last_modified: '2019-02-13T05:20:09.302717Z',
          original_files: [6676],
          computed_files: [399915, 1076690, 1079514],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        }
      ]
    },
    {
      count: 2822,
      next: 'http://api.refine.bio/v1/samples/?experiment_accession_code=ERP006132&limit=50&offset=50',
      previous: null,
      results: [
        {
          id: 5681,
          title: 'Danio rerio',
          accession_code: 'ERR657520',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 10812,
              processor: {
                id: 10,
                name: 'Salmon Quant',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 10814,
              processor: {
                id: 11,
                name: 'MultiQC',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:40.827343Z',
          last_modified: '2019-02-13T05:20:09.309276Z',
          original_files: [6614],
          computed_files: [13487, 1076690, 1079515],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5809,
          title: 'Danio rerio',
          accession_code: 'ERR657522',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 47803,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 47809,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:44.520005Z',
          last_modified: '2019-02-13T05:20:09.296100Z',
          original_files: [6742],
          computed_files: [61103, 1076690, 1079513],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5595,
          title: 'Danio rerio',
          accession_code: 'ERR657519',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 33067,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 33092,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:38.879278Z',
          last_modified: '2019-02-13T05:20:09.315939Z',
          original_files: [6524],
          computed_files: [41919, 1076690, 1079516],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5868,
          title: 'Danio rerio',
          accession_code: 'ERR657523',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 32188,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 32220,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:46.339153Z',
          last_modified: '2019-02-13T05:20:09.288725Z',
          original_files: [6801],
          computed_files: [40741, 1076690, 1079512],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5511,
          title: 'Danio rerio',
          accession_code: 'ERR657518',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 3549,
              processor: {
                id: 3,
                name: 'Salmon Quant',
                version: 'v1.0.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.1'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 3602,
              processor: {
                id: 4,
                name: 'Salmontools',
                version: 'v1.0.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.1'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 3603,
              processor: {
                id: 5,
                name: 'MultiQC',
                version: 'v1.0.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.1'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:36.981417Z',
          last_modified: '2019-02-13T05:20:09.326800Z',
          original_files: [6435],
          computed_files: [4227, 4228, 1076690, 1079517],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5926,
          title: 'Danio rerio',
          accession_code: 'ERR657524',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 26349,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 184347,
              processor: {
                id: 34,
                name: 'Salmon Quant',
                version: 'v1.2.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.2.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 184360,
              processor: {
                id: 35,
                name: 'MultiQC',
                version: 'v1.2.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.2.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:48.112651Z',
          last_modified: '2019-02-13T05:20:09.281926Z',
          original_files: [6859],
          computed_files: [32711, 230751, 1076690, 1079511],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5985,
          title: 'Danio rerio',
          accession_code: 'ERR657525',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 34342,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 34353,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:50.087543Z',
          last_modified: '2019-02-13T05:20:09.275051Z',
          original_files: [6918],
          computed_files: [43612, 1076690, 1079510],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6246,
          title: 'Danio rerio',
          accession_code: 'ERR657530',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 28546,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 28570,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:59.417451Z',
          last_modified: '2019-02-13T05:20:09.240117Z',
          original_files: [7179],
          computed_files: [35756, 1076690, 1079505],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6511,
          title: 'Danio rerio',
          accession_code: 'ERR657533',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 35081,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 35106,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:34:05.000826Z',
          last_modified: '2019-02-13T05:20:09.215618Z',
          original_files: [7469],
          computed_files: [44563, 1076690, 1079502],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6613,
          title: 'Danio rerio',
          accession_code: 'ERR657535',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 61901,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 61909,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:34:08.601281Z',
          last_modified: '2019-02-13T05:20:09.201281Z',
          original_files: [7574],
          computed_files: [79176, 1076690, 1079500],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6301,
          title: 'Danio rerio',
          accession_code: 'ERR657531',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 26860,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 26897,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:34:01.262915Z',
          last_modified: '2019-02-13T05:20:09.233096Z',
          original_files: [7234],
          computed_files: [33465, 1076690, 1079504],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6040,
          title: 'Danio rerio',
          accession_code: 'ERR657526',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 61849,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 61853,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:52.147203Z',
          last_modified: '2019-02-13T05:20:09.268193Z',
          original_files: [6973],
          computed_files: [79100, 1076690, 1079509],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6093,
          title: 'Danio rerio',
          accession_code: 'ERR657527',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 24981,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 25011,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:53.928655Z',
          last_modified: '2019-02-13T05:20:09.261310Z',
          original_files: [7026],
          computed_files: [30954, 1076690, 1079508],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6146,
          title: 'Danio rerio',
          accession_code: 'ERR657528',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 42062,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 42299,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 294115,
              processor: {
                id: 45,
                name: 'Salmon Quant',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 294124,
              processor: {
                id: 46,
                name: 'MultiQC',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:55.736033Z',
          last_modified: '2019-02-13T05:20:09.254248Z',
          original_files: [7079],
          computed_files: [53676, 54005, 361199, 1076690, 1079507],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6198,
          title: 'Danio rerio',
          accession_code: 'ERR657529',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 61639,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 61644,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:57.581382Z',
          last_modified: '2019-02-13T05:20:09.247055Z',
          original_files: [7131],
          computed_files: [78834, 1076690, 1079506],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6427,
          title: 'Danio rerio',
          accession_code: 'ERR657532',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 61641,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 61645,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:34:03.114941Z',
          last_modified: '2019-02-13T05:20:09.225386Z',
          original_files: [7385],
          computed_files: [78836, 1076690, 1079503],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 6563,
          title: 'Danio rerio',
          accession_code: 'ERR657534',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 33152,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 33175,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:34:06.743054Z',
          last_modified: '2019-02-13T05:20:09.208168Z',
          original_files: [7523],
          computed_files: [42039, 1076690, 1079501],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8679,
          title: 'Danio rerio',
          accession_code: 'ERR657536',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 39217,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 39228,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:17.778515Z',
          last_modified: '2019-02-13T05:20:09.194591Z',
          original_files: [9770],
          computed_files: [50008, 1076690, 1079499],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8700,
          title: 'Danio rerio',
          accession_code: 'ERR657537',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 94821,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 94830,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:19.528444Z',
          last_modified: '2019-02-13T05:20:09.187792Z',
          original_files: [9791],
          computed_files: [121888, 1076690, 1079498],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8722,
          title: 'Danio rerio',
          accession_code: 'ERR657538',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 94907,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 280254,
              processor: {
                id: 45,
                name: 'Salmon Quant',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 280271,
              processor: {
                id: 46,
                name: 'MultiQC',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:21.276357Z',
          last_modified: '2019-02-13T05:20:09.181004Z',
          original_files: [9813],
          computed_files: [121981, 345212, 1076690, 1079497],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8743,
          title: 'Danio rerio',
          accession_code: 'ERR657539',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 92419,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 92460,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:22.996742Z',
          last_modified: '2019-02-13T05:20:09.174220Z',
          original_files: [9834],
          computed_files: [118542, 1076690, 1079496],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8765,
          title: 'Danio rerio',
          accession_code: 'ERR657540',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 95021,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 95035,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:24.733976Z',
          last_modified: '2019-02-13T05:20:09.167209Z',
          original_files: [9856],
          computed_files: [122131, 1076690, 1079495],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8806,
          title: 'Danio rerio',
          accession_code: 'ERR657542',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 93496,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 93535,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:28.212767Z',
          last_modified: '2019-02-13T05:20:09.150852Z',
          original_files: [9898],
          computed_files: [120038, 1076690, 1079493],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9157,
          title: 'Danio rerio',
          accession_code: 'ERR659764',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 7827,
              processor: {
                id: 10,
                name: 'Salmon Quant',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 7855,
              processor: {
                id: 11,
                name: 'MultiQC',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:58.137809Z',
          last_modified: '2019-02-13T05:20:09.035564Z',
          original_files: [10258],
          computed_files: [9492, 1076690, 1079476],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9175,
          title: 'Danio rerio',
          accession_code: 'ERR659765',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 181025,
              processor: {
                id: 35,
                name: 'MultiQC',
                version: 'v1.2.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.2.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 293764,
              processor: {
                id: 45,
                name: 'Salmon Quant',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 293778,
              processor: {
                id: 46,
                name: 'MultiQC',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:59.893173Z',
          last_modified: '2019-02-13T05:20:09.029224Z',
          original_files: [10276],
          computed_files: [227595, 360793, 1076690, 1079475],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9213,
          title: 'Danio rerio',
          accession_code: 'ERR659767',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 7668,
              processor: {
                id: 10,
                name: 'Salmon Quant',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 7695,
              processor: {
                id: 11,
                name: 'MultiQC',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:37:03.541419Z',
          last_modified: '2019-02-13T05:20:09.016352Z',
          original_files: [10314],
          computed_files: [9237, 1076690, 1079473],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9310,
          title: 'Danio rerio',
          accession_code: 'ERR659773',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 33916,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 33935,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:37:13.963579Z',
          last_modified: '2019-02-13T05:20:08.969819Z',
          original_files: [10414],
          computed_files: [43062, 1076690, 1079467],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8886,
          title: 'Danio rerio',
          accession_code: 'ERR657546',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 14157,
              processor: {
                id: 10,
                name: 'Salmon Quant',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 14158,
              processor: {
                id: 11,
                name: 'MultiQC',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:35.349074Z',
          last_modified: '2019-02-13T05:20:09.122017Z',
          original_files: [9980],
          computed_files: [17803, 1076690, 1079489],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8907,
          title: 'Danio rerio',
          accession_code: 'ERR657547',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 92616,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 92651,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:37.126908Z',
          last_modified: '2019-02-13T05:20:09.115570Z',
          original_files: [10000],
          computed_files: [118782, 1076690, 1079488],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8991,
          title: 'Danio rerio',
          accession_code: 'ERR657551',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 41600,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 41612,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:44.242461Z',
          last_modified: '2019-02-13T05:20:09.088561Z',
          original_files: [10085],
          computed_files: [53099, 1076690, 1079484],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9032,
          title: 'Danio rerio',
          accession_code: 'ERR657553',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 30903,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 30918,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:47.690288Z',
          last_modified: '2019-02-13T05:20:09.075087Z',
          original_files: [10126],
          computed_files: [38965, 1076690, 1079482],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9053,
          title: 'Danio rerio',
          accession_code: 'ERR657554',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 8523,
              processor: {
                id: 10,
                name: 'Salmon Quant',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 8538,
              processor: {
                id: 11,
                name: 'MultiQC',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:49.450372Z',
          last_modified: '2019-02-13T05:20:09.068555Z',
          original_files: [10147],
          computed_files: [10479, 1076690, 1079481],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9073,
          title: 'Danio rerio',
          accession_code: 'ERR657555',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 72065,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 72067,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:51.120046Z',
          last_modified: '2019-02-13T05:20:09.062066Z',
          original_files: [10167],
          computed_files: [92380, 1076690, 1079480],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8786,
          title: 'Danio rerio',
          accession_code: 'ERR657541',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 94806,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 94817,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:26.479959Z',
          last_modified: '2019-02-13T05:20:09.159133Z',
          original_files: [9877],
          computed_files: [121868, 1076690, 1079494],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8826,
          title: 'Danio rerio',
          accession_code: 'ERR657543',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 70447,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 70449,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:29.957736Z',
          last_modified: '2019-02-13T05:20:09.142578Z',
          original_files: [9919],
          computed_files: [90256, 1076690, 1079492],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8846,
          title: 'Danio rerio',
          accession_code: 'ERR657544',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 190220,
              processor: {
                id: 40,
                name: 'Salmon Quant',
                version: 'v1.2.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.2.1'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 190478,
              processor: {
                id: 41,
                name: 'MultiQC',
                version: 'v1.2.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.2.1'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:31.716847Z',
          last_modified: '2019-02-13T05:20:09.135062Z',
          original_files: [9939],
          computed_files: [237553, 1076690, 1079491],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8865,
          title: 'Danio rerio',
          accession_code: 'ERR657545',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 92900,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 92927,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:33.560642Z',
          last_modified: '2019-02-13T05:20:09.128632Z',
          original_files: [9958],
          computed_files: [119147, 1076690, 1079490],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9192,
          title: 'Danio rerio',
          accession_code: 'ERR659766',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 235861,
              processor: {
                id: 46,
                name: 'MultiQC',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 282400,
              processor: {
                id: 45,
                name: 'Salmon Quant',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 282415,
              processor: {
                id: 46,
                name: 'MultiQC',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:37:01.634328Z',
          last_modified: '2019-02-13T05:20:09.022886Z',
          original_files: [10293],
          computed_files: [291778, 347744, 1076690, 1079474],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9246,
          title: 'Danio rerio',
          accession_code: 'ERR659769',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 7797,
              processor: {
                id: 10,
                name: 'Salmon Quant',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 7824,
              processor: {
                id: 11,
                name: 'MultiQC',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:37:07.012803Z',
          last_modified: '2019-02-13T05:20:09.003060Z',
          original_files: [10350],
          computed_files: [9442, 1076690, 1079471],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9262,
          title: 'Danio rerio',
          accession_code: 'ERR659770',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 88099,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 88102,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:37:08.738887Z',
          last_modified: '2019-02-13T05:20:08.996306Z',
          original_files: [10366],
          computed_files: [113223, 1076690, 1079470],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9294,
          title: 'Danio rerio',
          accession_code: 'ERR659772',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 93858,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 93886,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:37:12.245851Z',
          last_modified: '2019-02-13T05:20:08.979038Z',
          original_files: [10398],
          computed_files: [120543, 1076690, 1079468],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8928,
          title: 'Danio rerio',
          accession_code: 'ERR657548',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 31518,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 31530,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:38.920120Z',
          last_modified: '2019-02-13T05:20:09.108800Z',
          original_files: [10021],
          computed_files: [39797, 1076690, 1079487],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8949,
          title: 'Danio rerio',
          accession_code: 'ERR657549',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 7834,
              processor: {
                id: 10,
                name: 'Salmon Quant',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 7875,
              processor: {
                id: 11,
                name: 'MultiQC',
                version: 'v1.0.2',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.2'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:40.734586Z',
          last_modified: '2019-02-13T05:20:09.102441Z',
          original_files: [10042],
          computed_files: [9524, 1076690, 1079486],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 8970,
          title: 'Danio rerio',
          accession_code: 'ERR657550',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 25283,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 25317,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:42.510444Z',
          last_modified: '2019-02-13T05:20:09.095472Z',
          original_files: [10064],
          computed_files: [31344, 1076690, 1079485],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9012,
          title: 'Danio rerio',
          accession_code: 'ERR657552',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 93725,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 93766,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:45.982816Z',
          last_modified: '2019-02-13T05:20:09.081415Z',
          original_files: [10106],
          computed_files: [120391, 1076690, 1079483],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9094,
          title: 'Danio rerio',
          accession_code: 'ERR657556',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 94951,
              processor: {
                id: 16,
                name: 'Salmon Quant',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 94970,
              processor: {
                id: 17,
                name: 'MultiQC',
                version: 'v1.0.3',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.3'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:52.823823Z',
          last_modified: '2019-02-13T05:20:09.055593Z',
          original_files: [10189],
          computed_files: [122064, 1076690, 1079479],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9114,
          title: 'Danio rerio',
          accession_code: 'ERR657557',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 331612,
              processor: {
                id: 45,
                name: 'Salmon Quant',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 331618,
              processor: {
                id: 46,
                name: 'MultiQC',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:54.577247Z',
          last_modified: '2019-02-13T05:20:09.048452Z',
          original_files: [10215],
          computed_files: [401402, 1076690, 1079478],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9137,
          title: 'Danio rerio',
          accession_code: 'ERR657558',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 190370,
              processor: {
                id: 40,
                name: 'Salmon Quant',
                version: 'v1.2.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.2.1'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 190643,
              processor: {
                id: 41,
                name: 'MultiQC',
                version: 'v1.2.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.2.1'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:36:56.317171Z',
          last_modified: '2019-02-13T05:20:09.041880Z',
          original_files: [10237],
          computed_files: [237852, 1076690, 1079477],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 9229,
          title: 'Danio rerio',
          accession_code: 'ERR659768',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 4066,
              processor: {
                id: 3,
                name: 'Salmon Quant',
                version: 'v1.0.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.1'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 4128,
              processor: {
                id: 4,
                name: 'Salmontools',
                version: 'v1.0.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    'g++': '4:5.3.1-1ubuntu1',
                    cmake: '3.5.1-1ubuntu3',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    'data-refinery-common': '1.0.1'
                  },
                  cmd_line: {
                    'salmontools --version': 'Salmon Tools 0.1.0'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 4129,
              processor: {
                id: 5,
                name: 'MultiQC',
                version: 'v1.0.1',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.0.6',
                    multiqc: '1.5',
                    'data-refinery-common': '1.0.1'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:37:05.241558Z',
          last_modified: '2019-02-13T05:20:09.009809Z',
          original_files: [10330],
          computed_files: [4904, 4905, 1076690, 1079472],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        },
        {
          id: 5743,
          title: 'Danio rerio',
          accession_code: 'ERR657521',
          source_database: 'SRA',
          organism: {
            name: 'DANIO_RERIO',
            taxonomy_id: 7955
          },
          platform_accession_code: 'IlluminaHiSeq2500',
          platform_name: 'Illumina HiSeq 2500',
          pretty_platform: 'Illumina HiSeq 2500 (IlluminaHiSeq2500)',
          technology: 'RNA-SEQ',
          manufacturer: 'ILLUMINA',
          protocol_info: [
            {
              Reference: 'https://www.ebi.ac.uk/ena/data/view/ERP006132',
              Description:
                'Illumina sequencing of library 10756881, constructed from sample accession ERS499086 for study accession ERP006132.  This is part of an Illumina multiplexed sequencing run (13467_1).  This submission includes reads tagged with the sequence TGCTGATA.'
            }
          ],
          annotations: [],
          results: [
            {
              id: 330257,
              processor: {
                id: 45,
                name: 'Salmon Quant',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: {
                id: 86,
                assembly_name: 'GRCz11',
                organism_name: 'DANIO_RERIO',
                database_name: 'EnsemblMain',
                release_version: '93',
                index_type: 'TRANSCRIPTOME_SHORT',
                salmon_version: 'salmon 0.9.1',
                download_url:
                  'https://s3.amazonaws.com/data-refinery-s3-transcriptome-index-circleci-prod/DANIO_RERIO_TRANSCRIPTOME_SHORT.tar.gz',
                result_id: 85,
                last_modified: '2018-09-27T15:28:53.184909Z'
              }
            },
            {
              id: 330271,
              processor: {
                id: 46,
                name: 'MultiQC',
                version: 'v1.3.0',
                docker_image: 'dr_salmon',
                environment: {
                  os_pkg: {
                    python3: '3.5.1-3',
                    'default-jre': '2:1.8-56ubuntu2',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.2',
                    multiqc: '1.6',
                    'data-refinery-common': '1.3.0'
                  },
                  cmd_line: {
                    'salmon --version': 'salmon 0.9.1',
                    '/home/user/FastQC/fastqc --version': 'FastQC v0.11.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            },
            {
              id: 959822,
              processor: {
                id: 127,
                name: 'Tximport',
                version: 'v1.7.12-hotfix',
                docker_image: 'dr_salmon',
                environment: {
                  R: {
                    readr: '1.1.1',
                    rjson: '0.2.19',
                    tximport: '1.6.0',
                    Bioconductor: '3.5'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    numpy: '1.15.2',
                    Django: '2.1.5',
                    pandas: '0.23.4',
                    'data-refinery-common': '=v1.7.12-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-09-27T15:33:42.651300Z',
          last_modified: '2019-02-13T05:20:09.302717Z',
          original_files: [6676],
          computed_files: [399915, 1076690, 1079514],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['ERP006132']
        }
      ]
    }
  ],
  [
    {
      count: 2554,
      next: 'http://api.refine.bio/v1/samples/?experiment_accession_code=GSE57542&limit=10&offset=10',
      previous: null,
      results: [
        {
          id: 2011727,
          title: '1250V083011_dNS1_20121210_n143_01_11.RCC',
          accession_code: 'GSM1285069',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hugene10st',
          platform_name: '[HuGene-1_0-st] Affymetrix Human Gene 1.0 ST Array',
          pretty_platform: 'Affymetrix Human Gene 1.0 ST Array (hugene10st)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1285069',
            'Scan protocol': [
              'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
            ],
            'Label protocol': ['none'],
            'Data processing': [
              'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using Qiagen RNeasy columns.'
            ],
            'Hybridization protocol': [
              'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
            ]
          },
          annotations: [
            {
              data: {
                type: ['RNA'],
                title: ['1250V083011_dNS1_20121210_n143_01_11.RCC'],
                status: ['Public on Mar 01 2014'],
                label_ch1: ['n/a'],
                series_id: ['GSE53165', 'GSE57542'],
                taxid_ch1: ['9606'],
                platform_id: ['GPL18032'],
                contact_city: ['Cambridge'],
                contact_name: ['Nir,,Hacohen'],
                hyb_protocol: [
                  'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
                ],
                molecule_ch1: ['total RNA'],
                organism_ch1: ['Homo sapiens'],
                channel_count: ['1'],
                contact_email: ['nhacohen@partners.org'],
                contact_state: ['MA'],
                geo_accession: ['GSM1285069'],
                scan_protocol: [
                  'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
                ],
                data_row_count: ['414'],
                contact_address: ['7 Cambridge Center'],
                contact_country: ['USA'],
                data_processing: [
                  'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
                ],
                source_name_ch1: ['1250V083011'],
                submission_date: ['Dec 10 2013'],
                last_update_date: ['Mar 01 2014'],
                contact_institute: ['Broad Institute'],
                label_protocol_ch1: ['none'],
                supplementary_file: [
                  'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1285nnn/GSM1285069/suppl/GSM1285069_20121210_n143_01_11.RCC.gz'
                ],
                characteristics_ch1: [
                  'stimulation: dNS1',
                  'hours: 10',
                  'plate: V2',
                  'cartridge: n143',
                  'lane: 11',
                  'date of visit: 2011-08-30',
                  'time of visit: 8:13 AM',
                  'gender: M',
                  'age (years): 35',
                  'ethnicity: Caucasian',
                  'million pbmc: 29.555588',
                  'cd14 %: 96.4',
                  'cd16 %: 0.2',
                  'million dcs: 0.475',
                  'height (cm): 180',
                  'weight (kg): 85.65',
                  'systolic/diastolic: 104/78',
                  'bmi: 26.43518519'
                ],
                growth_protocol_ch1: [
                  'Monocyte-derived dendritic cells from each donor were cultured for 7 days in RPMI supplemented with 10% FBS, 100 ng/mL GM-CSF and 40 ng/mL IL-4.'
                ],
                extract_protocol_ch1: [
                  'Total RNA was isolated using Qiagen RNeasy columns.'
                ],
                treatment_protocol_ch1: [
                  'MoDCs were stimulated with 15 ng/mL LPS for 2.5 h or 5 h, influenza virus for 10 h, 100 U/mL IFN for 6.5 h, or left unstimulated.'
                ],
                'contact_zip/postal_code': ['02143']
              },
              is_ccdl: false,
              created_at: '2019-09-27T06:47:49.856246Z',
              last_modified: '2019-09-27T06:47:49.856246Z'
            }
          ],
          results: [
            {
              id: 2655522,
              processor: {
                id: 464,
                name: 'Submitter-processed',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_no_op',
                environment: {
                  R: {
                    dplyr: '0.7.8',
                    rlang: '0.3.1',
                    Bioconductor: '3.5',
                    AnnotationDbi: '1.40.0',
                    'illuminaRatv1.db': '1.26.0',
                    'illuminaHumanv1.db': '1.26.0',
                    'illuminaHumanv2.db': '1.26.0',
                    'illuminaHumanv3.db': '1.26.0',
                    'illuminaHumanv4.db': '1.26.0',
                    'illuminaMousev1.db': '1.26.0',
                    'illuminaMousev2.db': '1.26.0',
                    'illuminaMousev1p1.db': '1.26.0'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'cat /etc/identifier_refinery_url':
                      'https://zenodo.org/record/1410647/files/all_1536267482.zip'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: false,
          sex: 'male',
          age: '35.000',
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: 'caucasian',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-09-27T06:47:49.851378Z',
          last_modified: '2019-09-27T08:59:18.287330Z',
          original_files: [2716213, 2717627, 2723618, 2728673],
          computed_files: [4976380],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE57542']
        },
        {
          id: 563773,
          title: 'V4_rep [217665]',
          accession_code: 'GSM1281538',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hugene10st',
          platform_name: '[HuGene-1_0-st] Affymetrix Human Gene 1.0 ST Array',
          pretty_platform: 'Affymetrix Human Gene 1.0 ST Array (hugene10st)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1281538',
            'Scan protocol': [
              'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
            ],
            'Label protocol': [
              'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
            ],
            'Data processing': [
              'Data were processed using Bioconductor RMA package.',
              'probe group file: HuGene-1_0-st-v1.r4.pgf',
              'meta-probeset file: HuGene-1_0-st-v1.r4.mps'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using Qiagen RNeasy columns.'
            ],
            'Hybridization protocol': [
              'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
            ]
          },
          annotations: [
            {
              data: {
                type: ['RNA'],
                title: ['V4_rep [217665]'],
                status: ['Public on Mar 01 2014'],
                relation: ['Reanalyzed by: GSE86357'],
                label_ch1: ['biotin'],
                series_id: ['GSE53166', 'GSE57542'],
                taxid_ch1: ['9606'],
                platform_id: ['GPL6244'],
                contact_city: ['Cambridge'],
                contact_name: ['Nir,,Hacohen'],
                hyb_protocol: [
                  'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
                ],
                molecule_ch1: ['total RNA'],
                organism_ch1: ['Homo sapiens'],
                channel_count: ['1'],
                contact_email: ['nhacohen@partners.org'],
                contact_state: ['MA'],
                geo_accession: ['GSM1281538'],
                scan_protocol: [
                  'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
                ],
                data_row_count: ['26981'],
                contact_address: ['7 Cambridge Center'],
                contact_country: ['USA'],
                data_processing: [
                  'Data were processed using Bioconductor RMA package.',
                  'probe group file: HuGene-1_0-st-v1.r4.pgf',
                  'meta-probeset file: HuGene-1_0-st-v1.r4.mps'
                ],
                source_name_ch1: ['V4_rep'],
                submission_date: ['Dec 06 2013'],
                last_update_date: ['Sep 01 2016'],
                contact_institute: ['Broad Institute'],
                label_protocol_ch1: [
                  'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
                ],
                supplementary_file: [
                  'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1281nnn/GSM1281538/suppl/GSM1281538_EA10047_217665_HUGENE-1_0-ST-V1_V4_REP.CEL.gz'
                ],
                characteristics_ch1: [
                  'stimulation: dNS1',
                  'total rna (ug): 7.425',
                  'rna concentration (ug/ul): 0.265',
                  'cd4 %: 96.6',
                  'cd16 %: 0.7',
                  'age (years): 38',
                  'million modc: 0.39875',
                  'million monocyte: 1.3171536',
                  'million pbmc: 36.498402',
                  'ethnicity: Asian',
                  'Sex: M',
                  'individual id: IGTB1153'
                ],
                growth_protocol_ch1: [
                  'Monocyte-derived dendritic cells from each donor were cultured for 7 days in RPMI supplemented with 10% FBS, 100 ng/mL GM-CSF and 40 ng/mL IL-4.'
                ],
                extract_protocol_ch1: [
                  'Total RNA was isolated using Qiagen RNeasy columns.'
                ],
                treatment_protocol_ch1: [
                  'MoDCs were stimulated with 15 ng/mL LPS for 5 h, influenza virus for 10 h, or left unstimulated.'
                ],
                'contact_zip/postal_code': ['02143']
              },
              is_ccdl: false,
              created_at: '2018-12-14T04:22:59.346792Z',
              last_modified: '2018-12-14T04:22:59.346792Z'
            }
          ],
          results: [
            {
              id: 728702,
              processor: {
                id: 95,
                name: 'Affymetrix SCAN',
                version: 'v1.4.6',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.26.0',
                    affy: '1.56.0',
                    limma: '3.34.9',
                    oligo: '1.42.0',
                    affyio: '1.48.0',
                    'SCAN.UPC': '2.20.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.6',
                    AnnotationDbi: '1.40.0'
                  },
                  os_pkg: {
                    'r-base': '3.4.4-1xenial0',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    rpy2: '2.9.5',
                    Django: '2.1.2',
                    'data-refinery-common': '1.4.6'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'male',
          age: '38.000',
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: 'asian',
          subject: 'igtb1153',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-14T04:22:57.608457Z',
          last_modified: '2018-12-14T22:27:19.558172Z',
          original_files: [872389, 993603, 2723177],
          computed_files: [844648],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE53166', 'GSE57542']
        },
        {
          id: 2010591,
          title: '1663V012612_dNS1_20121206_n116_01_06.RCC',
          accession_code: 'GSM1284740',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hugene10st',
          platform_name: '[HuGene-1_0-st] Affymetrix Human Gene 1.0 ST Array',
          pretty_platform: 'Affymetrix Human Gene 1.0 ST Array (hugene10st)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1284740',
            'Scan protocol': [
              'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
            ],
            'Label protocol': ['none'],
            'Data processing': [
              'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using Qiagen RNeasy columns.'
            ],
            'Hybridization protocol': [
              'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
            ]
          },
          annotations: [
            {
              data: {
                type: ['RNA'],
                title: ['1663V012612_dNS1_20121206_n116_01_06.RCC'],
                status: ['Public on Mar 01 2014'],
                label_ch1: ['n/a'],
                series_id: ['GSE53165', 'GSE57542'],
                taxid_ch1: ['9606'],
                platform_id: ['GPL18032'],
                contact_city: ['Cambridge'],
                contact_name: ['Nir,,Hacohen'],
                hyb_protocol: [
                  'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
                ],
                molecule_ch1: ['total RNA'],
                organism_ch1: ['Homo sapiens'],
                channel_count: ['1'],
                contact_email: ['nhacohen@partners.org'],
                contact_state: ['MA'],
                geo_accession: ['GSM1284740'],
                scan_protocol: [
                  'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
                ],
                data_row_count: ['414'],
                contact_address: ['7 Cambridge Center'],
                contact_country: ['USA'],
                data_processing: [
                  'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
                ],
                source_name_ch1: ['1663V012612'],
                submission_date: ['Dec 10 2013'],
                last_update_date: ['Mar 01 2014'],
                contact_institute: ['Broad Institute'],
                label_protocol_ch1: ['none'],
                supplementary_file: [
                  'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1284nnn/GSM1284740/suppl/GSM1284740_20121206_n116_01_06.RCC.gz'
                ],
                characteristics_ch1: [
                  'stimulation: dNS1',
                  'hours: 10',
                  'plate: V2',
                  'cartridge: n116',
                  'lane: 6',
                  'date of visit: 2012-01-26',
                  'time of visit: 8:46 AM',
                  'gender: F',
                  'age (years): 32',
                  'ethnicity: East Asian',
                  'million pbmc: 27.5534555',
                  'cd14 %: 96.7',
                  'cd16 %: 1.3',
                  'million dcs: 0.1296',
                  'height (cm): 167',
                  'weight (kg): 65.85',
                  'systolic/diastolic: 90/60',
                  'bmi: 23.61145972'
                ],
                growth_protocol_ch1: [
                  'Monocyte-derived dendritic cells from each donor were cultured for 7 days in RPMI supplemented with 10% FBS, 100 ng/mL GM-CSF and 40 ng/mL IL-4.'
                ],
                extract_protocol_ch1: [
                  'Total RNA was isolated using Qiagen RNeasy columns.'
                ],
                treatment_protocol_ch1: [
                  'MoDCs were stimulated with 15 ng/mL LPS for 2.5 h or 5 h, influenza virus for 10 h, 100 U/mL IFN for 6.5 h, or left unstimulated.'
                ],
                'contact_zip/postal_code': ['02143']
              },
              is_ccdl: false,
              created_at: '2019-09-27T06:44:16.084097Z',
              last_modified: '2019-09-27T06:44:16.084097Z'
            }
          ],
          results: [
            {
              id: 2655482,
              processor: {
                id: 464,
                name: 'Submitter-processed',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_no_op',
                environment: {
                  R: {
                    dplyr: '0.7.8',
                    rlang: '0.3.1',
                    Bioconductor: '3.5',
                    AnnotationDbi: '1.40.0',
                    'illuminaRatv1.db': '1.26.0',
                    'illuminaHumanv1.db': '1.26.0',
                    'illuminaHumanv2.db': '1.26.0',
                    'illuminaHumanv3.db': '1.26.0',
                    'illuminaHumanv4.db': '1.26.0',
                    'illuminaMousev1.db': '1.26.0',
                    'illuminaMousev2.db': '1.26.0',
                    'illuminaMousev1p1.db': '1.26.0'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'cat /etc/identifier_refinery_url':
                      'https://zenodo.org/record/1410647/files/all_1536267482.zip'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: false,
          sex: 'female',
          age: '32.000',
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: 'east asian',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-09-27T06:44:16.079734Z',
          last_modified: '2019-09-27T08:58:41.258291Z',
          original_files: [2714935, 2717627, 2721643, 2723601],
          computed_files: [4976340],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE57542']
        },
        {
          id: 572360,
          title: '21V [209538]',
          accession_code: 'GSM1281478',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hugene10st',
          platform_name: '[HuGene-1_0-st] Affymetrix Human Gene 1.0 ST Array',
          pretty_platform: 'Affymetrix Human Gene 1.0 ST Array (hugene10st)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1281478',
            'Scan protocol': [
              'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
            ],
            'Label protocol': [
              'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
            ],
            'Data processing': [
              'Data were processed using Bioconductor RMA package.',
              'probe group file: HuGene-1_0-st-v1.r4.pgf',
              'meta-probeset file: HuGene-1_0-st-v1.r4.mps'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using Qiagen RNeasy columns.'
            ],
            'Hybridization protocol': [
              'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
            ]
          },
          annotations: [
            {
              data: {
                type: ['RNA'],
                title: ['21V [209538]'],
                status: ['Public on Mar 01 2014'],
                relation: ['Reanalyzed by: GSE86357'],
                label_ch1: ['biotin'],
                series_id: ['GSE53166', 'GSE57542'],
                taxid_ch1: ['9606'],
                platform_id: ['GPL6244'],
                contact_city: ['Cambridge'],
                contact_name: ['Nir,,Hacohen'],
                hyb_protocol: [
                  'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
                ],
                molecule_ch1: ['total RNA'],
                organism_ch1: ['Homo sapiens'],
                channel_count: ['1'],
                contact_email: ['nhacohen@partners.org'],
                contact_state: ['MA'],
                geo_accession: ['GSM1281478'],
                scan_protocol: [
                  'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
                ],
                data_row_count: ['26981'],
                contact_address: ['7 Cambridge Center'],
                contact_country: ['USA'],
                data_processing: [
                  'Data were processed using Bioconductor RMA package.',
                  'probe group file: HuGene-1_0-st-v1.r4.pgf',
                  'meta-probeset file: HuGene-1_0-st-v1.r4.mps'
                ],
                source_name_ch1: ['21V'],
                submission_date: ['Dec 06 2013'],
                last_update_date: ['Sep 01 2016'],
                contact_institute: ['Broad Institute'],
                label_protocol_ch1: [
                  'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
                ],
                supplementary_file: [
                  'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1281nnn/GSM1281478/suppl/GSM1281478_EA10047_209538_HUGENE-1_0-ST-V1_21V.CEL.gz'
                ],
                characteristics_ch1: [
                  'stimulation: dNS1',
                  'total rna (ug): 7.864',
                  'rna concentration (ug/ul): 0.281',
                  'cd4 %: 90.5',
                  'cd16 %: 0.9',
                  'age (years): 26',
                  'million modc: 0.300751857',
                  'million monocyte: 2.17936128',
                  'million pbmc: 49.35956',
                  'ethnicity: Caucasian',
                  'Sex: F',
                  'individual id: IGTB950'
                ],
                growth_protocol_ch1: [
                  'Monocyte-derived dendritic cells from each donor were cultured for 7 days in RPMI supplemented with 10% FBS, 100 ng/mL GM-CSF and 40 ng/mL IL-4.'
                ],
                extract_protocol_ch1: [
                  'Total RNA was isolated using Qiagen RNeasy columns.'
                ],
                treatment_protocol_ch1: [
                  'MoDCs were stimulated with 15 ng/mL LPS for 5 h, influenza virus for 10 h, or left unstimulated.'
                ],
                'contact_zip/postal_code': ['02143']
              },
              is_ccdl: false,
              created_at: '2018-12-14T04:40:45.558751Z',
              last_modified: '2018-12-14T04:40:45.558751Z'
            }
          ],
          results: [
            {
              id: 730136,
              processor: {
                id: 95,
                name: 'Affymetrix SCAN',
                version: 'v1.4.6',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.26.0',
                    affy: '1.56.0',
                    limma: '3.34.9',
                    oligo: '1.42.0',
                    affyio: '1.48.0',
                    'SCAN.UPC': '2.20.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.6',
                    AnnotationDbi: '1.40.0'
                  },
                  os_pkg: {
                    'r-base': '3.4.4-1xenial0',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    rpy2: '2.9.5',
                    Django: '2.1.2',
                    'data-refinery-common': '1.4.6'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'female',
          age: '26.000',
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: 'caucasian',
          subject: 'igtb950',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-14T04:40:44.245151Z',
          last_modified: '2018-12-14T23:11:36.255018Z',
          original_files: [880834, 1017150, 2722472],
          computed_files: [846080],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE53166', 'GSE57542']
        },
        {
          id: 2011751,
          title: '0695V060711_dNS1_20121209_n140_01_09.RCC',
          accession_code: 'GSM1285031',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hugene10st',
          platform_name: '[HuGene-1_0-st] Affymetrix Human Gene 1.0 ST Array',
          pretty_platform: 'Affymetrix Human Gene 1.0 ST Array (hugene10st)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1285031',
            'Scan protocol': [
              'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
            ],
            'Label protocol': ['none'],
            'Data processing': [
              'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using Qiagen RNeasy columns.'
            ],
            'Hybridization protocol': [
              'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
            ]
          },
          annotations: [
            {
              data: {
                type: ['RNA'],
                title: ['0695V060711_dNS1_20121209_n140_01_09.RCC'],
                status: ['Public on Mar 01 2014'],
                label_ch1: ['n/a'],
                series_id: ['GSE53165', 'GSE57542'],
                taxid_ch1: ['9606'],
                platform_id: ['GPL18032'],
                contact_city: ['Cambridge'],
                contact_name: ['Nir,,Hacohen'],
                hyb_protocol: [
                  'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
                ],
                molecule_ch1: ['total RNA'],
                organism_ch1: ['Homo sapiens'],
                channel_count: ['1'],
                contact_email: ['nhacohen@partners.org'],
                contact_state: ['MA'],
                geo_accession: ['GSM1285031'],
                scan_protocol: [
                  'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
                ],
                data_row_count: ['414'],
                contact_address: ['7 Cambridge Center'],
                contact_country: ['USA'],
                data_processing: [
                  'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
                ],
                source_name_ch1: ['0695V060711'],
                submission_date: ['Dec 10 2013'],
                last_update_date: ['Mar 01 2014'],
                contact_institute: ['Broad Institute'],
                label_protocol_ch1: ['none'],
                supplementary_file: [
                  'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1285nnn/GSM1285031/suppl/GSM1285031_20121209_n140_01_09.RCC.gz'
                ],
                characteristics_ch1: [
                  'stimulation: dNS1',
                  'hours: 10',
                  'plate: V1',
                  'cartridge: n140',
                  'lane: 9',
                  'date of visit: 2011-06-07',
                  'time of visit: 7:58 AM',
                  'gender: M',
                  'age (years): 38',
                  'ethnicity: Caucasian',
                  'million pbmc: 32.287787',
                  'cd14 %: 98.2',
                  'cd16 %: 0.1',
                  'million dcs: 0.45',
                  'height (cm): 179.5',
                  'weight (kg): 96.8',
                  'systolic/diastolic: 100/66',
                  'bmi: 30.04321816'
                ],
                growth_protocol_ch1: [
                  'Monocyte-derived dendritic cells from each donor were cultured for 7 days in RPMI supplemented with 10% FBS, 100 ng/mL GM-CSF and 40 ng/mL IL-4.'
                ],
                extract_protocol_ch1: [
                  'Total RNA was isolated using Qiagen RNeasy columns.'
                ],
                treatment_protocol_ch1: [
                  'MoDCs were stimulated with 15 ng/mL LPS for 2.5 h or 5 h, influenza virus for 10 h, 100 U/mL IFN for 6.5 h, or left unstimulated.'
                ],
                'contact_zip/postal_code': ['02143']
              },
              is_ccdl: false,
              created_at: '2019-09-27T06:47:54.807194Z',
              last_modified: '2019-09-27T06:47:54.807194Z'
            }
          ],
          results: [
            {
              id: 2655542,
              processor: {
                id: 464,
                name: 'Submitter-processed',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_no_op',
                environment: {
                  R: {
                    dplyr: '0.7.8',
                    rlang: '0.3.1',
                    Bioconductor: '3.5',
                    AnnotationDbi: '1.40.0',
                    'illuminaRatv1.db': '1.26.0',
                    'illuminaHumanv1.db': '1.26.0',
                    'illuminaHumanv2.db': '1.26.0',
                    'illuminaHumanv3.db': '1.26.0',
                    'illuminaHumanv4.db': '1.26.0',
                    'illuminaMousev1.db': '1.26.0',
                    'illuminaMousev2.db': '1.26.0',
                    'illuminaMousev1p1.db': '1.26.0'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'cat /etc/identifier_refinery_url':
                      'https://zenodo.org/record/1410647/files/all_1536267482.zip'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: false,
          sex: 'male',
          age: '38.000',
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: 'caucasian',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-09-27T06:47:54.794340Z',
          last_modified: '2019-09-27T08:59:32.015196Z',
          original_files: [2716238, 2717627, 2723605, 2728104],
          computed_files: [4976400],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE57542']
        },
        {
          id: 2010508,
          title: '1120V012011_unstim_20121210_n151_01_07.RCC',
          accession_code: 'GSM1285161',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hugene10st',
          platform_name: '[HuGene-1_0-st] Affymetrix Human Gene 1.0 ST Array',
          pretty_platform: 'Affymetrix Human Gene 1.0 ST Array (hugene10st)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1285161',
            'Scan protocol': [
              'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
            ],
            'Label protocol': ['none'],
            'Data processing': [
              'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using Qiagen RNeasy columns.'
            ],
            'Hybridization protocol': [
              'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
            ]
          },
          annotations: [
            {
              data: {
                type: ['RNA'],
                title: ['1120V012011_unstim_20121210_n151_01_07.RCC'],
                status: ['Public on Mar 01 2014'],
                label_ch1: ['n/a'],
                series_id: ['GSE53165', 'GSE57542'],
                taxid_ch1: ['9606'],
                platform_id: ['GPL18032'],
                contact_city: ['Cambridge'],
                contact_name: ['Nir,,Hacohen'],
                hyb_protocol: [
                  'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
                ],
                molecule_ch1: ['total RNA'],
                organism_ch1: ['Homo sapiens'],
                channel_count: ['1'],
                contact_email: ['nhacohen@partners.org'],
                contact_state: ['MA'],
                geo_accession: ['GSM1285161'],
                scan_protocol: [
                  'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
                ],
                data_row_count: ['414'],
                contact_address: ['7 Cambridge Center'],
                contact_country: ['USA'],
                data_processing: [
                  'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
                ],
                source_name_ch1: ['1120V012011'],
                submission_date: ['Dec 10 2013'],
                last_update_date: ['Mar 01 2014'],
                contact_institute: ['Broad Institute'],
                label_protocol_ch1: ['none'],
                supplementary_file: [
                  'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1285nnn/GSM1285161/suppl/GSM1285161_20121210_n151_01_07.RCC.gz'
                ],
                characteristics_ch1: [
                  'stimulation: unstim',
                  'hours: 0',
                  'plate: U1',
                  'cartridge: n151',
                  'lane: 7',
                  'date of visit: 2011-01-20',
                  'time of visit: 8:32 AM',
                  'gender: F',
                  'age (years): 45',
                  'ethnicity: East Asian',
                  'million pbmc: 18.55092765',
                  'cd14 %: 93.9',
                  'cd16 %: 1.5',
                  'million dcs: 0.1353',
                  'height (cm): 164',
                  'weight (kg): 78.9',
                  'systolic/diastolic: 162/96',
                  'bmi: 29.33521713'
                ],
                growth_protocol_ch1: [
                  'Monocyte-derived dendritic cells from each donor were cultured for 7 days in RPMI supplemented with 10% FBS, 100 ng/mL GM-CSF and 40 ng/mL IL-4.'
                ],
                extract_protocol_ch1: [
                  'Total RNA was isolated using Qiagen RNeasy columns.'
                ],
                treatment_protocol_ch1: [
                  'MoDCs were stimulated with 15 ng/mL LPS for 2.5 h or 5 h, influenza virus for 10 h, 100 U/mL IFN for 6.5 h, or left unstimulated.'
                ],
                'contact_zip/postal_code': ['02143']
              },
              is_ccdl: false,
              created_at: '2019-09-27T06:43:59.994279Z',
              last_modified: '2019-09-27T06:43:59.994279Z'
            }
          ],
          results: [
            {
              id: 2655565,
              processor: {
                id: 464,
                name: 'Submitter-processed',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_no_op',
                environment: {
                  R: {
                    dplyr: '0.7.8',
                    rlang: '0.3.1',
                    Bioconductor: '3.5',
                    AnnotationDbi: '1.40.0',
                    'illuminaRatv1.db': '1.26.0',
                    'illuminaHumanv1.db': '1.26.0',
                    'illuminaHumanv2.db': '1.26.0',
                    'illuminaHumanv3.db': '1.26.0',
                    'illuminaHumanv4.db': '1.26.0',
                    'illuminaMousev1.db': '1.26.0',
                    'illuminaMousev2.db': '1.26.0',
                    'illuminaMousev1p1.db': '1.26.0'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'cat /etc/identifier_refinery_url':
                      'https://zenodo.org/record/1410647/files/all_1536267482.zip'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: false,
          sex: 'female',
          age: '45.000',
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: 'east asian',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-09-27T06:43:59.989307Z',
          last_modified: '2019-09-27T08:59:45.459131Z',
          original_files: [2714846, 2717627, 2721550, 2723534],
          computed_files: [4976423],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE57542']
        },
        {
          id: 2010502,
          title: '1423V092211_dNS1_20121216_n199_01_05.RCC',
          accession_code: 'GSM1285723',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hugene10st',
          platform_name: '[HuGene-1_0-st] Affymetrix Human Gene 1.0 ST Array',
          pretty_platform: 'Affymetrix Human Gene 1.0 ST Array (hugene10st)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1285723',
            'Scan protocol': [
              'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
            ],
            'Label protocol': ['none'],
            'Data processing': [
              'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using Qiagen RNeasy columns.'
            ],
            'Hybridization protocol': [
              'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
            ]
          },
          annotations: [
            {
              data: {
                type: ['RNA'],
                title: ['1423V092211_dNS1_20121216_n199_01_05.RCC'],
                status: ['Public on Mar 01 2014'],
                label_ch1: ['n/a'],
                series_id: ['GSE53165', 'GSE57542'],
                taxid_ch1: ['9606'],
                platform_id: ['GPL18032'],
                contact_city: ['Cambridge'],
                contact_name: ['Nir,,Hacohen'],
                hyb_protocol: [
                  'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
                ],
                molecule_ch1: ['total RNA'],
                organism_ch1: ['Homo sapiens'],
                channel_count: ['1'],
                contact_email: ['nhacohen@partners.org'],
                contact_state: ['MA'],
                geo_accession: ['GSM1285723'],
                scan_protocol: [
                  'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
                ],
                data_row_count: ['414'],
                contact_address: ['7 Cambridge Center'],
                contact_country: ['USA'],
                data_processing: [
                  'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
                ],
                source_name_ch1: ['1423V092211'],
                submission_date: ['Dec 10 2013'],
                last_update_date: ['Mar 01 2014'],
                contact_institute: ['Broad Institute'],
                label_protocol_ch1: ['none'],
                supplementary_file: [
                  'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1285nnn/GSM1285723/suppl/GSM1285723_20121216_n199_01_05.RCC.gz'
                ],
                characteristics_ch1: [
                  'stimulation: dNS1',
                  'hours: 10',
                  'plate: V1',
                  'cartridge: n199',
                  'lane: 5',
                  'date of visit: 2011-09-22',
                  'time of visit: 8:01 AM',
                  'gender: M',
                  'age (years): 34',
                  'ethnicity: Caucasian',
                  'million pbmc: 38.996658',
                  'cd14 %: 91.3',
                  'cd16 %: 1',
                  'million dcs: 0.4644',
                  'height (cm): 182.11',
                  'weight (kg): 76.75',
                  'systolic/diastolic: 110/70',
                  'bmi: 23.14252787'
                ],
                growth_protocol_ch1: [
                  'Monocyte-derived dendritic cells from each donor were cultured for 7 days in RPMI supplemented with 10% FBS, 100 ng/mL GM-CSF and 40 ng/mL IL-4.'
                ],
                extract_protocol_ch1: [
                  'Total RNA was isolated using Qiagen RNeasy columns.'
                ],
                treatment_protocol_ch1: [
                  'MoDCs were stimulated with 15 ng/mL LPS for 2.5 h or 5 h, influenza virus for 10 h, 100 U/mL IFN for 6.5 h, or left unstimulated.'
                ],
                'contact_zip/postal_code': ['02143']
              },
              is_ccdl: false,
              created_at: '2019-09-27T06:43:58.732041Z',
              last_modified: '2019-09-27T06:43:58.732041Z'
            }
          ],
          results: [
            {
              id: 2655511,
              processor: {
                id: 464,
                name: 'Submitter-processed',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_no_op',
                environment: {
                  R: {
                    dplyr: '0.7.8',
                    rlang: '0.3.1',
                    Bioconductor: '3.5',
                    AnnotationDbi: '1.40.0',
                    'illuminaRatv1.db': '1.26.0',
                    'illuminaHumanv1.db': '1.26.0',
                    'illuminaHumanv2.db': '1.26.0',
                    'illuminaHumanv3.db': '1.26.0',
                    'illuminaHumanv4.db': '1.26.0',
                    'illuminaMousev1.db': '1.26.0',
                    'illuminaMousev2.db': '1.26.0',
                    'illuminaMousev1p1.db': '1.26.0'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'cat /etc/identifier_refinery_url':
                      'https://zenodo.org/record/1410647/files/all_1536267482.zip'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: false,
          sex: 'male',
          age: '34.000',
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: 'caucasian',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-09-27T06:43:58.702363Z',
          last_modified: '2019-09-27T08:59:08.671176Z',
          original_files: [2714838, 2717627, 2723577, 2721693],
          computed_files: [4976369],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE57542']
        },
        {
          id: 2010583,
          title: '1442V100411_LPS_20121207_n125_01_10.RCC',
          accession_code: 'GSM1284852',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hugene10st',
          platform_name: '[HuGene-1_0-st] Affymetrix Human Gene 1.0 ST Array',
          pretty_platform: 'Affymetrix Human Gene 1.0 ST Array (hugene10st)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1284852',
            'Scan protocol': [
              'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
            ],
            'Label protocol': ['none'],
            'Data processing': [
              'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using Qiagen RNeasy columns.'
            ],
            'Hybridization protocol': [
              'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
            ]
          },
          annotations: [
            {
              data: {
                type: ['RNA'],
                title: ['1442V100411_LPS_20121207_n125_01_10.RCC'],
                status: ['Public on Mar 01 2014'],
                label_ch1: ['n/a'],
                series_id: ['GSE53165', 'GSE57542'],
                taxid_ch1: ['9606'],
                platform_id: ['GPL18032'],
                contact_city: ['Cambridge'],
                contact_name: ['Nir,,Hacohen'],
                hyb_protocol: [
                  'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
                ],
                molecule_ch1: ['total RNA'],
                organism_ch1: ['Homo sapiens'],
                channel_count: ['1'],
                contact_email: ['nhacohen@partners.org'],
                contact_state: ['MA'],
                geo_accession: ['GSM1284852'],
                scan_protocol: [
                  'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
                ],
                data_row_count: ['414'],
                contact_address: ['7 Cambridge Center'],
                contact_country: ['USA'],
                data_processing: [
                  'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
                ],
                source_name_ch1: ['1442V100411'],
                submission_date: ['Dec 10 2013'],
                last_update_date: ['Mar 01 2014'],
                contact_institute: ['Broad Institute'],
                label_protocol_ch1: ['none'],
                supplementary_file: [
                  'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1284nnn/GSM1284852/suppl/GSM1284852_20121207_n125_01_10.RCC.gz'
                ],
                characteristics_ch1: [
                  'stimulation: LPS',
                  'hours: 2.5',
                  'plate: L2a',
                  'cartridge: n125',
                  'lane: 10',
                  'date of visit: 2011-10-04',
                  'time of visit: 7:45 AM',
                  'gender: F',
                  'age (years): 23',
                  'ethnicity: Caucasian',
                  'million pbmc: 48.332071',
                  'cd14 %: 93.5',
                  'cd16 %: 0.6',
                  'million dcs: 0.2106',
                  'height (cm): 156',
                  'weight (kg): 54.05',
                  'systolic/diastolic: 102/68',
                  'bmi: 22.20989481'
                ],
                growth_protocol_ch1: [
                  'Monocyte-derived dendritic cells from each donor were cultured for 7 days in RPMI supplemented with 10% FBS, 100 ng/mL GM-CSF and 40 ng/mL IL-4.'
                ],
                extract_protocol_ch1: [
                  'Total RNA was isolated using Qiagen RNeasy columns.'
                ],
                treatment_protocol_ch1: [
                  'MoDCs were stimulated with 15 ng/mL LPS for 2.5 h or 5 h, influenza virus for 10 h, 100 U/mL IFN for 6.5 h, or left unstimulated.'
                ],
                'contact_zip/postal_code': ['02143']
              },
              is_ccdl: false,
              created_at: '2019-09-27T06:44:14.514941Z',
              last_modified: '2019-09-27T06:44:14.514941Z'
            }
          ],
          results: [
            {
              id: 2655552,
              processor: {
                id: 464,
                name: 'Submitter-processed',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_no_op',
                environment: {
                  R: {
                    dplyr: '0.7.8',
                    rlang: '0.3.1',
                    Bioconductor: '3.5',
                    AnnotationDbi: '1.40.0',
                    'illuminaRatv1.db': '1.26.0',
                    'illuminaHumanv1.db': '1.26.0',
                    'illuminaHumanv2.db': '1.26.0',
                    'illuminaHumanv3.db': '1.26.0',
                    'illuminaHumanv4.db': '1.26.0',
                    'illuminaMousev1.db': '1.26.0',
                    'illuminaMousev2.db': '1.26.0',
                    'illuminaMousev1p1.db': '1.26.0'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'cat /etc/identifier_refinery_url':
                      'https://zenodo.org/record/1410647/files/all_1536267482.zip'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: false,
          sex: 'female',
          age: '23.000',
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: 'caucasian',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-09-27T06:44:14.510013Z',
          last_modified: '2019-09-27T08:59:37.007294Z',
          original_files: [2714926, 2717627, 2721672, 2723604],
          computed_files: [4976410],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE57542']
        },
        {
          id: 525453,
          title: '9L [209516]',
          accession_code: 'GSM1281467',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hugene10st',
          platform_name: '[HuGene-1_0-st] Affymetrix Human Gene 1.0 ST Array',
          pretty_platform: 'Affymetrix Human Gene 1.0 ST Array (hugene10st)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1281467',
            'Scan protocol': [
              'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
            ],
            'Label protocol': [
              'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
            ],
            'Data processing': [
              'Data were processed using Bioconductor RMA package.',
              'probe group file: HuGene-1_0-st-v1.r4.pgf',
              'meta-probeset file: HuGene-1_0-st-v1.r4.mps'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using Qiagen RNeasy columns.'
            ],
            'Hybridization protocol': [
              'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
            ]
          },
          annotations: [
            {
              data: {
                type: ['RNA'],
                title: ['9L [209516]'],
                status: ['Public on Mar 01 2014'],
                relation: ['Reanalyzed by: GSE86357'],
                label_ch1: ['biotin'],
                series_id: ['GSE53166', 'GSE57542'],
                taxid_ch1: ['9606'],
                platform_id: ['GPL6244'],
                contact_city: ['Cambridge'],
                contact_name: ['Nir,,Hacohen'],
                hyb_protocol: [
                  'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
                ],
                molecule_ch1: ['total RNA'],
                organism_ch1: ['Homo sapiens'],
                channel_count: ['1'],
                contact_email: ['nhacohen@partners.org'],
                contact_state: ['MA'],
                geo_accession: ['GSM1281467'],
                scan_protocol: [
                  'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
                ],
                data_row_count: ['26981'],
                contact_address: ['7 Cambridge Center'],
                contact_country: ['USA'],
                data_processing: [
                  'Data were processed using Bioconductor RMA package.',
                  'probe group file: HuGene-1_0-st-v1.r4.pgf',
                  'meta-probeset file: HuGene-1_0-st-v1.r4.mps'
                ],
                source_name_ch1: ['9L'],
                submission_date: ['Dec 06 2013'],
                last_update_date: ['Sep 01 2016'],
                contact_institute: ['Broad Institute'],
                label_protocol_ch1: [
                  'cDNA synthesis, labeling, hybridization, and scanning were performed by the company Expression Analysis (Durham, NC).'
                ],
                supplementary_file: [
                  'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1281nnn/GSM1281467/suppl/GSM1281467_EA10047_209516_HUGENE-1_0-ST-V1_9L.CEL.gz'
                ],
                characteristics_ch1: [
                  'stimulation: LPS',
                  'total rna (ug): 7.851',
                  'rna concentration (ug/ul): 0.28',
                  'cd4 %: 94.6',
                  'cd16 %: 0.5',
                  'age (years): 40',
                  'million modc: 0.191800172',
                  'million monocyte: 1.38985632',
                  'million pbmc: 43.64778',
                  'ethnicity: Asian',
                  'Sex: M',
                  'individual id: IGTB1043'
                ],
                growth_protocol_ch1: [
                  'Monocyte-derived dendritic cells from each donor were cultured for 7 days in RPMI supplemented with 10% FBS, 100 ng/mL GM-CSF and 40 ng/mL IL-4.'
                ],
                extract_protocol_ch1: [
                  'Total RNA was isolated using Qiagen RNeasy columns.'
                ],
                treatment_protocol_ch1: [
                  'MoDCs were stimulated with 15 ng/mL LPS for 5 h, influenza virus for 10 h, or left unstimulated.'
                ],
                'contact_zip/postal_code': ['02143']
              },
              is_ccdl: false,
              created_at: '2018-12-14T00:10:45.887948Z',
              last_modified: '2018-12-14T00:10:45.887948Z'
            }
          ],
          results: [
            {
              id: 1489679,
              processor: {
                id: 321,
                name: 'Affymetrix SCAN',
                version: 'v1.22.14-hotfix',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.26.0',
                    affy: '1.56.0',
                    limma: '3.34.9',
                    oligo: '1.42.0',
                    affyio: '1.48.0',
                    'SCAN.UPC': '2.20.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.5',
                    AnnotationDbi: '1.40.0'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    rpy2: '2.9.5',
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.22.14-hotfix'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: true,
          sex: 'male',
          age: '40.000',
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: 'asian',
          subject: 'igtb1043',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-14T00:10:32.295762Z',
          last_modified: '2019-08-06T20:17:09.203146Z',
          original_files: [812649, 1874481, 2722540],
          computed_files: [2645193],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE53166', 'GSE57542']
        },
        {
          id: 2010537,
          title: '0494V031011_unstim_20121210_n154_01_06.RCC',
          accession_code: 'GSM1285196',
          source_database: 'GEO',
          organism: {
            name: 'HOMO_SAPIENS',
            taxonomy_id: 9606
          },
          platform_accession_code: 'hugene10st',
          platform_name: '[HuGene-1_0-st] Affymetrix Human Gene 1.0 ST Array',
          pretty_platform: 'Affymetrix Human Gene 1.0 ST Array (hugene10st)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: {
            Reference:
              'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1285196',
            'Scan protocol': [
              'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
            ],
            'Label protocol': ['none'],
            'Data processing': [
              'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
            ],
            'Extraction protocol': [
              'Total RNA was isolated using Qiagen RNeasy columns.'
            ],
            'Hybridization protocol': [
              'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
            ]
          },
          annotations: [
            {
              data: {
                type: ['RNA'],
                title: ['0494V031011_unstim_20121210_n154_01_06.RCC'],
                status: ['Public on Mar 01 2014'],
                label_ch1: ['n/a'],
                series_id: ['GSE53165', 'GSE57542'],
                taxid_ch1: ['9606'],
                platform_id: ['GPL18032'],
                contact_city: ['Cambridge'],
                contact_name: ['Nir,,Hacohen'],
                hyb_protocol: [
                  'Lysates in Qiagen RLT buffer were hybridized for 12-24 hours with custom nCounter Gene Expression CodeSets.'
                ],
                molecule_ch1: ['total RNA'],
                organism_ch1: ['Homo sapiens'],
                channel_count: ['1'],
                contact_email: ['nhacohen@partners.org'],
                contact_state: ['MA'],
                geo_accession: ['GSM1285196'],
                scan_protocol: [
                  'Nanostring nCounter system (Nanostring; Seattle, WA) was used to digitally count transcripts in a multiplex reaction.'
                ],
                data_row_count: ['414'],
                contact_address: ['7 Cambridge Center'],
                contact_country: ['USA'],
                data_processing: [
                  'The raw data is first scaled by positive spike-ins, shifted by negative spike-ins, and a negative binomial is fit using DESeq based on the scale parameter estimated from the positive spike-ins'
                ],
                source_name_ch1: ['0494V031011'],
                submission_date: ['Dec 10 2013'],
                last_update_date: ['Mar 01 2014'],
                contact_institute: ['Broad Institute'],
                label_protocol_ch1: ['none'],
                supplementary_file: [
                  'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1285nnn/GSM1285196/suppl/GSM1285196_20121210_n154_01_06.RCC.gz'
                ],
                characteristics_ch1: [
                  'stimulation: unstim',
                  'hours: 0',
                  'plate: U1',
                  'cartridge: n154',
                  'lane: 6',
                  'date of visit: 2011-03-10',
                  'time of visit: 8:11 AM',
                  'gender: F',
                  'age (years): 25',
                  'ethnicity: Caucasian',
                  'million pbmc: 21.1315845',
                  'cd14 %: 88.5',
                  'cd16 %: 1.2',
                  'million dcs: 0.1599',
                  'height (cm): 168.5',
                  'weight (kg): 48.75',
                  'systolic/diastolic: 92/60',
                  'bmi: 17.17017848'
                ],
                growth_protocol_ch1: [
                  'Monocyte-derived dendritic cells from each donor were cultured for 7 days in RPMI supplemented with 10% FBS, 100 ng/mL GM-CSF and 40 ng/mL IL-4.'
                ],
                extract_protocol_ch1: [
                  'Total RNA was isolated using Qiagen RNeasy columns.'
                ],
                treatment_protocol_ch1: [
                  'MoDCs were stimulated with 15 ng/mL LPS for 2.5 h or 5 h, influenza virus for 10 h, 100 U/mL IFN for 6.5 h, or left unstimulated.'
                ],
                'contact_zip/postal_code': ['02143']
              },
              is_ccdl: false,
              created_at: '2019-09-27T06:44:05.584589Z',
              last_modified: '2019-09-27T06:44:05.584589Z'
            }
          ],
          results: [
            {
              id: 2655579,
              processor: {
                id: 464,
                name: 'Submitter-processed',
                version: 'v1.27.16-hotfix',
                docker_image: 'dr_no_op',
                environment: {
                  R: {
                    dplyr: '0.7.8',
                    rlang: '0.3.1',
                    Bioconductor: '3.5',
                    AnnotationDbi: '1.40.0',
                    'illuminaRatv1.db': '1.26.0',
                    'illuminaHumanv1.db': '1.26.0',
                    'illuminaHumanv2.db': '1.26.0',
                    'illuminaHumanv3.db': '1.26.0',
                    'illuminaHumanv4.db': '1.26.0',
                    'illuminaMousev1.db': '1.26.0',
                    'illuminaMousev2.db': '1.26.0',
                    'illuminaMousev1p1.db': '1.26.0'
                  },
                  os_pkg: {
                    'r-base': 'r-base',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    Django: '2.1.8',
                    'data-refinery-common': '=v1.27.16-hotfix'
                  },
                  cmd_line: {
                    'cat /etc/identifier_refinery_url':
                      'https://zenodo.org/record/1410647/files/all_1536267482.zip'
                  },
                  os_distribution: 'Ubuntu 16.04.6 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url: '',
          has_raw: false,
          sex: 'female',
          age: '25.000',
          specimen_part: '',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: 'caucasian',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2019-09-27T06:44:05.579565Z',
          last_modified: '2019-09-27T09:00:01.587249Z',
          original_files: [2714880, 2717627, 2721724, 2724063],
          computed_files: [4976438],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['GSE57542']
        }
      ]
    }
  ],
  [
    {
      count: 32,
      next: 'http://api.refine.bio/v1/samples/?experiment_accession_code=E-MTAB-1811&limit=10&offset=10',
      previous: null,
      results: [
        {
          id: 668387,
          title: 'Sample2 RNA',
          accession_code: 'E-MTAB-1811-Sample2',
          source_database: 'ARRAY_EXPRESS',
          organism: {
            name: 'LEPIDIUM_SATIVUM',
            taxonomy_id: 33125
          },
          platform_accession_code: 'ath1121501',
          platform_name:
            '[ATH1-121501] Affymetrix Arabidopsis ATH1 Genome Array',
          pretty_platform:
            'Affymetrix Arabidopsis ATH1 Genome Array (ath1121501)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: [
            {
              Text: 'P-AFFY-3 GeneChip hybridization',
              Type: 'nucleic acid hybridization to array protocol',
              Accession: 'P-AFFY-3',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Title: Affymetrix CEL analysis. Description:',
              Type: 'feature_extraction',
              Accession: 'P-AFFY-6',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "After-ripened Lepidium sativum L. FR1 ('Gartenkresse, einfache') seeds (Juliwa, Heidelberg, Germany) were incubated in petri dishes on two layers of filter paper with 6 ml 1/10 Murashige-Skoog salts in continuous white light (ca. 100 µmol•s-1•m-2) at the temperature indicated.",
              Type: 'growth protocol',
              Accession: 'P-MTAB-34006',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Four biological replicates of seeds were sampled after 10h of imbibition at 11 degrees C, 18 degrees C, 27 degrees C and 32 degrees C.  Two tissues (micropylar endosperm cap and radicle) were dissected from individual seeds. The dissected tissues (ca. 200 endosperm caps (Cap), and ca. 50 radicles (Rad)) were pooled for each sample, frozen in liquid nitrogen and stored at -80 ºC.',
              Type: 'treatment protocol',
              Accession: 'P-MTAB-34007',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'CTAB/Chloroform extraction followed by RNA clean-up with the Qiagen Rneasy kit, followed by quantity and quality controls',
              Type: 'nucleic acid extraction protocol',
              Accession: 'P-MTAB-34008',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "The Affymetrix 3' IVT-Express Labeling Kit (#901229) was used to synthesize Biotin-labeled cRNA. From each RNA sample 100 ng was used as input for the labeling reactions.",
              Type: 'nucleic acid labeling protocol',
              Accession: 'P-MTAB-34009',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            }
          ],
          annotations: [
            {
              data: {
                file: [
                  {
                    url: 'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip/101650-073_51-11degreesCAP.CEL',
                    name: '101650-073_51-11degreesCAP.CEL',
                    type: 'data',
                    comment: {
                      name: 'ArrayExpress FTP file',
                      value:
                        'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip'
                    }
                  }
                ],
                assay: {
                  name: 'Assay 2'
                },
                source: {
                  name: 'Sample2'
                },
                extract: {
                  name: 'Sample2 RNA'
                },
                variable: [
                  {
                    name: 'temperature',
                    unit: 'degree celsius',
                    value: 11
                  },
                  {
                    name: 'organism part',
                    value: 'micropylar endosperm'
                  }
                ],
                characteristic: [
                  {
                    value: 'Lepidium sativum',
                    category: 'organism'
                  },
                  {
                    value: 'seed',
                    category: 'developmental stage'
                  },
                  {
                    value: 'fresh_sample',
                    category: 'specimen with known storage state'
                  },
                  {
                    value: 'micropylar endosperm',
                    category: 'organism part'
                  },
                  {
                    unit: 'degree celsius',
                    value: 11,
                    category: 'temperature'
                  }
                ],
                'labeled-extract': {
                  name: 'Sample2 labeled_extract',
                  label: 'biotin'
                }
              },
              is_ccdl: false,
              created_at: '2018-12-19T15:45:44.678041Z',
              last_modified: '2018-12-19T15:45:44.678041Z'
            }
          ],
          results: [
            {
              id: 793900,
              processor: {
                id: 99,
                name: 'Affymetrix SCAN',
                version: 'v1.4.7',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.26.0',
                    affy: '1.56.0',
                    limma: '3.34.9',
                    oligo: '1.42.0',
                    affyio: '1.48.0',
                    'SCAN.UPC': '2.20.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.6',
                    AnnotationDbi: '1.40.0'
                  },
                  os_pkg: {
                    'r-base': '3.4.4-1xenial0',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    rpy2: '2.9.5',
                    Django: '2.1.2',
                    'data-refinery-common': '1.4.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url:
            'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/samples',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'micropylar endosperm',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T15:45:43.241884Z',
          last_modified: '2018-12-19T20:12:12.895784Z',
          original_files: [1091778],
          computed_files: [909757],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['E-MTAB-1811']
        },
        {
          id: 669107,
          title: 'Sample3 RNA',
          accession_code: 'E-MTAB-1811-Sample3',
          source_database: 'ARRAY_EXPRESS',
          organism: {
            name: 'LEPIDIUM_SATIVUM',
            taxonomy_id: 33125
          },
          platform_accession_code: 'ath1121501',
          platform_name:
            '[ATH1-121501] Affymetrix Arabidopsis ATH1 Genome Array',
          pretty_platform:
            'Affymetrix Arabidopsis ATH1 Genome Array (ath1121501)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: [
            {
              Text: 'P-AFFY-3 GeneChip hybridization',
              Type: 'nucleic acid hybridization to array protocol',
              Accession: 'P-AFFY-3',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Title: Affymetrix CEL analysis. Description:',
              Type: 'feature_extraction',
              Accession: 'P-AFFY-6',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "After-ripened Lepidium sativum L. FR1 ('Gartenkresse, einfache') seeds (Juliwa, Heidelberg, Germany) were incubated in petri dishes on two layers of filter paper with 6 ml 1/10 Murashige-Skoog salts in continuous white light (ca. 100 µmol•s-1•m-2) at the temperature indicated.",
              Type: 'growth protocol',
              Accession: 'P-MTAB-34006',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Four biological replicates of seeds were sampled after 10h of imbibition at 11 degrees C, 18 degrees C, 27 degrees C and 32 degrees C.  Two tissues (micropylar endosperm cap and radicle) were dissected from individual seeds. The dissected tissues (ca. 200 endosperm caps (Cap), and ca. 50 radicles (Rad)) were pooled for each sample, frozen in liquid nitrogen and stored at -80 ºC.',
              Type: 'treatment protocol',
              Accession: 'P-MTAB-34007',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'CTAB/Chloroform extraction followed by RNA clean-up with the Qiagen Rneasy kit, followed by quantity and quality controls',
              Type: 'nucleic acid extraction protocol',
              Accession: 'P-MTAB-34008',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "The Affymetrix 3' IVT-Express Labeling Kit (#901229) was used to synthesize Biotin-labeled cRNA. From each RNA sample 100 ng was used as input for the labeling reactions.",
              Type: 'nucleic acid labeling protocol',
              Accession: 'P-MTAB-34009',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            }
          ],
          annotations: [
            {
              data: {
                file: [
                  {
                    url: 'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip/101650-074_52-11degreesCAP.CEL',
                    name: '101650-074_52-11degreesCAP.CEL',
                    type: 'data',
                    comment: {
                      name: 'ArrayExpress FTP file',
                      value:
                        'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip'
                    }
                  }
                ],
                assay: {
                  name: 'Assay 3'
                },
                source: {
                  name: 'Sample3'
                },
                extract: {
                  name: 'Sample3 RNA'
                },
                variable: [
                  {
                    name: 'temperature',
                    unit: 'degree celsius',
                    value: 11
                  },
                  {
                    name: 'organism part',
                    value: 'micropylar endosperm'
                  }
                ],
                characteristic: [
                  {
                    value: 'Lepidium sativum',
                    category: 'organism'
                  },
                  {
                    value: 'seed',
                    category: 'developmental stage'
                  },
                  {
                    value: 'fresh_sample',
                    category: 'specimen with known storage state'
                  },
                  {
                    value: 'micropylar endosperm',
                    category: 'organism part'
                  },
                  {
                    unit: 'degree celsius',
                    value: 11,
                    category: 'temperature'
                  }
                ],
                'labeled-extract': {
                  name: 'Sample3 labeled_extract',
                  label: 'biotin'
                }
              },
              is_ccdl: false,
              created_at: '2018-12-19T15:45:54.001593Z',
              last_modified: '2018-12-19T15:45:54.001593Z'
            }
          ],
          results: [
            {
              id: 794056,
              processor: {
                id: 99,
                name: 'Affymetrix SCAN',
                version: 'v1.4.7',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.26.0',
                    affy: '1.56.0',
                    limma: '3.34.9',
                    oligo: '1.42.0',
                    affyio: '1.48.0',
                    'SCAN.UPC': '2.20.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.6',
                    AnnotationDbi: '1.40.0'
                  },
                  os_pkg: {
                    'r-base': '3.4.4-1xenial0',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    rpy2: '2.9.5',
                    Django: '2.1.2',
                    'data-refinery-common': '1.4.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url:
            'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/samples',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'micropylar endosperm',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T15:45:51.318361Z',
          last_modified: '2018-12-19T20:12:33.127522Z',
          original_files: [1092282],
          computed_files: [909913],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['E-MTAB-1811']
        },
        {
          id: 669802,
          title: 'Sample4 RNA',
          accession_code: 'E-MTAB-1811-Sample4',
          source_database: 'ARRAY_EXPRESS',
          organism: {
            name: 'LEPIDIUM_SATIVUM',
            taxonomy_id: 33125
          },
          platform_accession_code: 'ath1121501',
          platform_name:
            '[ATH1-121501] Affymetrix Arabidopsis ATH1 Genome Array',
          pretty_platform:
            'Affymetrix Arabidopsis ATH1 Genome Array (ath1121501)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: [
            {
              Text: 'P-AFFY-3 GeneChip hybridization',
              Type: 'nucleic acid hybridization to array protocol',
              Accession: 'P-AFFY-3',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Title: Affymetrix CEL analysis. Description:',
              Type: 'feature_extraction',
              Accession: 'P-AFFY-6',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "After-ripened Lepidium sativum L. FR1 ('Gartenkresse, einfache') seeds (Juliwa, Heidelberg, Germany) were incubated in petri dishes on two layers of filter paper with 6 ml 1/10 Murashige-Skoog salts in continuous white light (ca. 100 µmol•s-1•m-2) at the temperature indicated.",
              Type: 'growth protocol',
              Accession: 'P-MTAB-34006',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Four biological replicates of seeds were sampled after 10h of imbibition at 11 degrees C, 18 degrees C, 27 degrees C and 32 degrees C.  Two tissues (micropylar endosperm cap and radicle) were dissected from individual seeds. The dissected tissues (ca. 200 endosperm caps (Cap), and ca. 50 radicles (Rad)) were pooled for each sample, frozen in liquid nitrogen and stored at -80 ºC.',
              Type: 'treatment protocol',
              Accession: 'P-MTAB-34007',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'CTAB/Chloroform extraction followed by RNA clean-up with the Qiagen Rneasy kit, followed by quantity and quality controls',
              Type: 'nucleic acid extraction protocol',
              Accession: 'P-MTAB-34008',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "The Affymetrix 3' IVT-Express Labeling Kit (#901229) was used to synthesize Biotin-labeled cRNA. From each RNA sample 100 ng was used as input for the labeling reactions.",
              Type: 'nucleic acid labeling protocol',
              Accession: 'P-MTAB-34009',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            }
          ],
          annotations: [
            {
              data: {
                file: [
                  {
                    url: 'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip/101650-075_53-11degreesCAP.CEL',
                    name: '101650-075_53-11degreesCAP.CEL',
                    type: 'data',
                    comment: {
                      name: 'ArrayExpress FTP file',
                      value:
                        'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip'
                    }
                  }
                ],
                assay: {
                  name: 'Assay 4'
                },
                source: {
                  name: 'Sample4'
                },
                extract: {
                  name: 'Sample4 RNA'
                },
                variable: [
                  {
                    name: 'temperature',
                    unit: 'degree celsius',
                    value: 11
                  },
                  {
                    name: 'organism part',
                    value: 'micropylar endosperm'
                  }
                ],
                characteristic: [
                  {
                    value: 'Lepidium sativum',
                    category: 'organism'
                  },
                  {
                    value: 'seed',
                    category: 'developmental stage'
                  },
                  {
                    value: 'fresh_sample',
                    category: 'specimen with known storage state'
                  },
                  {
                    value: 'micropylar endosperm',
                    category: 'organism part'
                  },
                  {
                    unit: 'degree celsius',
                    value: 11,
                    category: 'temperature'
                  }
                ],
                'labeled-extract': {
                  name: 'Sample4 labeled_extract',
                  label: 'biotin'
                }
              },
              is_ccdl: false,
              created_at: '2018-12-19T15:46:02.657613Z',
              last_modified: '2018-12-19T15:46:02.657613Z'
            }
          ],
          results: [
            {
              id: 794216,
              processor: {
                id: 99,
                name: 'Affymetrix SCAN',
                version: 'v1.4.7',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.26.0',
                    affy: '1.56.0',
                    limma: '3.34.9',
                    oligo: '1.42.0',
                    affyio: '1.48.0',
                    'SCAN.UPC': '2.20.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.6',
                    AnnotationDbi: '1.40.0'
                  },
                  os_pkg: {
                    'r-base': '3.4.4-1xenial0',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    rpy2: '2.9.5',
                    Django: '2.1.2',
                    'data-refinery-common': '1.4.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url:
            'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/samples',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'micropylar endosperm',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T15:46:00.884893Z',
          last_modified: '2018-12-19T20:12:51.155335Z',
          original_files: [1092745],
          computed_files: [910073],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['E-MTAB-1811']
        },
        {
          id: 670516,
          title: 'Sample5 RNA',
          accession_code: 'E-MTAB-1811-Sample5',
          source_database: 'ARRAY_EXPRESS',
          organism: {
            name: 'LEPIDIUM_SATIVUM',
            taxonomy_id: 33125
          },
          platform_accession_code: 'ath1121501',
          platform_name:
            '[ATH1-121501] Affymetrix Arabidopsis ATH1 Genome Array',
          pretty_platform:
            'Affymetrix Arabidopsis ATH1 Genome Array (ath1121501)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: [
            {
              Text: 'P-AFFY-3 GeneChip hybridization',
              Type: 'nucleic acid hybridization to array protocol',
              Accession: 'P-AFFY-3',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Title: Affymetrix CEL analysis. Description:',
              Type: 'feature_extraction',
              Accession: 'P-AFFY-6',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "After-ripened Lepidium sativum L. FR1 ('Gartenkresse, einfache') seeds (Juliwa, Heidelberg, Germany) were incubated in petri dishes on two layers of filter paper with 6 ml 1/10 Murashige-Skoog salts in continuous white light (ca. 100 µmol•s-1•m-2) at the temperature indicated.",
              Type: 'growth protocol',
              Accession: 'P-MTAB-34006',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Four biological replicates of seeds were sampled after 10h of imbibition at 11 degrees C, 18 degrees C, 27 degrees C and 32 degrees C.  Two tissues (micropylar endosperm cap and radicle) were dissected from individual seeds. The dissected tissues (ca. 200 endosperm caps (Cap), and ca. 50 radicles (Rad)) were pooled for each sample, frozen in liquid nitrogen and stored at -80 ºC.',
              Type: 'treatment protocol',
              Accession: 'P-MTAB-34007',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'CTAB/Chloroform extraction followed by RNA clean-up with the Qiagen Rneasy kit, followed by quantity and quality controls',
              Type: 'nucleic acid extraction protocol',
              Accession: 'P-MTAB-34008',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "The Affymetrix 3' IVT-Express Labeling Kit (#901229) was used to synthesize Biotin-labeled cRNA. From each RNA sample 100 ng was used as input for the labeling reactions.",
              Type: 'nucleic acid labeling protocol',
              Accession: 'P-MTAB-34009',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            }
          ],
          annotations: [
            {
              data: {
                file: [
                  {
                    url: 'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip/101650-076_55-11degreesRAD.CEL',
                    name: '101650-076_55-11degreesRAD.CEL',
                    type: 'data',
                    comment: {
                      name: 'ArrayExpress FTP file',
                      value:
                        'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip'
                    }
                  }
                ],
                assay: {
                  name: 'Assay 5'
                },
                source: {
                  name: 'Sample5'
                },
                extract: {
                  name: 'Sample5 RNA'
                },
                variable: [
                  {
                    name: 'temperature',
                    unit: 'degree celsius',
                    value: 11
                  },
                  {
                    name: 'organism part',
                    value: 'radicle'
                  }
                ],
                characteristic: [
                  {
                    value: 'Lepidium sativum',
                    category: 'organism'
                  },
                  {
                    value: 'seed',
                    category: 'developmental stage'
                  },
                  {
                    value: 'fresh_sample',
                    category: 'specimen with known storage state'
                  },
                  {
                    value: 'radicle',
                    category: 'organism part'
                  },
                  {
                    unit: 'degree celsius',
                    value: 11,
                    category: 'temperature'
                  }
                ],
                'labeled-extract': {
                  name: 'Sample5 labeled_extract',
                  label: 'biotin'
                }
              },
              is_ccdl: false,
              created_at: '2018-12-19T15:46:12.481578Z',
              last_modified: '2018-12-19T15:46:12.481578Z'
            }
          ],
          results: [
            {
              id: 794394,
              processor: {
                id: 99,
                name: 'Affymetrix SCAN',
                version: 'v1.4.7',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.26.0',
                    affy: '1.56.0',
                    limma: '3.34.9',
                    oligo: '1.42.0',
                    affyio: '1.48.0',
                    'SCAN.UPC': '2.20.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.6',
                    AnnotationDbi: '1.40.0'
                  },
                  os_pkg: {
                    'r-base': '3.4.4-1xenial0',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    rpy2: '2.9.5',
                    Django: '2.1.2',
                    'data-refinery-common': '1.4.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url:
            'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/samples',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'radicle',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T15:46:10.414968Z',
          last_modified: '2018-12-19T20:13:12.608761Z',
          original_files: [1093220],
          computed_files: [910251],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['E-MTAB-1811']
        },
        {
          id: 672445,
          title: 'Sample8 RNA',
          accession_code: 'E-MTAB-1811-Sample8',
          source_database: 'ARRAY_EXPRESS',
          organism: {
            name: 'LEPIDIUM_SATIVUM',
            taxonomy_id: 33125
          },
          platform_accession_code: 'ath1121501',
          platform_name:
            '[ATH1-121501] Affymetrix Arabidopsis ATH1 Genome Array',
          pretty_platform:
            'Affymetrix Arabidopsis ATH1 Genome Array (ath1121501)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: [
            {
              Text: 'P-AFFY-3 GeneChip hybridization',
              Type: 'nucleic acid hybridization to array protocol',
              Accession: 'P-AFFY-3',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Title: Affymetrix CEL analysis. Description:',
              Type: 'feature_extraction',
              Accession: 'P-AFFY-6',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "After-ripened Lepidium sativum L. FR1 ('Gartenkresse, einfache') seeds (Juliwa, Heidelberg, Germany) were incubated in petri dishes on two layers of filter paper with 6 ml 1/10 Murashige-Skoog salts in continuous white light (ca. 100 µmol•s-1•m-2) at the temperature indicated.",
              Type: 'growth protocol',
              Accession: 'P-MTAB-34006',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Four biological replicates of seeds were sampled after 10h of imbibition at 11 degrees C, 18 degrees C, 27 degrees C and 32 degrees C.  Two tissues (micropylar endosperm cap and radicle) were dissected from individual seeds. The dissected tissues (ca. 200 endosperm caps (Cap), and ca. 50 radicles (Rad)) were pooled for each sample, frozen in liquid nitrogen and stored at -80 ºC.',
              Type: 'treatment protocol',
              Accession: 'P-MTAB-34007',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'CTAB/Chloroform extraction followed by RNA clean-up with the Qiagen Rneasy kit, followed by quantity and quality controls',
              Type: 'nucleic acid extraction protocol',
              Accession: 'P-MTAB-34008',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "The Affymetrix 3' IVT-Express Labeling Kit (#901229) was used to synthesize Biotin-labeled cRNA. From each RNA sample 100 ng was used as input for the labeling reactions.",
              Type: 'nucleic acid labeling protocol',
              Accession: 'P-MTAB-34009',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            }
          ],
          annotations: [
            {
              data: {
                file: [
                  {
                    url: 'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip/101650-105_Repl-11degreesRAD.CEL',
                    name: '101650-105_Repl-11degreesRAD.CEL',
                    type: 'data',
                    comment: {
                      name: 'ArrayExpress FTP file',
                      value:
                        'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip'
                    }
                  }
                ],
                assay: {
                  name: 'Assay 8'
                },
                source: {
                  name: 'Sample8'
                },
                extract: {
                  name: 'Sample8 RNA'
                },
                variable: [
                  {
                    name: 'temperature',
                    unit: 'degree celsius',
                    value: 11
                  },
                  {
                    name: 'organism part',
                    value: 'radicle'
                  }
                ],
                characteristic: [
                  {
                    value: 'Lepidium sativum',
                    category: 'organism'
                  },
                  {
                    value: 'seed',
                    category: 'developmental stage'
                  },
                  {
                    value: 'fresh_sample',
                    category: 'specimen with known storage state'
                  },
                  {
                    value: 'radicle',
                    category: 'organism part'
                  },
                  {
                    unit: 'degree celsius',
                    value: 11,
                    category: 'temperature'
                  }
                ],
                'labeled-extract': {
                  name: 'Sample8 labeled_extract',
                  label: 'biotin'
                }
              },
              is_ccdl: false,
              created_at: '2018-12-19T15:46:39.105757Z',
              last_modified: '2018-12-19T15:46:39.105757Z'
            }
          ],
          results: [
            {
              id: 794883,
              processor: {
                id: 99,
                name: 'Affymetrix SCAN',
                version: 'v1.4.7',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.26.0',
                    affy: '1.56.0',
                    limma: '3.34.9',
                    oligo: '1.42.0',
                    affyio: '1.48.0',
                    'SCAN.UPC': '2.20.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.6',
                    AnnotationDbi: '1.40.0'
                  },
                  os_pkg: {
                    'r-base': '3.4.4-1xenial0',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    rpy2: '2.9.5',
                    Django: '2.1.2',
                    'data-refinery-common': '1.4.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url:
            'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/samples',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'radicle',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T15:46:37.050771Z',
          last_modified: '2018-12-19T20:14:07.933012Z',
          original_files: [1094492],
          computed_files: [910740],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['E-MTAB-1811']
        },
        {
          id: 671182,
          title: 'Sample6 RNA',
          accession_code: 'E-MTAB-1811-Sample6',
          source_database: 'ARRAY_EXPRESS',
          organism: {
            name: 'LEPIDIUM_SATIVUM',
            taxonomy_id: 33125
          },
          platform_accession_code: 'ath1121501',
          platform_name:
            '[ATH1-121501] Affymetrix Arabidopsis ATH1 Genome Array',
          pretty_platform:
            'Affymetrix Arabidopsis ATH1 Genome Array (ath1121501)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: [
            {
              Text: 'P-AFFY-3 GeneChip hybridization',
              Type: 'nucleic acid hybridization to array protocol',
              Accession: 'P-AFFY-3',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Title: Affymetrix CEL analysis. Description:',
              Type: 'feature_extraction',
              Accession: 'P-AFFY-6',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "After-ripened Lepidium sativum L. FR1 ('Gartenkresse, einfache') seeds (Juliwa, Heidelberg, Germany) were incubated in petri dishes on two layers of filter paper with 6 ml 1/10 Murashige-Skoog salts in continuous white light (ca. 100 µmol•s-1•m-2) at the temperature indicated.",
              Type: 'growth protocol',
              Accession: 'P-MTAB-34006',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Four biological replicates of seeds were sampled after 10h of imbibition at 11 degrees C, 18 degrees C, 27 degrees C and 32 degrees C.  Two tissues (micropylar endosperm cap and radicle) were dissected from individual seeds. The dissected tissues (ca. 200 endosperm caps (Cap), and ca. 50 radicles (Rad)) were pooled for each sample, frozen in liquid nitrogen and stored at -80 ºC.',
              Type: 'treatment protocol',
              Accession: 'P-MTAB-34007',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'CTAB/Chloroform extraction followed by RNA clean-up with the Qiagen Rneasy kit, followed by quantity and quality controls',
              Type: 'nucleic acid extraction protocol',
              Accession: 'P-MTAB-34008',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "The Affymetrix 3' IVT-Express Labeling Kit (#901229) was used to synthesize Biotin-labeled cRNA. From each RNA sample 100 ng was used as input for the labeling reactions.",
              Type: 'nucleic acid labeling protocol',
              Accession: 'P-MTAB-34009',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            }
          ],
          annotations: [
            {
              data: {
                file: [
                  {
                    url: 'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip/101650-077_56-11degreesRAD.CEL',
                    name: '101650-077_56-11degreesRAD.CEL',
                    type: 'data',
                    comment: {
                      name: 'ArrayExpress FTP file',
                      value:
                        'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip'
                    }
                  }
                ],
                assay: {
                  name: 'Assay 6'
                },
                source: {
                  name: 'Sample6'
                },
                extract: {
                  name: 'Sample6 RNA'
                },
                variable: [
                  {
                    name: 'temperature',
                    unit: 'degree celsius',
                    value: 11
                  },
                  {
                    name: 'organism part',
                    value: 'radicle'
                  }
                ],
                characteristic: [
                  {
                    value: 'Lepidium sativum',
                    category: 'organism'
                  },
                  {
                    value: 'seed',
                    category: 'developmental stage'
                  },
                  {
                    value: 'fresh_sample',
                    category: 'specimen with known storage state'
                  },
                  {
                    value: 'radicle',
                    category: 'organism part'
                  },
                  {
                    unit: 'degree celsius',
                    value: 11,
                    category: 'temperature'
                  }
                ],
                'labeled-extract': {
                  name: 'Sample6 labeled_extract',
                  label: 'biotin'
                }
              },
              is_ccdl: false,
              created_at: '2018-12-19T15:46:20.531934Z',
              last_modified: '2018-12-19T15:46:20.531934Z'
            }
          ],
          results: [
            {
              id: 794603,
              processor: {
                id: 99,
                name: 'Affymetrix SCAN',
                version: 'v1.4.7',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.26.0',
                    affy: '1.56.0',
                    limma: '3.34.9',
                    oligo: '1.42.0',
                    affyio: '1.48.0',
                    'SCAN.UPC': '2.20.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.6',
                    AnnotationDbi: '1.40.0'
                  },
                  os_pkg: {
                    'r-base': '3.4.4-1xenial0',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    rpy2: '2.9.5',
                    Django: '2.1.2',
                    'data-refinery-common': '1.4.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url:
            'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/samples',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'radicle',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T15:46:19.618987Z',
          last_modified: '2018-12-19T20:13:35.079999Z',
          original_files: [1093665],
          computed_files: [910460],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['E-MTAB-1811']
        },
        {
          id: 671811,
          title: 'Sample7 RNA',
          accession_code: 'E-MTAB-1811-Sample7',
          source_database: 'ARRAY_EXPRESS',
          organism: {
            name: 'LEPIDIUM_SATIVUM',
            taxonomy_id: 33125
          },
          platform_accession_code: 'ath1121501',
          platform_name:
            '[ATH1-121501] Affymetrix Arabidopsis ATH1 Genome Array',
          pretty_platform:
            'Affymetrix Arabidopsis ATH1 Genome Array (ath1121501)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: [
            {
              Text: 'P-AFFY-3 GeneChip hybridization',
              Type: 'nucleic acid hybridization to array protocol',
              Accession: 'P-AFFY-3',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Title: Affymetrix CEL analysis. Description:',
              Type: 'feature_extraction',
              Accession: 'P-AFFY-6',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "After-ripened Lepidium sativum L. FR1 ('Gartenkresse, einfache') seeds (Juliwa, Heidelberg, Germany) were incubated in petri dishes on two layers of filter paper with 6 ml 1/10 Murashige-Skoog salts in continuous white light (ca. 100 µmol•s-1•m-2) at the temperature indicated.",
              Type: 'growth protocol',
              Accession: 'P-MTAB-34006',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Four biological replicates of seeds were sampled after 10h of imbibition at 11 degrees C, 18 degrees C, 27 degrees C and 32 degrees C.  Two tissues (micropylar endosperm cap and radicle) were dissected from individual seeds. The dissected tissues (ca. 200 endosperm caps (Cap), and ca. 50 radicles (Rad)) were pooled for each sample, frozen in liquid nitrogen and stored at -80 ºC.',
              Type: 'treatment protocol',
              Accession: 'P-MTAB-34007',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'CTAB/Chloroform extraction followed by RNA clean-up with the Qiagen Rneasy kit, followed by quantity and quality controls',
              Type: 'nucleic acid extraction protocol',
              Accession: 'P-MTAB-34008',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "The Affymetrix 3' IVT-Express Labeling Kit (#901229) was used to synthesize Biotin-labeled cRNA. From each RNA sample 100 ng was used as input for the labeling reactions.",
              Type: 'nucleic acid labeling protocol',
              Accession: 'P-MTAB-34009',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            }
          ],
          annotations: [
            {
              data: {
                file: [
                  {
                    url: 'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip/101650-079_58-11degreesRAD.CEL',
                    name: '101650-079_58-11degreesRAD.CEL',
                    type: 'data',
                    comment: {
                      name: 'ArrayExpress FTP file',
                      value:
                        'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip'
                    }
                  }
                ],
                assay: {
                  name: 'Assay 7'
                },
                source: {
                  name: 'Sample7'
                },
                extract: {
                  name: 'Sample7 RNA'
                },
                variable: [
                  {
                    name: 'temperature',
                    unit: 'degree celsius',
                    value: 11
                  },
                  {
                    name: 'organism part',
                    value: 'radicle'
                  }
                ],
                characteristic: [
                  {
                    value: 'Lepidium sativum',
                    category: 'organism'
                  },
                  {
                    value: 'seed',
                    category: 'developmental stage'
                  },
                  {
                    value: 'fresh_sample',
                    category: 'specimen with known storage state'
                  },
                  {
                    value: 'radicle',
                    category: 'organism part'
                  },
                  {
                    unit: 'degree celsius',
                    value: 11,
                    category: 'temperature'
                  }
                ],
                'labeled-extract': {
                  name: 'Sample7 labeled_extract',
                  label: 'biotin'
                }
              },
              is_ccdl: false,
              created_at: '2018-12-19T15:46:29.666313Z',
              last_modified: '2018-12-19T15:46:29.666313Z'
            }
          ],
          results: [
            {
              id: 794710,
              processor: {
                id: 99,
                name: 'Affymetrix SCAN',
                version: 'v1.4.7',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.26.0',
                    affy: '1.56.0',
                    limma: '3.34.9',
                    oligo: '1.42.0',
                    affyio: '1.48.0',
                    'SCAN.UPC': '2.20.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.6',
                    AnnotationDbi: '1.40.0'
                  },
                  os_pkg: {
                    'r-base': '3.4.4-1xenial0',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    rpy2: '2.9.5',
                    Django: '2.1.2',
                    'data-refinery-common': '1.4.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url:
            'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/samples',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'radicle',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T15:46:27.691285Z',
          last_modified: '2018-12-19T20:13:49.611952Z',
          original_files: [1094089],
          computed_files: [910566],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['E-MTAB-1811']
        },
        {
          id: 673029,
          title: 'Sample9 RNA',
          accession_code: 'E-MTAB-1811-Sample9',
          source_database: 'ARRAY_EXPRESS',
          organism: {
            name: 'LEPIDIUM_SATIVUM',
            taxonomy_id: 33125
          },
          platform_accession_code: 'ath1121501',
          platform_name:
            '[ATH1-121501] Affymetrix Arabidopsis ATH1 Genome Array',
          pretty_platform:
            'Affymetrix Arabidopsis ATH1 Genome Array (ath1121501)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: [
            {
              Text: 'P-AFFY-3 GeneChip hybridization',
              Type: 'nucleic acid hybridization to array protocol',
              Accession: 'P-AFFY-3',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Title: Affymetrix CEL analysis. Description:',
              Type: 'feature_extraction',
              Accession: 'P-AFFY-6',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "After-ripened Lepidium sativum L. FR1 ('Gartenkresse, einfache') seeds (Juliwa, Heidelberg, Germany) were incubated in petri dishes on two layers of filter paper with 6 ml 1/10 Murashige-Skoog salts in continuous white light (ca. 100 µmol•s-1•m-2) at the temperature indicated.",
              Type: 'growth protocol',
              Accession: 'P-MTAB-34006',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Four biological replicates of seeds were sampled after 10h of imbibition at 11 degrees C, 18 degrees C, 27 degrees C and 32 degrees C.  Two tissues (micropylar endosperm cap and radicle) were dissected from individual seeds. The dissected tissues (ca. 200 endosperm caps (Cap), and ca. 50 radicles (Rad)) were pooled for each sample, frozen in liquid nitrogen and stored at -80 ºC.',
              Type: 'treatment protocol',
              Accession: 'P-MTAB-34007',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'CTAB/Chloroform extraction followed by RNA clean-up with the Qiagen Rneasy kit, followed by quantity and quality controls',
              Type: 'nucleic acid extraction protocol',
              Accession: 'P-MTAB-34008',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "The Affymetrix 3' IVT-Express Labeling Kit (#901229) was used to synthesize Biotin-labeled cRNA. From each RNA sample 100 ng was used as input for the labeling reactions.",
              Type: 'nucleic acid labeling protocol',
              Accession: 'P-MTAB-34009',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            }
          ],
          annotations: [
            {
              data: {
                file: [
                  {
                    url: 'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip/101650-080_59-18degreesCAP.CEL',
                    name: '101650-080_59-18degreesCAP.CEL',
                    type: 'data',
                    comment: {
                      name: 'ArrayExpress FTP file',
                      value:
                        'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip'
                    }
                  }
                ],
                assay: {
                  name: 'Assay 9'
                },
                source: {
                  name: 'Sample9'
                },
                extract: {
                  name: 'Sample9 RNA'
                },
                variable: [
                  {
                    name: 'temperature',
                    unit: 'degree celsius',
                    value: 18
                  },
                  {
                    name: 'organism part',
                    value: 'micropylar endosperm'
                  }
                ],
                characteristic: [
                  {
                    value: 'Lepidium sativum',
                    category: 'organism'
                  },
                  {
                    value: 'seed',
                    category: 'developmental stage'
                  },
                  {
                    value: 'fresh_sample',
                    category: 'specimen with known storage state'
                  },
                  {
                    value: 'micropylar endosperm',
                    category: 'organism part'
                  },
                  {
                    unit: 'degree celsius',
                    value: 18,
                    category: 'temperature'
                  }
                ],
                'labeled-extract': {
                  name: 'Sample9 labeled_extract',
                  label: 'biotin'
                }
              },
              is_ccdl: false,
              created_at: '2018-12-19T15:46:50.194593Z',
              last_modified: '2018-12-19T15:46:50.194593Z'
            }
          ],
          results: [
            {
              id: 795037,
              processor: {
                id: 99,
                name: 'Affymetrix SCAN',
                version: 'v1.4.7',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.26.0',
                    affy: '1.56.0',
                    limma: '3.34.9',
                    oligo: '1.42.0',
                    affyio: '1.48.0',
                    'SCAN.UPC': '2.20.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.6',
                    AnnotationDbi: '1.40.0'
                  },
                  os_pkg: {
                    'r-base': '3.4.4-1xenial0',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    rpy2: '2.9.5',
                    Django: '2.1.2',
                    'data-refinery-common': '1.4.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url:
            'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/samples',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'micropylar endosperm',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T15:46:47.663926Z',
          last_modified: '2018-12-19T20:14:26.480126Z',
          original_files: [1094881],
          computed_files: [910894],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['E-MTAB-1811']
        },
        {
          id: 673580,
          title: 'Sample10 RNA',
          accession_code: 'E-MTAB-1811-Sample10',
          source_database: 'ARRAY_EXPRESS',
          organism: {
            name: 'LEPIDIUM_SATIVUM',
            taxonomy_id: 33125
          },
          platform_accession_code: 'ath1121501',
          platform_name:
            '[ATH1-121501] Affymetrix Arabidopsis ATH1 Genome Array',
          pretty_platform:
            'Affymetrix Arabidopsis ATH1 Genome Array (ath1121501)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: [
            {
              Text: 'P-AFFY-3 GeneChip hybridization',
              Type: 'nucleic acid hybridization to array protocol',
              Accession: 'P-AFFY-3',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Title: Affymetrix CEL analysis. Description:',
              Type: 'feature_extraction',
              Accession: 'P-AFFY-6',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "After-ripened Lepidium sativum L. FR1 ('Gartenkresse, einfache') seeds (Juliwa, Heidelberg, Germany) were incubated in petri dishes on two layers of filter paper with 6 ml 1/10 Murashige-Skoog salts in continuous white light (ca. 100 µmol•s-1•m-2) at the temperature indicated.",
              Type: 'growth protocol',
              Accession: 'P-MTAB-34006',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Four biological replicates of seeds were sampled after 10h of imbibition at 11 degrees C, 18 degrees C, 27 degrees C and 32 degrees C.  Two tissues (micropylar endosperm cap and radicle) were dissected from individual seeds. The dissected tissues (ca. 200 endosperm caps (Cap), and ca. 50 radicles (Rad)) were pooled for each sample, frozen in liquid nitrogen and stored at -80 ºC.',
              Type: 'treatment protocol',
              Accession: 'P-MTAB-34007',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'CTAB/Chloroform extraction followed by RNA clean-up with the Qiagen Rneasy kit, followed by quantity and quality controls',
              Type: 'nucleic acid extraction protocol',
              Accession: 'P-MTAB-34008',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "The Affymetrix 3' IVT-Express Labeling Kit (#901229) was used to synthesize Biotin-labeled cRNA. From each RNA sample 100 ng was used as input for the labeling reactions.",
              Type: 'nucleic acid labeling protocol',
              Accession: 'P-MTAB-34009',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            }
          ],
          annotations: [
            {
              data: {
                file: [
                  {
                    url: 'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip/101650-081_60-18degreesCAP.CEL',
                    name: '101650-081_60-18degreesCAP.CEL',
                    type: 'data',
                    comment: {
                      name: 'ArrayExpress FTP file',
                      value:
                        'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip'
                    }
                  }
                ],
                assay: {
                  name: 'Assay 10'
                },
                source: {
                  name: 'Sample10'
                },
                extract: {
                  name: 'Sample10 RNA'
                },
                variable: [
                  {
                    name: 'temperature',
                    unit: 'degree celsius',
                    value: 18
                  },
                  {
                    name: 'organism part',
                    value: 'micropylar endosperm'
                  }
                ],
                characteristic: [
                  {
                    value: 'Lepidium sativum',
                    category: 'organism'
                  },
                  {
                    value: 'seed',
                    category: 'developmental stage'
                  },
                  {
                    value: 'fresh_sample',
                    category: 'specimen with known storage state'
                  },
                  {
                    value: 'micropylar endosperm',
                    category: 'organism part'
                  },
                  {
                    unit: 'degree celsius',
                    value: 18,
                    category: 'temperature'
                  }
                ],
                'labeled-extract': {
                  name: 'Sample10 labeled_extract',
                  label: 'biotin'
                }
              },
              is_ccdl: false,
              created_at: '2018-12-19T15:46:59.602260Z',
              last_modified: '2018-12-19T15:46:59.602260Z'
            }
          ],
          results: [
            {
              id: 795170,
              processor: {
                id: 99,
                name: 'Affymetrix SCAN',
                version: 'v1.4.7',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.26.0',
                    affy: '1.56.0',
                    limma: '3.34.9',
                    oligo: '1.42.0',
                    affyio: '1.48.0',
                    'SCAN.UPC': '2.20.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.6',
                    AnnotationDbi: '1.40.0'
                  },
                  os_pkg: {
                    'r-base': '3.4.4-1xenial0',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    rpy2: '2.9.5',
                    Django: '2.1.2',
                    'data-refinery-common': '1.4.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url:
            'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/samples',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'micropylar endosperm',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T15:46:57.718350Z',
          last_modified: '2018-12-19T20:14:42.072077Z',
          original_files: [1095231],
          computed_files: [911026],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['E-MTAB-1811']
        },
        {
          id: 667549,
          title: 'Sample1 RNA',
          accession_code: 'E-MTAB-1811-Sample1',
          source_database: 'ARRAY_EXPRESS',
          organism: {
            name: 'LEPIDIUM_SATIVUM',
            taxonomy_id: 33125
          },
          platform_accession_code: 'ath1121501',
          platform_name:
            '[ATH1-121501] Affymetrix Arabidopsis ATH1 Genome Array',
          pretty_platform:
            'Affymetrix Arabidopsis ATH1 Genome Array (ath1121501)',
          technology: 'MICROARRAY',
          manufacturer: 'AFFYMETRIX',
          protocol_info: [
            {
              Text: 'P-AFFY-3 GeneChip hybridization',
              Type: 'nucleic acid hybridization to array protocol',
              Accession: 'P-AFFY-3',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Title: Affymetrix CEL analysis. Description:',
              Type: 'feature_extraction',
              Accession: 'P-AFFY-6',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "After-ripened Lepidium sativum L. FR1 ('Gartenkresse, einfache') seeds (Juliwa, Heidelberg, Germany) were incubated in petri dishes on two layers of filter paper with 6 ml 1/10 Murashige-Skoog salts in continuous white light (ca. 100 µmol•s-1•m-2) at the temperature indicated.",
              Type: 'growth protocol',
              Accession: 'P-MTAB-34006',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'Four biological replicates of seeds were sampled after 10h of imbibition at 11 degrees C, 18 degrees C, 27 degrees C and 32 degrees C.  Two tissues (micropylar endosperm cap and radicle) were dissected from individual seeds. The dissected tissues (ca. 200 endosperm caps (Cap), and ca. 50 radicles (Rad)) were pooled for each sample, frozen in liquid nitrogen and stored at -80 ºC.',
              Type: 'treatment protocol',
              Accession: 'P-MTAB-34007',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: 'CTAB/Chloroform extraction followed by RNA clean-up with the Qiagen Rneasy kit, followed by quantity and quality controls',
              Type: 'nucleic acid extraction protocol',
              Accession: 'P-MTAB-34008',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            },
            {
              Text: "The Affymetrix 3' IVT-Express Labeling Kit (#901229) was used to synthesize Biotin-labeled cRNA. From each RNA sample 100 ng was used as input for the labeling reactions.",
              Type: 'nucleic acid labeling protocol',
              Accession: 'P-MTAB-34009',
              Reference:
                'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/protocols'
            }
          ],
          annotations: [
            {
              data: {
                file: [
                  {
                    url: 'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip/101650-072_49-11degreesCAP.CEL',
                    name: '101650-072_49-11degreesCAP.CEL',
                    type: 'data',
                    comment: {
                      name: 'ArrayExpress FTP file',
                      value:
                        'ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MTAB/E-MTAB-1811/E-MTAB-1811.raw.1.zip'
                    }
                  }
                ],
                assay: {
                  name: 'Assay 1'
                },
                source: {
                  name: 'Sample1'
                },
                extract: {
                  name: 'Sample1 RNA'
                },
                variable: [
                  {
                    name: 'temperature',
                    unit: 'degree celsius',
                    value: 11
                  },
                  {
                    name: 'organism part',
                    value: 'micropylar endosperm'
                  }
                ],
                characteristic: [
                  {
                    value: 'Lepidium sativum',
                    category: 'organism'
                  },
                  {
                    value: 'seed',
                    category: 'developmental stage'
                  },
                  {
                    value: 'fresh_sample',
                    category: 'specimen with known storage state'
                  },
                  {
                    value: 'micropylar endosperm',
                    category: 'organism part'
                  },
                  {
                    unit: 'degree celsius',
                    value: 11,
                    category: 'temperature'
                  }
                ],
                'labeled-extract': {
                  name: 'Sample1 labeled_extract',
                  label: 'biotin'
                }
              },
              is_ccdl: false,
              created_at: '2018-12-19T15:45:34.765755Z',
              last_modified: '2018-12-19T15:45:34.765755Z'
            }
          ],
          results: [
            {
              id: 793713,
              processor: {
                id: 99,
                name: 'Affymetrix SCAN',
                version: 'v1.4.7',
                docker_image: 'dr_affymetrix',
                environment: {
                  R: {
                    sva: '3.26.0',
                    affy: '1.56.0',
                    limma: '3.34.9',
                    oligo: '1.42.0',
                    affyio: '1.48.0',
                    'SCAN.UPC': '2.20.0',
                    Brainarray: '22.0.0',
                    Bioconductor: '3.6',
                    AnnotationDbi: '1.40.0'
                  },
                  os_pkg: {
                    'r-base': '3.4.4-1xenial0',
                    python3: '3.5.1-3',
                    'python3-pip': '8.1.1-2ubuntu0.4'
                  },
                  python: {
                    rpy2: '2.9.5',
                    Django: '2.1.2',
                    'data-refinery-common': '1.4.7'
                  },
                  os_distribution: 'Ubuntu 16.04.5 LTS'
                }
              },
              organism_index: null
            }
          ],
          source_archive_url:
            'https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/E-MTAB-1811/samples',
          has_raw: true,
          sex: '',
          age: null,
          specimen_part: 'micropylar endosperm',
          genotype: '',
          disease: '',
          disease_stage: '',
          cell_line: '',
          treatment: '',
          race: '',
          subject: '',
          compound: '',
          time: '',
          is_processed: true,
          is_unable_to_be_processed: false,
          created_at: '2018-12-19T15:45:33.410406Z',
          last_modified: '2018-12-19T20:11:46.779759Z',
          original_files: [1091222],
          computed_files: [909570],
          last_processor_job: null,
          last_downloader_job: null,
          most_recent_smashable_file: null,
          most_recent_quant_file: null,
          contributed_metadata: {},
          contributed_keywords: {},
          experiment_accession_codes: ['E-MTAB-1811']
        }
      ]
    }
  ]
]
