// Taken from the endpoint: https://api.refine.bio/v1/search/
/* TEMPORARY the following prop is added to the mock data for demo purpose
prop name: `status` 
   - ''(default)
   - added
   - processing 
   - add_remaining
   - not_supported
   - request
   - unavailable
*/
export default {
  count: 38030,
  next: 'http://api.refine.bio/v1/search/?limit=10&num_downloadable_samples__gt=0&offset=10&ordering=_score',
  previous: null,
  facets: {
    has_publication: {
      false: 721263,
      true: 251865
    },
    downloadable_organism_names: {
      HOMO_SAPIENS: 521539,
      MUS_MUSCULUS: 292681,
      ARABIDOPSIS_THALIANA: 23999,
      RATTUS_NORVEGICUS: 39311,
      DROSOPHILA_MELANOGASTER: 17239,
      SACCHAROMYCES_CEREVISIAE: 12989,
      DANIO_RERIO: 28598,
      CAENORHABDITIS_ELEGANS: 7764,
      SUS_SCROFA: 5974,
      ZEA_MAYS: 3391,
      GALLUS_GALLUS: 4038,
      GLYCINE_MAX: 5153,
      ORYZA_SATIVA: 2520,
      PSEUDOMONAS_AERUGINOSA: 2263,
      BOS_TAURUS: 5526,
      ESCHERICHIA_COLI: 1387,
      MACACA_MULATTA: 2307,
      TRITICUM_AESTIVUM: 2083,
      XENOPUS_LAEVIS: 759,
      POPULUS_TRICHOCARPA: 1698,
      HORDEUM_VULGARE: 2206,
      ANOPHELES_GAMBIAE: 808,
      OVIS_ARIES: 925,
      PSEUDOMONAS_AERUGINOSA_PAO1: 342,
      STAPHYLOCOCCUS_AUREUS: 420,
      SCHIZOSACCHAROMYCES_POMBE: 310,
      VITIS_VINIFERA: 793,
      ORYZA_SATIVA_INDICA_GROUP: 706,
      MACACA_FASCICULARIS: 979,
      POPULUS_TREMULA_X_POPULUS_ALBA: 246,
      'ESCHERICHIA_COLI_STR._K-12_SUBSTR._MG1655': 551,
      GOSSYPIUM_HIRSUTUM: 318,
      CITRUS_SINENSIS: 248,
      ' ESCHERICHIA_COLI_K-12': 152,
      MUSTELA_PUTORIUS_FURO: 217,
      PLASMODIUM_FALCIPARUM: 106,
      POPULUS_X_CANADENSIS: 143,
      BOS_INDICUS: 57,
      LEPIDIUM_SATIVUM: 139,
      VITIS_HYBRID_CULTIVAR: 251,
      VITIS_RIPARIA: 251
    },
    platform_accession_codes: {
      hgu133plus2: 146159,
      mouse4302: 59093,
      IlluminaHiSeq2000: 125755,
      IlluminaHiSeq2500: 142474,
      mogene10st: 29454,
      hugene10st: 38914,
      ath1121501: 17553,
      hgu133a: 50743,
      NextSeq500: 48894,
      'Illumina_HumanHT-12_V4.0': 25203,
      mogene20st: 8487,
      rat2302: 21304,
      IlluminaGenomeAnalyzerIIx: 9676,
      hugene20st: 3879,
      hgu133a2: 24621,
      'HuEx-1_0-st': 25055,
      mouse430a2: 8776,
      IlluminaHiSeq4000: 20752,
      'Illumina_MouseWG-6_v2.0': 5873,
      IlluminaGenomeAnalyzerII: 6374,
      hta20: 12895,
      mgu74av2: 7353,
      moe430a: 7306,
      yeast2: 4320,
      drosophila2: 4684,
      'Illumina_MouseRef-8_v2.0': 5092,
      IlluminaGenomeAnalyzer: 2590,
      'Illumina_HumanHT-12_V3.0': 11388,
      ragene10st: 3232,
      IlluminaHiSeq1500: 4359,
      rice: 3020,
      IonTorrentProton: 4424,
      porcine: 3486,
      hugene11st: 8579,
      mogene11st: 4234,
      celegans: 2469,
      hgu219: 12227,
      ygs98: 2558,
      paeg1a: 1314,
      chicken: 2211,
      ecoli2: 1710,
      zebrafish: 1852,
      mogene21st: 2599,
      hugene21st: 4373,
      bovine: 2389,
      hthgu133pluspm: 12261,
      rae230a: 4895,
      hgu95av2: 3408,
      rgu34a: 3465,
      ' Illumina_HumanWG-6_V3.0': 2003,
      IlluminaHiSeq1000: 2333,
      'Illumina_Human-6_V2.0': 3778,
      IlluminaHiSeq3000: 3488,
      drosgenome1: 1271,
      '  Illumina_HumanRef-8_v3.0': 3812,
      wheat: 2006,
      hthgu133a: 13003,
      clariomsmouse: 1090,
      rhesus: 1734,
      ragene20st: 2910,
      barley1: 2206,
      'Illumina HiSeq 2000': 957,
      maize: 735,
      ' Illumina_HumanRef-8_v2.0': 2357,
      clariomshuman: 787,
      'Illumina_RatRef-12_V1.0': 913,
      soybean: 3896,
      hgu95a: 2868,
      IlluminaHiScanSQ: 1364,
      ' Illumina_Mouse-6_v1.1': 723,
      xenopuslaevis: 386,
      saureus: 420,
      poplar: 517,
      xlaevis2: 301,
      aragene10st: 525,
      vitisvinifera: 1038,
      NextSeq550: 1280,
      '    Illumina_Human-6_v1.0': 819,
      aragene11st: 685,
      hgu133b: 1150,
      plasmodiumanopheles: 537,
      'Illumina_MouseRef-8_V1.1': 627,
      UNKNOWN: 644,
      mgu74a: 710,
      GPL13534: 446,
      htmg430a: 3198,
      ragene11st: 530,
      GPL80: 592,
      'Illumina Genome Analyzer II': 98,
      moe430b: 465,
      ag: 149,
      bovgene10st: 277,
      cotton: 318,
      ragene21st: 462,
      zebgene10st: 257,
      'Illumina Genome Analyzer': 84,
      'Illumina Genome Analyzer IIx': 116,
      citrus: 248,
      porgene10st: 222,
      chigene10st: 416,
      elegene10st: 152,
      'Illumina HiSeq 2500': 101,
      GPL16384: 116,
      mu11ksuba: 272,
      GPL3718: 219,
      drogene10st: 123,
      porgene11st: 299,
      GPL5082: 139,
      clariomsrat: 137,
      mgu74bv2: 132,
      primeview: 869,
      zebgene11st: 304,
      IlluminaMiSeq: 944,
      canine2: 156,
      GPL5175: 60,
      GPL5811: 39,
      GPL6096: 109,
      'Illumina_Mouse-6_V1.0': 116,
      'Illumina_MouseRef-8_V1.0': 190,
      bovgene11st: 181,
      hgu95b: 616,
      GPL10787: 20,
      GPL6801: 575,
      IonTorrentS5: 82,
      equgene10st: 0,
      mgu74cv2: 105,
      GPL16284: 60,
      GPL4133: 58,
      clariomshumanht: 144,
      hgu95c: 518,
      htmg430pm: 150,
      huex10st: 111,
      ovigene11st: 97,
      rae230b: 233,
      'Illumina_HumanRef-8_v1.0': 38,
      chigene11st: 59,
      rhegene10st: 110,
      GPL1352: 68,
      GPL14550: 11,
      GPL4091: 46,
      GPL8490: 47,
      drogene11st: 38,
      ovigene10st: 37,
      GPL2005: 184,
      GPL4134: 22,
      IonTorrentS5XL: 65,
      mu11ksubb: 82,
      'ABSOLiDSystem3.0': 204,
      GPL10989: 18,
      GPL13497: 6,
      GPL15018: 12,
      GPL15076: 56,
      GPL18609: 180,
      GPL2004: 18,
      GPL23541: 31,
      GPL3720: 15,
      GPL7546: 75,
      GPL9128: 19,
      GPL9138: 14,
      'Illumina HiScanSQ': 12,
      elegene11st: 34,
      mta10: 6,
      rcngene10st: 12,
      rcngene11st: 12,
      rgu34b: 37,
      rusgene10st: 24,
      soygene11st: 35,
      GPL13118: 4,
      GPL13147: 16,
      GPL13328: 24,
      GPL15236: 5,
      GPL16131: 12,
      GPL16304: 25,
      GPL16558: 6,
      GPL2040: 23,
      GPL21145: 64,
      GPL3562: 48,
      GPL4124: 14,
      GPL4910: 8,
      GPL5770: 2,
      GPL6193: 2,
      GPL6673: 12,
      GPL7202: 8,
      GPL7566: 87,
      GPL8179: 24,
      GPL8234: 4,
      'Illumina HiSeq 4000': 6,
      IlluminaMiniSeq: 44,
      IlluminaNovaSeq6000: 167,
      MinION: 127,
      clariomdhuman: 8,
      hgu95d: 15,
      hgu95e: 15,
      rgu34c: 17,
      rhegene11st: 16,
      rusgene11st: 44
    },
    technology: {
      microarray: 609303,
      'rna-seq': 350551,
      unknown: 1200
    }
  },
  results: [
    {
      status: '',
      id: 67685,
      title:
        'Drug-induced change in gene expression across NCI-60 cell lines after exposure to 15 anticancer agents for 2, 6 and 24h',
      publication_title:
        'The NCI Transcriptional Pharmacodynamics Workbench: A Tool to Examine Dynamic Expression Profiling of Therapeutic Response in the NCI-60 Cell Line Panel.',
      description:
        'To identify patterns of drug-induced gene modulation that occur across different cell types, we measured gene expression changes across NCI-60 cell lines after exposure to 15 anticancer agents. The results were integrated into a database and set of interactive analysis tools, the NCI Transcriptional Pharmacodynamics Workbench (NCI TPW), intended to allow exploration of gene expression modulation, including by molecular pathway, drug target, and association with drug sensitivity. We identified common transcriptional responses across drugs and cell types and uncovered cell signaling pathway–specific gene expression changes associated with drug sensitivity. We also demonstrated the value of this tool for investigating clinically relevant molecular hypotheses, utilizing the NCI TPW to assess drug-induced expression changes in genes associated with immune function and epithelial-mesenchymal transition, and to identify candidate biomarkers for drug activity. The NCI TPW provides a comprehensive resource to facilitate understanding of tumor cell characteristics that define sensitivity to anticancer drugs.',
      technology: 'MICROARRAY',
      accession_code: 'GSE116436',
      alternate_accession_code: 'E-GEOD-116436',
      submitter_institution: 'National Cancer Institute',
      has_publication: false,
      publication_doi: '',
      publication_authors: [
        'Monks A',
        'Zhao Y',
        'Hose C',
        'Hamed H',
        'Krushkal J',
        'Fang J',
        'Sonkin D',
        'Palmisano A',
        'Polley EC',
        'Fogli LK',
        'Konaté MM',
        'Miller SB',
        'Simpson MA',
        'Voth AR',
        'Li MC',
        'Harris E',
        'Wu X',
        'Connelly JW',
        'Rapisarda A',
        'Teicher BA',
        'Simon R',
        'Doroshow JH'
      ],
      sample_metadata_fields: ['specimen_part'],
      platform_names: [
        '[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array',
        '[HT_HG-U133A] Affymetrix HT Human Genome U133A Array'
      ],
      platform_accession_codes: ['hgu133a2', 'hthgu133a'],
      organism_names: ['HOMO_SAPIENS'],
      downloadable_organism_names: ['HOMO_SAPIENS'],
      pubmed_id: '30355619',
      num_total_samples: 7623,
      num_processed_samples: 6633,
      num_downloadable_samples: 6633,
      source_first_published: '2018-06-29T00:00:00+00:00'
    },
    {
      status: '',
      id: 49187,
      title:
        'RNA-seq analysis of single cells of the oligodendrocyte lineage from nine distinct regions of the anterior-posterior and dorsal-ventral axis of the mouse juvenile central nervous system',
      publication_title:
        'Oligodendrocyte heterogeneity in the mouse juvenile and adult central nervous system.',
      description:
        '5069 transcriptomes of single oligodendrocyte cells from spinal cord, substantia nigra-ventral tegmental area, striatum, amygdala, hypothalamic nuclei, zona incerta, hippocampus, and somatosensory cortex of male and female mice between post-natal day 21 and 90. The study aimed at identifying diverse populations of oligodendrocytes, and revealing dynamics of oligodendrocyte maturation. Overall design: 5069 individual cells were sampled from CNS regions of mice of various strains as detailed in the protocols section',
      technology: 'RNA-SEQ',
      accession_code: 'SRP066613',
      alternate_accession_code: 'GSE75330',
      submitter_institution: '',
      has_publication: true,
      publication_doi: '',
      publication_authors: [
        'Marques S',
        'Zeisel A',
        'Codeluppi S',
        'van Bruggen D',
        'Mendanha Falcão A',
        'Xiao L',
        'Li H',
        'Häring M',
        'Hochgerner H',
        'Romanov RA',
        'Gyllborg D',
        'Muñoz Manchado A',
        'La Manno G',
        'Lönnerberg P',
        'Floriddia EM',
        'Rezayee F',
        'Ernfors P',
        'Arenas E',
        'Hjerling-Leffler J',
        'Harkany T',
        'Richardson WD',
        'Linnarsson S',
        'Castelo-Branco G'
      ],
      sample_metadata_fields: ['sex', 'cell_line', 'treatment', 'subject'],
      platform_names: ['Illumina HiSeq 2000'],
      platform_accession_codes: ['IlluminaHiSeq2000'],
      organism_names: ['MUS_MUSCULUS'],
      downloadable_organism_names: ['MUS_MUSCULUS'],
      pubmed_id: '27284195',
      num_total_samples: 6282,
      num_processed_samples: 6282,
      num_downloadable_samples: 6282,
      source_first_published: '2016-06-14T00:00:00+00:00'
    },
    {
      status: 'processing',
      id: 61798,
      title: 'Microarray Innovations in LEukemia (MILE) study',
      publication_title:
        'An international standardization programme towards the application of gene expression profiling in routine leukaemia diagnostics: the Microarray Innovations in LEukemia study prephase.',
      description:
        'This SuperSeries is composed of the SubSeries listed below.',
      technology: 'MICROARRAY',
      accession_code: 'GSE13204',
      alternate_accession_code: 'E-GEOD-13204',
      submitter_institution: 'Roche Molecular Systems',
      has_publication: false,
      publication_doi: '',
      publication_authors: [
        'Kohlmann A',
        'Kipps TJ',
        'Rassenti LZ',
        'Downing JR',
        'Shurtleff SA',
        'Mills KI',
        'Gilkes AF',
        'Hofmann WK',
        'Basso G',
        "Dell'orto MC",
        'Foà R',
        'Chiaretti S',
        'De Vos J',
        'Rauhut S',
        'Papenhausen PR',
        'Hernández JM',
        'Lumbreras E',
        'Yeoh AE',
        'Koay ES',
        'Li R',
        'Liu WM',
        'Williams PM',
        'Wieczorek L',
        'Haferlach T'
      ],
      sample_metadata_fields: [],
      platform_names: [],
      platform_accession_codes: [],
      organism_names: ['HOMO_SAPIENS'],
      downloadable_organism_names: ['HOMO_SAPIENS'],
      pubmed_id: '18573112',
      num_total_samples: 3248,
      num_processed_samples: 1492,
      num_downloadable_samples: 1492,
      source_first_published: '2008-10-15T00:00:00+00:00'
    },
    {
      status: 'add_remaining',
      id: 26156,
      title:
        'Host gene expression signatures of H1N1, H3N2, HRV, RSV virus infection in adults',
      publication_title:
        'An individualized predictor of health and disease using paired reference and target samples.',
      description:
        "Consider the problem of designing a panel of complex biomarkers to predict a patient's health or disease state when one can pair his or her current test sample, called a target sample, with the patient's previously acquired healthy sample, called a reference sample. As contrasted to a population averaged reference, this reference sample is individualized. Automated predictor algorithms that compare and contrast the paired samples to each other could result in a new generation of test panels that compare to a person's healthy reference to enhance predictive accuracy. This study develops such an individualized predictor and illustrates the added value of including the healthy reference for design of predictive gene expression panels. The objective is to predict each subject's state of infection, e.g., neither exposed nor infected, exposed but not infected, pre-acute phase of infection, acute phase of infection, post-acute phase of infection. Using gene microarray data collected in a large-scale serially sampled respiratory virus challenge study, we quantify the diagnostic advantage of pairing a person's baseline reference with his or her target sample.",
      technology: 'MICROARRAY',
      accession_code: 'GSE73072',
      alternate_accession_code: 'E-GEOD-73072',
      submitter_institution: 'University of Michigan',
      has_publication: false,
      publication_doi: '',
      publication_authors: [
        'Liu TY',
        'Burke T',
        'Park LP',
        'Woods CW',
        'Zaas AK',
        'Ginsburg GS',
        'Hero AO'
      ],
      sample_metadata_fields: ['specimen_part', 'subject', 'time'],
      platform_names: ['[HG-U133A_2] Affymetrix Human Genome U133A 2.0 Array'],
      platform_accession_codes: ['hgu133a2'],
      organism_names: ['HOMO_SAPIENS'],
      downloadable_organism_names: ['HOMO_SAPIENS'],
      pubmed_id: '26801061',
      num_total_samples: 2886,
      num_processed_samples: 2886,
      num_downloadable_samples: 2886,
      source_first_published: '2015-09-15T00:00:00+00:00'
    },
    {
      status: 'not_supported',
      id: 63899,
      title: 'Transcriptomics and methylomics of human monocytes',
      publication_title:
        'Age-related variations in the methylome associated with gene expression in human monocytes and T cells.',
      description:
        'This SuperSeries is composed of the SubSeries listed below.',
      technology: 'MICROARRAY',
      accession_code: 'GSE56047',
      alternate_accession_code: 'E-GEOD-56047',
      submitter_institution: 'Wake Forest School of Medicine',
      has_publication: false,
      publication_doi: '',
      publication_authors: [
        'Reynolds LM',
        'Taylor JR',
        'Ding J',
        'Lohman K',
        'Johnson C',
        'Siscovick D',
        'Burke G',
        'Post W',
        'Shea S',
        'Jacobs DR Jr',
        'Stunnenberg H',
        'Kritchevsky SB',
        'Hoeschele I',
        'McCall CE',
        'Herrington D',
        'Tracy RP',
        'Liu Y'
      ],
      sample_metadata_fields: ['age'],
      platform_names: ['Illumina HumanHT-12 V4.0 expression beadchip'],
      platform_accession_codes: ['Illumina_HumanHT-12_V4.0'],
      organism_names: ['HOMO_SAPIENS'],
      downloadable_organism_names: ['HOMO_SAPIENS'],
      pubmed_id: '25404168',
      num_total_samples: 2832,
      num_processed_samples: 57,
      num_downloadable_samples: 57,
      source_first_published: '2014-03-19T00:00:00+00:00'
    },
    {
      status: 'request',
      id: 440,
      title:
        'Transcriptome_profiling_of_embryos_collected_for_one_or_more_alleles_identified_by_the_zebrafish_mutation_project',
      publication_title: '',
      description:
        'Paired-end sequence data from the IlluminaHiSeq was prepared for zebrafish embryos collected for one or more alleles identified by the Zebrafish Mutation Project for transcriptome profiling.',
      technology: 'RNA-SEQ',
      accession_code: 'ERP006132',
      alternate_accession_code: null,
      submitter_institution: '',
      has_publication: false,
      publication_doi: '',
      publication_authors: [],
      sample_metadata_fields: [],
      platform_names: ['IlluminaHiSeq2500', 'IlluminaHiSeq2000'],
      platform_accession_codes: ['IlluminaHiSeq2500', 'IlluminaHiSeq2000'],
      organism_names: ['DANIO_RERIO'],
      downloadable_organism_names: ['DANIO_RERIO'],
      pubmed_id: '',
      num_total_samples: 2822,
      num_processed_samples: 2822,
      num_downloadable_samples: 2822,
      source_first_published: '2014-10-15T00:00:00+00:00'
    },
    {
      status: 'unavailable',
      id: 45717,
      title:
        'Temporal patterning of apical progenitors and their daughter neurons in the developing neocortex',
      publication_title:
        'Temporal patterning of apical progenitors and their daughter neurons in the developing neocortex.',
      description:
        'During cortical development, distinct subtypes of glutamatergic neurons are sequentially born and differentiate from dynamic populations of progenitors. How progenitors and their daughter cells are temporally patterned remains unknown. Here, we trace the transcriptional trajectories of successive generations of apical progenitors (APs) and isochronic cohorts of their daughter neurons in the developing mouse neocortex using high temporal resolution parallel single-cell RNA sequencing. We identify and functionally characterize a core set of evolutionarily-conserved temporally patterned genes which drive APs from internally-driven states to more exteroceptive states, revealing a progressively increasing role for extracellular signals as corticogenesis unfolds. These embryonic age-dependent AP molecular states are reflected in their neuronal progeny as successive ground states, onto which essentially conserved early post-mitotic differentiation programs are applied. Thus, temporally unfolding molecular birthmarks present in progenitors act in their post-mitotic progeny as seeds for adult neuronal diversity. Overall design: Investigation of the transcriptional dynamics in time-locked cohorts of cortical cells across embryonic neurogenesis. Flashtag is injected at 4 ages (E12, E13, E14, E15), and cells collected 1H, 24H, 96H after birth (= a total of 12 conditions) and analyzed by single cell transcriptomics.',
      technology: 'RNA-SEQ',
      accession_code: 'SRP158666',
      alternate_accession_code: 'GSE118953',
      submitter_institution: '',
      has_publication: true,
      publication_doi: '',
      publication_authors: [
        'Telley L',
        'Agirman G',
        'Prados J',
        'Amberg N',
        'Fièvre S',
        'Oberst P',
        'Bartolini G',
        'Vitali I',
        'Cadilhac C',
        'Hippenmeyer S',
        'Nguyen L',
        'Dayer A',
        'Jabaudon D'
      ],
      sample_metadata_fields: ['subject'],
      platform_names: ['Illumina HiSeq 2500'],
      platform_accession_codes: ['IlluminaHiSeq2500'],
      organism_names: ['MUS_MUSCULUS'],
      downloadable_organism_names: ['MUS_MUSCULUS'],
      pubmed_id: '31073041',
      num_total_samples: 2756,
      num_processed_samples: 2756,
      num_downloadable_samples: 2756,
      source_first_published: '2019-05-11T00:00:00+00:00'
    },
    {
      id: 65064,
      title: 'Spatio-temporal transcriptome of the human brain',
      publication_title: 'Spatio-temporal transcriptome of the human brain.',
      description:
        'The development of the human brain is a complex and precisely regulated process that unfolds over a protracted period of time. Human-specific features of this process, especially the ways in which highly complex neural circuits of the cerebral cortex form, are likely to be important factors in the evolution of human specializations. However, in addition to giving us remarkable cognitive and motor abilities, the formation of intricate neural circuits may have also increased our susceptibility to psychiatric and neurodegenerative disorders. Furthermore, substantial evidence suggests that the symptoms and progression of many brain disorders are dramatically influenced by genetic and developmental processes that define regional cell phenotypes and connectivity. Sex differences also play an important role in brain development and function and are a risk factor for several brain disorders, such as autism spectrum disorders (ASD) and depression. Thus understanding the spatiotemporal dynamics and functional organization of the brain transcriptome is essential to teasing out the keys to human neurodevelopment, sexual dimorphism, and evolution as well as our increased susceptibility to certain brain disorders.  Most transcriptome studies of the developing brain have been restricted to rodents, and those performed in humans and nonhuman primates have included relatively small sample sizes and predominantly focused on few regions or developmental time points. Because many prominent features of human brain development significantly diverge from those of well-characterized model organisms, the translation of knowledge across species is difficult, and it is likely that many underlying genetic processes have gone undetected. In this study, we have taken a genome-wide approach to analyze the human transcriptome at single-exon resolution with ~1.4 million exon-level probe sets in 16 brain regions from donors representing both sexes and multiple ethnicities, across pre and postnatal development, including adolescence, and adulthood. We also generated genome-wide genotype data for 2.5 million single nucleotide polymorphisms (SNPs) and copy number variants (CNVs) for each specimen. Our analyses of the data revealed several features of the human brain transcriptome: spatiotemporal expression dynamics of individual and functionally related groups of genes, differential exon usage, sex-specific expression patterns and exon usage, and organization of the transcriptome into functional modules. We also profiled developmental trajectories of genes important for neurobiological themes and genes associated with ASD and schizophrenia. Finally, we present associations between specific SNPs and gene expression levels in different brain regions across development. The dataset presented here provides research opportunities and a wealth of information not previously available to the scientific community.',
      technology: 'MICROARRAY',
      accession_code: 'GSE25219',
      alternate_accession_code: 'E-GEOD-25219',
      submitter_institution: 'Yale University',
      has_publication: false,
      publication_doi: '',
      publication_authors: [
        'Kang HJ',
        'Kawasawa YI',
        'Cheng F',
        'Zhu Y',
        'Xu X',
        'Li M',
        'Sousa AM',
        'Pletikos M',
        'Meyer KA',
        'Sedmak G',
        'Guennel T',
        'Shin Y',
        'Johnson MB',
        'Krsnik Z',
        'Mayer S',
        'Fertuzinhos S',
        'Umlauf S',
        'Lisgo SN',
        'Vortmeyer A',
        'Weinberger DR',
        'Mane S',
        'Hyde TM',
        'Huttner A',
        'Reimers M',
        'Kleinman JE',
        'Sestan N'
      ],
      sample_metadata_fields: ['sex', 'age'],
      platform_names: [
        '[HuEx-1_0-st] Affymetrix Human Exon 1.0 ST Array [transcript (gene) version]'
      ],
      platform_accession_codes: ['HuEx-1_0-st'],
      organism_names: ['HOMO_SAPIENS'],
      downloadable_organism_names: ['HOMO_SAPIENS'],
      pubmed_id: '22031440',
      num_total_samples: 2680,
      num_processed_samples: 2667,
      num_downloadable_samples: 2667,
      source_first_published: '2010-11-09T00:00:00+00:00'
    },
    {
      id: 61636,
      title:
        'IMI MARCAR Project: towards novel biomarkers for cancer risk assessment',
      publication_title:
        'Phenobarbital mediates an epigenetic switch at the constitutive androstane receptor (CAR) target gene Cyp2b10 in the liver of B6C3F1 mice.',
      description:
        'This SuperSeries is composed of the SubSeries listed below.',
      technology: 'MICROARRAY',
      accession_code: 'GSE68387',
      alternate_accession_code: 'E-GEOD-68387',
      submitter_institution: 'Novartis',
      has_publication: false,
      publication_doi: '',
      publication_authors: [
        'Lempiäinen H',
        'Müller A',
        'Brasa S',
        'Teo SS',
        'Roloff TC',
        'Morawiec L',
        'Zamurovic N',
        'Vicart A',
        'Funhoff E',
        'Couttet P',
        'Schübeler D',
        'Grenet O',
        'Marlowe J',
        'Moggs J',
        'Terranova R'
      ],
      sample_metadata_fields: [
        'sex',
        'age',
        'specimen_part',
        'treatment',
        'subject',
        'time'
      ],
      platform_names: [
        '[RaGene-2_0-st] Affymetrix Rat Gene 2.0 ST Array',
        '[Mouse430A_2] Affymetrix Mouse Genome 430A 2.0 Array',
        '[RAE230A] Affymetrix Rat Expression 230A Array',
        '[HT_HG-U133_Plus_PM] Affymetrix HT HG-U133+ PM Array Plate',
        '[Rat230_2] Affymetrix Rat Genome 230 2.0 Array',
        '[Mouse430_2] Affymetrix Mouse Genome 430 2.0 Array'
      ],
      platform_accession_codes: [
        'mouse430a2',
        'ragene20st',
        'hthgu133pluspm',
        'mouse4302',
        'rae230a',
        'rat2302'
      ],
      organism_names: ['MUS_MUSCULUS', 'HOMO_SAPIENS', 'RATTUS_NORVEGICUS'],
      downloadable_organism_names: [
        'MUS_MUSCULUS',
        'HOMO_SAPIENS',
        'RATTUS_NORVEGICUS'
      ],
      pubmed_id: '21455306',
      num_total_samples: 2666,
      num_processed_samples: 1938,
      num_downloadable_samples: 1938,
      source_first_published: '2015-04-29T00:00:00+00:00'
    },
    {
      id: 62525,
      title:
        'Expression data measured by Nanostring and microarray of monocyte-derived dendritic cells from healthy individuals stimulated with LPS, influenza, or IFN-beta, or left unstimulated',
      publication_title:
        'Common genetic variants modulate pathogen-sensing responses in human dendritic cells.',
      description:
        'This SuperSeries is composed of the SubSeries listed below.',
      technology: 'MICROARRAY',
      accession_code: 'GSE57542',
      alternate_accession_code: 'E-GEOD-57542',
      submitter_institution: 'Broad Institute',
      has_publication: false,
      publication_doi: '',
      publication_authors: [
        'Lee MN',
        'Ye C',
        'Villani AC',
        'Raj T',
        'Li W',
        'Eisenhaure TM',
        'Imboywa SH',
        'Chipendo PI',
        'Ran FA',
        'Slowikowski K',
        'Ward LD',
        'Raddassi K',
        'McCabe C',
        'Lee MH',
        'Frohlich IY',
        'Hafler DA',
        'Kellis M',
        'Raychaudhuri S',
        'Zhang F',
        'Stranger BE',
        'Benoist CO',
        'De Jager PL',
        'Regev A',
        'Hacohen N'
      ],
      sample_metadata_fields: ['sex', 'age', 'race', 'subject'],
      platform_names: ['[HuGene-1_0-st] Affymetrix Human Gene 1.0 ST Array'],
      platform_accession_codes: ['hugene10st'],
      organism_names: ['HOMO_SAPIENS'],
      downloadable_organism_names: ['HOMO_SAPIENS'],
      pubmed_id: '24604203',
      num_total_samples: 2554,
      num_processed_samples: 228,
      num_downloadable_samples: 228,
      source_first_published: '2014-05-12T00:00:00+00:00'
    }
  ]
}
